{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# - SNVs: uk_all\n",
    "\n",
    "Generation of results for SNVs in project 321. In this notebook there is the generation of some of the commands for the calling of SNVs in the cohort, and the annotation of results for non-sense mutations in TSGs and driver events that have been annotated in the oncoKB database.\n",
    "\n",
    "Alseo, in the last part, there is a section to check how many of those calls were also identified in the output of the matched analysis.\n",
    "\n",
    "The information of the workflow in order to generate the different files to call SNVs in an unmatched-setting are specified in the file design_steps_SNVs.txt (/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/design_steps_SNVs.txt)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/gutierj2/default_python3/lib/python3.6/site-packages/statsmodels/tools/_testing.py:19: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.\n",
      "  import pandas.util.testing as tm\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "import isabl_cli as ii\n",
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.set_option('display.max_rows', None)\n",
    "pd.set_option('display.max_columns', None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Retrieving 148 from experiments API endpoint...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158604_T01_01_WG01 /work/isabl/data/analyses/29/38/252938/IID_H158604_T01_01_WG01.bam\n",
      "IID_H158605_T01_01_WG01 /work/isabl/data/analyses/29/40/252940/IID_H158605_T01_01_WG01.bam\n",
      "IID_H158606_T01_01_WG01 /work/isabl/data/analyses/29/42/252942/IID_H158606_T01_01_WG01.bam\n",
      "IID_H158607_T01_01_WG01 /work/isabl/data/analyses/29/44/252944/IID_H158607_T01_01_WG01.bam\n",
      "IID_H158608_T01_01_WG01 /work/isabl/data/analyses/29/46/252946/IID_H158608_T01_01_WG01.bam\n",
      "IID_H158609_T01_01_WG01 /work/isabl/data/analyses/29/48/252948/IID_H158609_T01_01_WG01.bam\n",
      "IID_H158610_T01_01_WG01 /work/isabl/data/analyses/29/50/252950/IID_H158610_T01_01_WG01.bam\n",
      "IID_H158611_T01_01_WG01 /work/isabl/data/analyses/29/52/252952/IID_H158611_T01_01_WG01.bam\n",
      "IID_H158612_T01_01_WG01 /work/isabl/data/analyses/29/54/252954/IID_H158612_T01_01_WG01.bam\n",
      "IID_H158613_T01_01_WG01 /work/isabl/data/analyses/29/56/252956/IID_H158613_T01_01_WG01.bam\n",
      "IID_H158614_T01_01_WG01 /work/isabl/data/analyses/29/58/252958/IID_H158614_T01_01_WG01.bam\n",
      "IID_H158615_T01_01_WG01 /work/isabl/data/analyses/29/60/252960/IID_H158615_T01_01_WG01.bam\n",
      "IID_H158616_T01_01_WG01 /work/isabl/data/analyses/29/62/252962/IID_H158616_T01_01_WG01.bam\n",
      "IID_H158617_T01_01_WG01 /work/isabl/data/analyses/29/64/252964/IID_H158617_T01_01_WG01.bam\n",
      "IID_H158618_T01_01_WG01 /work/isabl/data/analyses/29/66/252966/IID_H158618_T01_01_WG01.bam\n",
      "IID_H158619_T01_01_WG01 /work/isabl/data/analyses/29/68/252968/IID_H158619_T01_01_WG01.bam\n",
      "IID_H158620_T01_01_WG01 /work/isabl/data/analyses/29/70/252970/IID_H158620_T01_01_WG01.bam\n",
      "IID_H158621_T01_01_WG01 /work/isabl/data/analyses/29/72/252972/IID_H158621_T01_01_WG01.bam\n",
      "IID_H158622_T01_01_WG01 /work/isabl/data/analyses/29/74/252974/IID_H158622_T01_01_WG01.bam\n",
      "IID_H158623_T01_01_WG01 /work/isabl/data/analyses/29/76/252976/IID_H158623_T01_01_WG01.bam\n",
      "IID_H158624_T01_01_WG01 /work/isabl/data/analyses/29/78/252978/IID_H158624_T01_01_WG01.bam\n",
      "IID_H158625_T01_01_WG01 /work/isabl/data/analyses/29/80/252980/IID_H158625_T01_01_WG01.bam\n",
      "IID_H158626_T01_01_WG01 /work/isabl/data/analyses/29/82/252982/IID_H158626_T01_01_WG01.bam\n",
      "IID_H158627_T01_01_WG01 /work/isabl/data/analyses/29/84/252984/IID_H158627_T01_01_WG01.bam\n",
      "IID_H158628_T01_01_WG01 /work/isabl/data/analyses/29/86/252986/IID_H158628_T01_01_WG01.bam\n",
      "IID_H158629_T01_01_WG01 /work/isabl/data/analyses/29/88/252988/IID_H158629_T01_01_WG01.bam\n",
      "IID_H158630_T01_01_WG01 /work/isabl/data/analyses/29/90/252990/IID_H158630_T01_01_WG01.bam\n",
      "IID_H158631_T01_01_WG01 /work/isabl/data/analyses/29/92/252992/IID_H158631_T01_01_WG01.bam\n",
      "IID_H158632_T01_01_WG01 /work/isabl/data/analyses/29/94/252994/IID_H158632_T01_01_WG01.bam\n",
      "IID_H158633_T01_01_WG01 /work/isabl/data/analyses/29/96/252996/IID_H158633_T01_01_WG01.bam\n",
      "IID_H158634_T01_01_WG01 /work/isabl/data/analyses/98/98/259898/IID_H158634_T01_01_WG01.bam\n",
      "IID_H158635_T01_01_WG01 /work/isabl/data/analyses/29/99/252999/IID_H158635_T01_01_WG01.bam\n",
      "IID_H158636_T01_01_WG01 /work/isabl/data/analyses/34/25/253425/IID_H158636_T01_01_WG01.bam\n",
      "IID_H158637_T01_01_WG01 /work/isabl/data/analyses/34/27/253427/IID_H158637_T01_01_WG01.bam\n",
      "IID_H158638_T01_01_WG01 /work/isabl/data/analyses/34/29/253429/IID_H158638_T01_01_WG01.bam\n",
      "IID_H158639_T01_01_WG01 /work/isabl/data/analyses/34/31/253431/IID_H158639_T01_01_WG01.bam\n",
      "IID_H158640_T01_01_WG01 /work/isabl/data/analyses/34/33/253433/IID_H158640_T01_01_WG01.bam\n",
      "IID_H158641_T01_01_WG01 /work/isabl/data/analyses/34/35/253435/IID_H158641_T01_01_WG01.bam\n",
      "IID_H158642_T01_01_WG01 /work/isabl/data/analyses/34/37/253437/IID_H158642_T01_01_WG01.bam\n",
      "IID_H158643_T01_01_WG01 /work/isabl/data/analyses/34/38/253438/IID_H158643_T01_01_WG01.bam\n",
      "IID_H158644_T01_01_WG01 /work/isabl/data/analyses/34/39/253439/IID_H158644_T01_01_WG01.bam\n",
      "IID_H158645_T01_01_WG01 /work/isabl/data/analyses/34/40/253440/IID_H158645_T01_01_WG01.bam\n",
      "IID_H158646_T01_01_WG01 /work/isabl/data/analyses/34/41/253441/IID_H158646_T01_01_WG01.bam\n",
      "IID_H158647_T01_01_WG01 /work/isabl/data/analyses/34/42/253442/IID_H158647_T01_01_WG01.bam\n",
      "IID_H158648_T01_01_WG01 /work/isabl/data/analyses/34/43/253443/IID_H158648_T01_01_WG01.bam\n",
      "IID_H158649_T01_01_WG01 /work/isabl/data/analyses/34/44/253444/IID_H158649_T01_01_WG01.bam\n",
      "IID_H158650_T01_01_WG01 /work/isabl/data/analyses/36/50/253650/IID_H158650_T01_01_WG01.bam\n",
      "IID_H158651_T01_01_WG01 /work/isabl/data/analyses/36/64/253664/IID_H158651_T01_01_WG01.bam\n",
      "IID_H158662_T01_01_WG01 /work/isabl/data/analyses/01/36/260136/IID_H158662_T01_01_WG01.bam\n",
      "IID_H158663_T01_01_WG01 /work/isabl/data/analyses/01/38/260138/IID_H158663_T01_01_WG01.bam\n",
      "IID_H158664_T01_01_WG01 /work/isabl/data/analyses/01/40/260140/IID_H158664_T01_01_WG01.bam\n",
      "IID_H158665_T01_01_WG01 /work/isabl/data/analyses/01/51/260151/IID_H158665_T01_01_WG01.bam\n",
      "IID_H158666_T01_01_WG01 /work/isabl/data/analyses/01/53/260153/IID_H158666_T01_01_WG01.bam\n",
      "IID_H158667_T01_01_WG01 /work/isabl/data/analyses/01/55/260155/IID_H158667_T01_01_WG01.bam\n",
      "IID_H158668_T01_01_WG01 /work/isabl/data/analyses/01/57/260157/IID_H158668_T01_01_WG01.bam\n",
      "IID_H158669_T01_01_WG01 /work/isabl/data/analyses/01/59/260159/IID_H158669_T01_01_WG01.bam\n",
      "IID_H158670_T01_01_WG01 /work/isabl/data/analyses/01/61/260161/IID_H158670_T01_01_WG01.bam\n",
      "IID_H158671_T01_01_WG01 /work/isabl/data/analyses/01/63/260163/IID_H158671_T01_01_WG01.bam\n"
     ]
    }
   ],
   "source": [
    "for exper in ii.get_experiments(projects=321):\n",
    "    sample = exper.system_id\n",
    "    if ('_T' in sample) & ('WG' in sample):\n",
    "        bam = ii.Experiment(sample).bam_files.GRCh37.url\n",
    "        if bam:\n",
    "            print(sample,bam)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# - TSGs: command"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>sample</th>\n",
       "      <th>bam</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>IID_H158604_T01_01_WG01</td>\n",
       "      <td>/work/isabl/data/analyses/29/38/252938/IID_H15...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>IID_H158605_T01_01_WG01</td>\n",
       "      <td>/work/isabl/data/analyses/29/40/252940/IID_H15...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>IID_H158606_T01_01_WG01</td>\n",
       "      <td>/work/isabl/data/analyses/29/42/252942/IID_H15...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>IID_H158607_T01_01_WG01</td>\n",
       "      <td>/work/isabl/data/analyses/29/44/252944/IID_H15...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>/work/isabl/data/analyses/29/46/252946/IID_H15...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                    sample                                                bam\n",
       "0  IID_H158604_T01_01_WG01  /work/isabl/data/analyses/29/38/252938/IID_H15...\n",
       "1  IID_H158605_T01_01_WG01  /work/isabl/data/analyses/29/40/252940/IID_H15...\n",
       "2  IID_H158606_T01_01_WG01  /work/isabl/data/analyses/29/42/252942/IID_H15...\n",
       "3  IID_H158607_T01_01_WG01  /work/isabl/data/analyses/29/44/252944/IID_H15...\n",
       "4  IID_H158608_T01_01_WG01  /work/isabl/data/analyses/29/46/252946/IID_H15..."
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ALL_proj321_bams=pd.read_csv('/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/ALL_proj321_bams.list',sep='\\t',names=['sample','bam'])\n",
    "ALL_proj321_bams.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_singularity_mutect.out       -J singIID_H158604_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/38/252938/IID_H158604_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_singularity_mutect.out       -J singIID_H158605_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/40/252940/IID_H158605_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_singularity_mutect.out       -J singIID_H158606_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/42/252942/IID_H158606_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_singularity_mutect.out       -J singIID_H158607_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/44/252944/IID_H158607_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_singularity_mutect.out       -J singIID_H158608_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/46/252946/IID_H158608_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_singularity_mutect.out       -J singIID_H158609_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/48/252948/IID_H158609_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_singularity_mutect.out       -J singIID_H158610_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/50/252950/IID_H158610_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_singularity_mutect.out       -J singIID_H158611_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/52/252952/IID_H158611_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_singularity_mutect.out       -J singIID_H158612_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/54/252954/IID_H158612_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_singularity_mutect.out       -J singIID_H158613_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/56/252956/IID_H158613_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_singularity_mutect.out       -J singIID_H158614_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/58/252958/IID_H158614_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_singularity_mutect.out       -J singIID_H158615_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/60/252960/IID_H158615_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_singularity_mutect.out       -J singIID_H158616_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/62/252962/IID_H158616_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_singularity_mutect.out       -J singIID_H158617_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/64/252964/IID_H158617_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_singularity_mutect.out       -J singIID_H158618_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/66/252966/IID_H158618_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_singularity_mutect.out       -J singIID_H158619_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/68/252968/IID_H158619_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_singularity_mutect.out       -J singIID_H158620_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/70/252970/IID_H158620_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_singularity_mutect.out       -J singIID_H158621_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/72/252972/IID_H158621_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_singularity_mutect.out       -J singIID_H158622_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/74/252974/IID_H158622_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_singularity_mutect.out       -J singIID_H158623_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/76/252976/IID_H158623_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_singularity_mutect.out       -J singIID_H158624_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/78/252978/IID_H158624_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_singularity_mutect.out       -J singIID_H158625_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/80/252980/IID_H158625_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_singularity_mutect.out       -J singIID_H158626_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/82/252982/IID_H158626_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_singularity_mutect.out       -J singIID_H158627_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/84/252984/IID_H158627_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_singularity_mutect.out       -J singIID_H158628_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/86/252986/IID_H158628_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_singularity_mutect.out       -J singIID_H158629_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/88/252988/IID_H158629_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_singularity_mutect.out       -J singIID_H158630_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/90/252990/IID_H158630_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_singularity_mutect.out       -J singIID_H158631_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/92/252992/IID_H158631_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_singularity_mutect.out       -J singIID_H158632_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/94/252994/IID_H158632_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_singularity_mutect.out       -J singIID_H158633_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/96/252996/IID_H158633_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_singularity_mutect.out       -J singIID_H158634_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/98/98/259898/IID_H158634_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_singularity_mutect.out       -J singIID_H158635_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/29/99/252999/IID_H158635_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_singularity_mutect.out       -J singIID_H158636_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/25/253425/IID_H158636_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_singularity_mutect.out       -J singIID_H158637_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/27/253427/IID_H158637_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_singularity_mutect.out       -J singIID_H158638_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/29/253429/IID_H158638_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_singularity_mutect.out       -J singIID_H158639_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/31/253431/IID_H158639_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_singularity_mutect.out       -J singIID_H158640_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/33/253433/IID_H158640_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_singularity_mutect.out       -J singIID_H158641_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/35/253435/IID_H158641_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_singularity_mutect.out       -J singIID_H158642_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/37/253437/IID_H158642_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_singularity_mutect.out       -J singIID_H158643_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/38/253438/IID_H158643_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_singularity_mutect.out       -J singIID_H158644_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/39/253439/IID_H158644_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_singularity_mutect.out       -J singIID_H158645_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/40/253440/IID_H158645_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_singularity_mutect.out       -J singIID_H158646_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/41/253441/IID_H158646_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_singularity_mutect.out       -J singIID_H158647_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/42/253442/IID_H158647_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_singularity_mutect.out       -J singIID_H158648_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/43/253443/IID_H158648_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_singularity_mutect.out       -J singIID_H158649_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/34/44/253444/IID_H158649_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_singularity_mutect.out       -J singIID_H158650_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/36/50/253650/IID_H158650_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_singularity_mutect.out       -J singIID_H158651_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/36/64/253664/IID_H158651_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_singularity_mutect.out       -J singIID_H158662_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/36/260136/IID_H158662_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_singularity_mutect.out       -J singIID_H158663_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/38/260138/IID_H158663_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_singularity_mutect.out       -J singIID_H158664_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/40/260140/IID_H158664_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_singularity_mutect.out       -J singIID_H158665_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/51/260151/IID_H158665_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_singularity_mutect.out       -J singIID_H158666_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/53/260153/IID_H158666_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_singularity_mutect.out       -J singIID_H158667_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/55/260155/IID_H158667_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_singularity_mutect.out       -J singIID_H158668_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/57/260157/IID_H158668_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_singularity_mutect.out       -J singIID_H158669_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/59/260159/IID_H158669_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_singularity_mutect.out       -J singIID_H158670_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/61/260161/IID_H158670_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01.vcf.gz &\n",
      "\n",
      "bsub       -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_singularity_mutect.err       -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_singularity_mutect.out       -J singIID_H158671_T01_01_WG01       -M 2       -W 40000       /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec         --scratch /ifs/work/leukgen/temp         --home /ifs/res/leukgen/home/gutierj2         --workdir /ifs/work/leukgen/temp         --bind /ifs:/ifs,/work:/work         /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg           /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2           -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta           -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed           -I /work/isabl/data/analyses/01/63/260163/IID_H158671_T01_01_WG01.bam           -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01.vcf.gz &\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for index,row in ALL_proj321_bams.iterrows():\n",
    "    sample=row['sample']\n",
    "    bam=row['bam']\n",
    "    print('bsub \\\n",
    "      -e /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/',sample,'_singularity_mutect.err \\\n",
    "      -o /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/',sample,'_singularity_mutect.out \\\n",
    "      -J sing',sample,' \\\n",
    "      -M 2 \\\n",
    "      -W 40000 \\\n",
    "      /opt/common/CentOS_6-dev/singularity/singularity-2.4.2/bin/singularity exec \\\n",
    "        --scratch /ifs/work/leukgen/temp \\\n",
    "        --home /ifs/res/leukgen/home/gutierj2 \\\n",
    "        --workdir /ifs/work/leukgen/temp \\\n",
    "        --bind /ifs:/ifs,/work:/work \\\n",
    "        /juno/work/isabl/local/toil_mutect/v1.2.7/toil_mutect-v1.2.7.simg \\\n",
    "          /work/isabl/home/gutierj2/software/gatk-4.1.3.0/gatk Mutect2 \\\n",
    "          -R /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta \\\n",
    "          -L /work/isabl/home/gutierj2/tsg_unmatched/tsg_coordinates.bed \\\n",
    "          -I ',bam,' \\\n",
    "          -O /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/',sample,'.vcf.gz &','\\n',sep='')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- Normal command: bcftools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "58\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01.vcf.gz']"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vcfs_tsg_list='/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/*.vcf.gz'\n",
    "vcfs_tsg_list=!ls {vcfs_tsg_list}\n",
    "print(len(vcfs_tsg_list))\n",
    "vcfs_tsg_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158604_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158605_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158606_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158607_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158608_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158609_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158610_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158611_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158612_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158613_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158614_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158615_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158616_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158617_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158618_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158619_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158620_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158621_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158622_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158623_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158624_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158625_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158626_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158627_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158628_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158629_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158630_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158631_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158632_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158633_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158634_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158635_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158636_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158637_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158638_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158639_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158640_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158641_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158642_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158643_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158644_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158645_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158646_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158647_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158648_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158649_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158650_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158651_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158662_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158663_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158664_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158665_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158666_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158667_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158668_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158669_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158670_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01.vcf.gz \n",
      "\n",
      "bcftools norm    --no-version    -m -    --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta    --output  IID_H158671_T01_01_WG01.norm.vcf      /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01.vcf.gz \n",
      "\n"
     ]
    }
   ],
   "source": [
    "for vcf in vcfs_tsg_list:\n",
    "    sample=vcf.split('/')[9].split('.vcf')[0]\n",
    "    output=sample+'.norm.vcf'\n",
    "    print('bcftools norm \\\n",
    "   --no-version \\\n",
    "   -m - \\\n",
    "   --fasta-ref /work/isabl/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta \\\n",
    "   --output ',output,' \\\n",
    "   ',vcf,'\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "58\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_2.norm.vcf\n",
      "/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_2.norm.vcf\n"
     ]
    }
   ],
   "source": [
    "vcfsnorm_tsg_list='/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/*_2.norm.vcf'\n",
    "vcfsnorm_tsg_list=!ls {vcfsnorm_tsg_list}\n",
    "print(len(vcfsnorm_tsg_list))\n",
    "for n in vcfsnorm_tsg_list:\n",
    "    print(n)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_2.norm.vcf.gz\n",
      "bgzip -c  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_2.norm.vcf > /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_2.norm.vcf.gz\n"
     ]
    }
   ],
   "source": [
    "for vcf in vcfsnorm_tsg_list:\n",
    "    sample=vcf.split('/')[9].split('_2.norm')[0]\n",
    "    output=vcf+'.gz'\n",
    "    print('bgzip -c ',vcf,'>',output)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "58\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_2.norm.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_2.norm.vcf.gz']"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vcfsnormgz_tsg_list='/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/*_2.norm.vcf.gz'\n",
    "vcfsnormgz_tsg_list=!ls {vcfsnormgz_tsg_list}\n",
    "print(len(vcfsnormgz_tsg_list))\n",
    "vcfsnormgz_tsg_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158604_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158605_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158606_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158607_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158608_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158609_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158610_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158611_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158612_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158613_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158614_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158615_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158616_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158617_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158618_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158619_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158620_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158621_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158622_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158623_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158624_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158625_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158626_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158627_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158628_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158629_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158630_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158631_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158632_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158633_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158634_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158635_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158636_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158637_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158638_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158639_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158640_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158641_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158642_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158643_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158644_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158645_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158646_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158647_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158648_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158649_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158650_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158651_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158662_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158663_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158664_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158665_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158666_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158667_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158668_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158669_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158670_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_2.norm.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs     --output_prefix IID_H158671_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n"
     ]
    }
   ],
   "source": [
    "for vcf in vcfsnormgz_tsg_list:\n",
    "    sample=vcf.split('/')[9].split('.norm.vcf')[0]\n",
    "    print('click_annotvcf annotvcf \\\n",
    "    --input_vcf ',vcf,' \\\n",
    "    --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs \\\n",
    "    --output_prefix ',sample,' \\\n",
    "    --assembly GRCH37D5 \\\n",
    "    --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta \\\n",
    "    --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz \\\n",
    "    --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/ \\\n",
    "    --ensembl-version 91 \\\n",
    "    --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female \\\n",
    "    --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female \\\n",
    "    --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP \\\n",
    "    --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 ','\\n',sep='')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- Stop codons:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "58\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_2.output.annot.tsv.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_2.output.annot.tsv.gz']"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "annots_list='/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/*_2.output.annot.tsv.gz'\n",
    "annots_list=!ls {annots_list}\n",
    "print(len(annots_list))\n",
    "annots_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158604_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158605_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158606_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158607_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158608_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158609_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158610_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158611_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158612_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158613_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158614_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158615_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158616_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158617_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158618_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158619_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158620_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158621_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158622_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158623_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158624_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158625_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158626_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158627_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158628_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158629_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158630_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158631_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158632_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158633_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158634_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158635_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158636_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158637_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158638_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158639_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158640_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158641_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158642_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158643_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158644_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158645_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158646_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158647_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158648_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158649_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158650_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158651_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158662_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158663_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158664_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158665_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158666_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158667_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158668_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158669_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158670_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "less  /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_2.output.annot.tsv.gz | grep 'codon_variant:stop_gained'| awk '{if ($97!~/rs/) print $0}' > IID_H158671_T01_01_WG01_2.output.annot.stop.tsv.gz\n"
     ]
    }
   ],
   "source": [
    "for annots in annots_list:\n",
    "    sample=annots.split('/')[9].split('_2')[0]\n",
    "    output=sample+'_2.output.annot.stop.tsv.gz'\n",
    "    print('less ',annots,'| grep \\'codon_variant:stop_gained\\'| awk \\'{if ($97!~/rs/) print $0}\\' >',output)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- List mpileup 2:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "58\n"
     ]
    }
   ],
   "source": [
    "annots_stop='/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/*_2.output.annot.stop.tsv.gz'\n",
    "annots_stop=!ls {annots_stop}\n",
    "print(len(annots_stop))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Retrieving 148 from experiments API endpoint...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158604_T01_01_WG01\t/work/isabl/data/analyses/29/38/252938/IID_H158604_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158604_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158605_T01_01_WG01\t/work/isabl/data/analyses/29/40/252940/IID_H158605_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158605_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158606_T01_01_WG01\t/work/isabl/data/analyses/29/42/252942/IID_H158606_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158606_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158607_T01_01_WG01\t/work/isabl/data/analyses/29/44/252944/IID_H158607_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158607_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158608_T01_01_WG01\t/work/isabl/data/analyses/29/46/252946/IID_H158608_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158608_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158609_T01_01_WG01\t/work/isabl/data/analyses/29/48/252948/IID_H158609_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158609_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158610_T01_01_WG01\t/work/isabl/data/analyses/29/50/252950/IID_H158610_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158610_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158611_T01_01_WG01\t/work/isabl/data/analyses/29/52/252952/IID_H158611_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158611_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158612_T01_01_WG01\t/work/isabl/data/analyses/29/54/252954/IID_H158612_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158612_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158613_T01_01_WG01\t/work/isabl/data/analyses/29/56/252956/IID_H158613_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158613_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158614_T01_01_WG01\t/work/isabl/data/analyses/29/58/252958/IID_H158614_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158614_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158615_T01_01_WG01\t/work/isabl/data/analyses/29/60/252960/IID_H158615_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158615_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158616_T01_01_WG01\t/work/isabl/data/analyses/29/62/252962/IID_H158616_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158616_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158617_T01_01_WG01\t/work/isabl/data/analyses/29/64/252964/IID_H158617_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158617_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158618_T01_01_WG01\t/work/isabl/data/analyses/29/66/252966/IID_H158618_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158618_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158619_T01_01_WG01\t/work/isabl/data/analyses/29/68/252968/IID_H158619_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158619_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158620_T01_01_WG01\t/work/isabl/data/analyses/29/70/252970/IID_H158620_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158620_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158621_T01_01_WG01\t/work/isabl/data/analyses/29/72/252972/IID_H158621_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158621_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158622_T01_01_WG01\t/work/isabl/data/analyses/29/74/252974/IID_H158622_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158623_T01_01_WG01\t/work/isabl/data/analyses/29/76/252976/IID_H158623_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158623_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158624_T01_01_WG01\t/work/isabl/data/analyses/29/78/252978/IID_H158624_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158624_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158625_T01_01_WG01\t/work/isabl/data/analyses/29/80/252980/IID_H158625_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158625_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158626_T01_01_WG01\t/work/isabl/data/analyses/29/82/252982/IID_H158626_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158626_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158627_T01_01_WG01\t/work/isabl/data/analyses/29/84/252984/IID_H158627_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158627_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158628_T01_01_WG01\t/work/isabl/data/analyses/29/86/252986/IID_H158628_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158628_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158629_T01_01_WG01\t/work/isabl/data/analyses/29/88/252988/IID_H158629_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158629_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158630_T01_01_WG01\t/work/isabl/data/analyses/29/90/252990/IID_H158630_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158630_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158631_T01_01_WG01\t/work/isabl/data/analyses/29/92/252992/IID_H158631_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158631_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158632_T01_01_WG01\t/work/isabl/data/analyses/29/94/252994/IID_H158632_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158633_T01_01_WG01\t/work/isabl/data/analyses/29/96/252996/IID_H158633_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158633_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158634_T01_01_WG01\t/work/isabl/data/analyses/98/98/259898/IID_H158634_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158634_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158635_T01_01_WG01\t/work/isabl/data/analyses/29/99/252999/IID_H158635_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158636_T01_01_WG01\t/work/isabl/data/analyses/34/25/253425/IID_H158636_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158636_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158637_T01_01_WG01\t/work/isabl/data/analyses/34/27/253427/IID_H158637_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158637_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158638_T01_01_WG01\t/work/isabl/data/analyses/34/29/253429/IID_H158638_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158638_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158639_T01_01_WG01\t/work/isabl/data/analyses/34/31/253431/IID_H158639_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158639_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158640_T01_01_WG01\t/work/isabl/data/analyses/34/33/253433/IID_H158640_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158640_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158641_T01_01_WG01\t/work/isabl/data/analyses/34/35/253435/IID_H158641_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158642_T01_01_WG01\t/work/isabl/data/analyses/34/37/253437/IID_H158642_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158642_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158643_T01_01_WG01\t/work/isabl/data/analyses/34/38/253438/IID_H158643_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158643_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158644_T01_01_WG01\t/work/isabl/data/analyses/34/39/253439/IID_H158644_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158645_T01_01_WG01\t/work/isabl/data/analyses/34/40/253440/IID_H158645_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158645_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158646_T01_01_WG01\t/work/isabl/data/analyses/34/41/253441/IID_H158646_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158646_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158647_T01_01_WG01\t/work/isabl/data/analyses/34/42/253442/IID_H158647_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158647_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158648_T01_01_WG01\t/work/isabl/data/analyses/34/43/253443/IID_H158648_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158648_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158649_T01_01_WG01\t/work/isabl/data/analyses/34/44/253444/IID_H158649_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158649_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158650_T01_01_WG01\t/work/isabl/data/analyses/36/50/253650/IID_H158650_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158650_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158651_T01_01_WG01\t/work/isabl/data/analyses/36/64/253664/IID_H158651_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158651_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158662_T01_01_WG01\t/work/isabl/data/analyses/01/36/260136/IID_H158662_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158662_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158663_T01_01_WG01\t/work/isabl/data/analyses/01/38/260138/IID_H158663_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158663_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158664_T01_01_WG01\t/work/isabl/data/analyses/01/40/260140/IID_H158664_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158664_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158665_T01_01_WG01\t/work/isabl/data/analyses/01/51/260151/IID_H158665_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158665_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158666_T01_01_WG01\t/work/isabl/data/analyses/01/53/260153/IID_H158666_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158666_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158667_T01_01_WG01\t/work/isabl/data/analyses/01/55/260155/IID_H158667_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158667_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158668_T01_01_WG01\t/work/isabl/data/analyses/01/57/260157/IID_H158668_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158668_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158669_T01_01_WG01\t/work/isabl/data/analyses/01/59/260159/IID_H158669_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158670_T01_01_WG01\t/work/isabl/data/analyses/01/61/260161/IID_H158670_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158670_T01_01_WG01_2.output.annot.stop.tsv.gz\n",
      "IID_H158671_T01_01_WG01\t/work/isabl/data/analyses/01/63/260163/IID_H158671_T01_01_WG01.bam\t/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158671_T01_01_WG01_2.output.annot.stop.tsv.gz\n"
     ]
    }
   ],
   "source": [
    "count=0\n",
    "for exper in ii.get_experiments(projects=321):\n",
    "    sample = exper.system_id\n",
    "    if ('_T' in sample) & ('WG' in sample):\n",
    "        bam = ii.Experiment(sample).bam_files.GRCh37.url\n",
    "        if bam:\n",
    "            print(sample,bam,annots_stop[count],sep='\\t')\n",
    "            count+=1"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# - Annotation: Stops"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "6\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158622_T01_01_WG01_2.output.annot.stop.bcftools.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158632_T01_01_WG01_2.output.annot.stop.bcftools.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158635_T01_01_WG01_2.output.annot.stop.bcftools.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158641_T01_01_WG01_2.output.annot.stop.bcftools.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158644_T01_01_WG01_2.output.annot.stop.bcftools.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/IID_H158669_T01_01_WG01_2.output.annot.stop.bcftools.tsv']"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "stop_list='/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/TSGs/*_2.output.annot.stop.bcftools.tsv'\n",
    "stop_list=!ls {stop_list}\n",
    "print(len(stop_list))\n",
    "stop_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "stop_total=pd.DataFrame()\n",
    "for n in stop_list:\n",
    "    sample=n.split('/')[9].split('_2')[0]\n",
    "    stop_ann=pd.read_csv(n,sep='\\t',comment='#')\n",
    "    stop_ann=stop_ann[['CHR','START','REF','ALT','QUAL','VAG_GENE','VAG_cDNA_CHANGE','VAG_PROTEIN_CHANGE','VEP_Consequence','VEP_IMPACT']]\n",
    "    stop_ann=stop_ann.drop_duplicates(subset=None, keep='first', inplace=False)\n",
    "    stop_ann['VAG_PROTEIN_CHANGE']=stop_ann['VAG_PROTEIN_CHANGE'].str.replace('p.', '', regex=False)\n",
    "    stop_ann['Sample']=sample\n",
    "    stop_total=stop_total.append(stop_ann)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>CHR</th>\n",
       "      <th>START</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>VAG_GENE</th>\n",
       "      <th>VAG_cDNA_CHANGE</th>\n",
       "      <th>VAG_PROTEIN_CHANGE</th>\n",
       "      <th>VEP_Consequence</th>\n",
       "      <th>VEP_IMPACT</th>\n",
       "      <th>Sample</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>3</td>\n",
       "      <td>47139465</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SETD2</td>\n",
       "      <td>c.5122C&gt;T</td>\n",
       "      <td>R1708*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158622_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>13</td>\n",
       "      <td>48953730</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>RB1</td>\n",
       "      <td>c.1333C&gt;T</td>\n",
       "      <td>R445*</td>\n",
       "      <td>stop_gained&amp;splice_region_variant</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>17</td>\n",
       "      <td>29556079</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NF1</td>\n",
       "      <td>c.2446C&gt;T</td>\n",
       "      <td>R816*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>17</td>\n",
       "      <td>29562981</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NF1</td>\n",
       "      <td>c.3916C&gt;T</td>\n",
       "      <td>R1306*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>X</td>\n",
       "      <td>44950066</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>KDM6A</td>\n",
       "      <td>c.3835C&gt;T</td>\n",
       "      <td>R1279*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>12</td>\n",
       "      <td>49434325</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>KMT2D</td>\n",
       "      <td>c.7228C&gt;T</td>\n",
       "      <td>R2410*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158635_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>20</td>\n",
       "      <td>31021250</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASXL1</td>\n",
       "      <td>c.1249C&gt;T</td>\n",
       "      <td>R417*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158641_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>3</td>\n",
       "      <td>47165518</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SETD2</td>\n",
       "      <td>c.608C&gt;A</td>\n",
       "      <td>S203*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158644_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>12</td>\n",
       "      <td>49433388</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>KMT2D</td>\n",
       "      <td>c.8059C&gt;T</td>\n",
       "      <td>R2687*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158669_T01_01_WG01</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  CHR     START REF ALT  QUAL VAG_GENE VAG_cDNA_CHANGE VAG_PROTEIN_CHANGE  \\\n",
       "0   3  47139465   G   A   NaN    SETD2       c.5122C>T             R1708*   \n",
       "0  13  48953730   C   T   NaN      RB1       c.1333C>T              R445*   \n",
       "1  17  29556079   C   T   NaN      NF1       c.2446C>T              R816*   \n",
       "2  17  29562981   C   T   NaN      NF1       c.3916C>T             R1306*   \n",
       "3   X  44950066   C   T   NaN    KDM6A       c.3835C>T             R1279*   \n",
       "0  12  49434325   G   A   NaN    KMT2D       c.7228C>T             R2410*   \n",
       "0  20  31021250   C   T   NaN    ASXL1       c.1249C>T              R417*   \n",
       "0   3  47165518   G   T   NaN    SETD2        c.608C>A              S203*   \n",
       "0  12  49433388   G   A   NaN    KMT2D       c.8059C>T             R2687*   \n",
       "\n",
       "                     VEP_Consequence VEP_IMPACT                   Sample  \n",
       "0                        stop_gained       HIGH  IID_H158622_T01_01_WG01  \n",
       "0  stop_gained&splice_region_variant       HIGH  IID_H158632_T01_01_WG01  \n",
       "1                        stop_gained       HIGH  IID_H158632_T01_01_WG01  \n",
       "2                        stop_gained       HIGH  IID_H158632_T01_01_WG01  \n",
       "3                        stop_gained       HIGH  IID_H158632_T01_01_WG01  \n",
       "0                        stop_gained       HIGH  IID_H158635_T01_01_WG01  \n",
       "0                        stop_gained       HIGH  IID_H158641_T01_01_WG01  \n",
       "0                        stop_gained       HIGH  IID_H158644_T01_01_WG01  \n",
       "0                        stop_gained       HIGH  IID_H158669_T01_01_WG01  "
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "stop_total.sort_values(by=['Sample'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [],
   "source": [
    "keys=[]\n",
    "for index,row in stop_total.iterrows():\n",
    "    key=str(row['CHR'])+'_'+str(row['START'])+'_'+row['Sample']\n",
    "    keys.append(key)\n",
    "stop_total['keys']=keys"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158622_T01_01_WG01\n",
      "IID_H158632_T01_01_WG01\n",
      "IID_H158635_T01_01_WG01\n",
      "IID_H158641_T01_01_WG01\n",
      "IID_H158644_T01_01_WG01\n",
      "IID_H158669_T01_01_WG01\n",
      "SVsupport_stops:\n",
      "9\n",
      "No SVsupport_stops:\n",
      "0\n"
     ]
    }
   ],
   "source": [
    "samples_noDetect=list(stop_total['Sample'].unique())\n",
    "SV_support_stops=list()\n",
    "SV_no_support_stops=list()\n",
    "\n",
    "for n in samples_noDetect:\n",
    "    #isabl\n",
    "    list_annotSVs=!isabl get-outdirs -fi application__name ANNOT_SNVS -fi targets__system_id {n}\n",
    "    path=list_annotSVs[1]+'/drivers/*.snvs.output.annot.coding.tsv.gz'\n",
    "    print(n)\n",
    "\n",
    "    svs_pass=!ls {path}\n",
    "    \n",
    "    #iterate df\n",
    "    df=stop_total[stop_total['Sample']==n]\n",
    "    df=df.drop_duplicates(subset=['keys'])\n",
    "    #print(df.shape)\n",
    "    #print('- ',n,':','\\n',df[['chr_seg','start_seg','end_seg','CNV','Sample','Fraction']],'\\n')\n",
    "    chrom_list=list()\n",
    "    \n",
    "    #SV_support=list()\n",
    "    for index,row in df.iterrows():\n",
    "        gene=row['VAG_GENE']\n",
    "        alteration=row['VAG_PROTEIN_CHANGE']\n",
    "\n",
    "        \n",
    "        #Awk command + print\n",
    "        awk_output=!zless {svs_pass[0]} | grep {gene} | grep {alteration}\n",
    "        awk_output=list(set(awk_output))\n",
    "        #print(awk_output)\n",
    "        #if (not chrom in chrom_list):\n",
    "            #for n2 in awk_output:\n",
    "                #print(n2)\n",
    "            #print('\\n')\n",
    "            #chrom_list.append(chrom)\n",
    "        \n",
    "        #SV support\n",
    "        \n",
    "        if(len(awk_output)==0):\n",
    "            SV_no_support_stops.append(row['keys'])\n",
    "        else:            \n",
    "            SV_support_stops.append(row['keys'])\n",
    "\n",
    "print('SVsupport_stops:')\n",
    "SV_support_stops=list(set(SV_support_stops))\n",
    "print(len(SV_support_stops))\n",
    "#for n3 in sorted(SV_support):\n",
    "    #print(n3)\n",
    "print('No SVsupport_stops:')\n",
    "SV_no_support_stops=list(set(SV_no_support_stops))\n",
    "print(len(SV_no_support_stops))\n",
    "#for n3 in sorted(SV_no_support):\n",
    "    #print(n3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [],
   "source": [
    "stop_total_matched=stop_total.copy()\n",
    "\n",
    "match_support=[]\n",
    "for index,row in stop_total_matched.iterrows():\n",
    "    if(row['keys'] in SV_support_stops):\n",
    "        match_support.append('Detected')\n",
    "    else:\n",
    "        match_support.append('Non Detected')\n",
    "stop_total_matched['Matched']=match_support"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>CHR</th>\n",
       "      <th>START</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>VAG_GENE</th>\n",
       "      <th>VAG_cDNA_CHANGE</th>\n",
       "      <th>VAG_PROTEIN_CHANGE</th>\n",
       "      <th>VEP_Consequence</th>\n",
       "      <th>VEP_IMPACT</th>\n",
       "      <th>Sample</th>\n",
       "      <th>keys</th>\n",
       "      <th>Matched</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>3</td>\n",
       "      <td>47139465</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SETD2</td>\n",
       "      <td>c.5122C&gt;T</td>\n",
       "      <td>R1708*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158622_T01_01_WG01</td>\n",
       "      <td>3_47139465_IID_H158622_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0</td>\n",
       "      <td>13</td>\n",
       "      <td>48953730</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>RB1</td>\n",
       "      <td>c.1333C&gt;T</td>\n",
       "      <td>R445*</td>\n",
       "      <td>stop_gained&amp;splice_region_variant</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>13_48953730_IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>17</td>\n",
       "      <td>29556079</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NF1</td>\n",
       "      <td>c.2446C&gt;T</td>\n",
       "      <td>R816*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>17_29556079_IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2</td>\n",
       "      <td>17</td>\n",
       "      <td>29562981</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NF1</td>\n",
       "      <td>c.3916C&gt;T</td>\n",
       "      <td>R1306*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>17_29562981_IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>3</td>\n",
       "      <td>X</td>\n",
       "      <td>44950066</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>KDM6A</td>\n",
       "      <td>c.3835C&gt;T</td>\n",
       "      <td>R1279*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>X_44950066_IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>0</td>\n",
       "      <td>12</td>\n",
       "      <td>49434325</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>KMT2D</td>\n",
       "      <td>c.7228C&gt;T</td>\n",
       "      <td>R2410*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158635_T01_01_WG01</td>\n",
       "      <td>12_49434325_IID_H158635_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>0</td>\n",
       "      <td>20</td>\n",
       "      <td>31021250</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASXL1</td>\n",
       "      <td>c.1249C&gt;T</td>\n",
       "      <td>R417*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158641_T01_01_WG01</td>\n",
       "      <td>20_31021250_IID_H158641_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>0</td>\n",
       "      <td>3</td>\n",
       "      <td>47165518</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SETD2</td>\n",
       "      <td>c.608C&gt;A</td>\n",
       "      <td>S203*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158644_T01_01_WG01</td>\n",
       "      <td>3_47165518_IID_H158644_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>0</td>\n",
       "      <td>12</td>\n",
       "      <td>49433388</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>KMT2D</td>\n",
       "      <td>c.8059C&gt;T</td>\n",
       "      <td>R2687*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158669_T01_01_WG01</td>\n",
       "      <td>12_49433388_IID_H158669_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   index CHR     START REF ALT  QUAL VAG_GENE VAG_cDNA_CHANGE  \\\n",
       "0      0   3  47139465   G   A   NaN    SETD2       c.5122C>T   \n",
       "1      0  13  48953730   C   T   NaN      RB1       c.1333C>T   \n",
       "2      1  17  29556079   C   T   NaN      NF1       c.2446C>T   \n",
       "3      2  17  29562981   C   T   NaN      NF1       c.3916C>T   \n",
       "4      3   X  44950066   C   T   NaN    KDM6A       c.3835C>T   \n",
       "5      0  12  49434325   G   A   NaN    KMT2D       c.7228C>T   \n",
       "6      0  20  31021250   C   T   NaN    ASXL1       c.1249C>T   \n",
       "7      0   3  47165518   G   T   NaN    SETD2        c.608C>A   \n",
       "8      0  12  49433388   G   A   NaN    KMT2D       c.8059C>T   \n",
       "\n",
       "  VAG_PROTEIN_CHANGE                    VEP_Consequence VEP_IMPACT  \\\n",
       "0             R1708*                        stop_gained       HIGH   \n",
       "1              R445*  stop_gained&splice_region_variant       HIGH   \n",
       "2              R816*                        stop_gained       HIGH   \n",
       "3             R1306*                        stop_gained       HIGH   \n",
       "4             R1279*                        stop_gained       HIGH   \n",
       "5             R2410*                        stop_gained       HIGH   \n",
       "6              R417*                        stop_gained       HIGH   \n",
       "7              S203*                        stop_gained       HIGH   \n",
       "8             R2687*                        stop_gained       HIGH   \n",
       "\n",
       "                    Sample                                 keys   Matched  \n",
       "0  IID_H158622_T01_01_WG01   3_47139465_IID_H158622_T01_01_WG01  Detected  \n",
       "1  IID_H158632_T01_01_WG01  13_48953730_IID_H158632_T01_01_WG01  Detected  \n",
       "2  IID_H158632_T01_01_WG01  17_29556079_IID_H158632_T01_01_WG01  Detected  \n",
       "3  IID_H158632_T01_01_WG01  17_29562981_IID_H158632_T01_01_WG01  Detected  \n",
       "4  IID_H158632_T01_01_WG01   X_44950066_IID_H158632_T01_01_WG01  Detected  \n",
       "5  IID_H158635_T01_01_WG01  12_49434325_IID_H158635_T01_01_WG01  Detected  \n",
       "6  IID_H158641_T01_01_WG01  20_31021250_IID_H158641_T01_01_WG01  Detected  \n",
       "7  IID_H158644_T01_01_WG01   3_47165518_IID_H158644_T01_01_WG01  Detected  \n",
       "8  IID_H158669_T01_01_WG01  12_49433388_IID_H158669_T01_01_WG01  Detected  "
      ]
     },
     "execution_count": 62,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "stop_total_matched.reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['CHR', 'START', 'REF', 'ALT', 'QUAL', 'VAG_GENE', 'VAG_cDNA_CHANGE',\n",
       "       'VAG_PROTEIN_CHANGE', 'VEP_Consequence', 'VEP_IMPACT', 'Sample', 'keys',\n",
       "       'Matched'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 69,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "stop_total_matched.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>CHR</th>\n",
       "      <th>START</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>VAG_GENE</th>\n",
       "      <th>VAG_cDNA_CHANGE</th>\n",
       "      <th>VAG_PROTEIN_CHANGE</th>\n",
       "      <th>VEP_Consequence</th>\n",
       "      <th>VEP_IMPACT</th>\n",
       "      <th>Sample</th>\n",
       "      <th>Matched</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>3</td>\n",
       "      <td>47139465</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>SETD2</td>\n",
       "      <td>c.5122C&gt;T</td>\n",
       "      <td>R1708*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158622_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0</td>\n",
       "      <td>13</td>\n",
       "      <td>48953730</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>RB1</td>\n",
       "      <td>c.1333C&gt;T</td>\n",
       "      <td>R445*</td>\n",
       "      <td>stop_gained&amp;splice_region_variant</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>17</td>\n",
       "      <td>29556079</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NF1</td>\n",
       "      <td>c.2446C&gt;T</td>\n",
       "      <td>R816*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2</td>\n",
       "      <td>17</td>\n",
       "      <td>29562981</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>NF1</td>\n",
       "      <td>c.3916C&gt;T</td>\n",
       "      <td>R1306*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>3</td>\n",
       "      <td>X</td>\n",
       "      <td>44950066</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>KDM6A</td>\n",
       "      <td>c.3835C&gt;T</td>\n",
       "      <td>R1279*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>0</td>\n",
       "      <td>12</td>\n",
       "      <td>49434325</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>KMT2D</td>\n",
       "      <td>c.7228C&gt;T</td>\n",
       "      <td>R2410*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158635_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>0</td>\n",
       "      <td>20</td>\n",
       "      <td>31021250</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>ASXL1</td>\n",
       "      <td>c.1249C&gt;T</td>\n",
       "      <td>R417*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158641_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>0</td>\n",
       "      <td>3</td>\n",
       "      <td>47165518</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>SETD2</td>\n",
       "      <td>c.608C&gt;A</td>\n",
       "      <td>S203*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158644_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>0</td>\n",
       "      <td>12</td>\n",
       "      <td>49433388</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>KMT2D</td>\n",
       "      <td>c.8059C&gt;T</td>\n",
       "      <td>R2687*</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>IID_H158669_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   index CHR     START REF ALT VAG_GENE VAG_cDNA_CHANGE VAG_PROTEIN_CHANGE  \\\n",
       "0      0   3  47139465   G   A    SETD2       c.5122C>T             R1708*   \n",
       "1      0  13  48953730   C   T      RB1       c.1333C>T              R445*   \n",
       "2      1  17  29556079   C   T      NF1       c.2446C>T              R816*   \n",
       "3      2  17  29562981   C   T      NF1       c.3916C>T             R1306*   \n",
       "4      3   X  44950066   C   T    KDM6A       c.3835C>T             R1279*   \n",
       "5      0  12  49434325   G   A    KMT2D       c.7228C>T             R2410*   \n",
       "6      0  20  31021250   C   T    ASXL1       c.1249C>T              R417*   \n",
       "7      0   3  47165518   G   T    SETD2        c.608C>A              S203*   \n",
       "8      0  12  49433388   G   A    KMT2D       c.8059C>T             R2687*   \n",
       "\n",
       "                     VEP_Consequence VEP_IMPACT                   Sample  \\\n",
       "0                        stop_gained       HIGH  IID_H158622_T01_01_WG01   \n",
       "1  stop_gained&splice_region_variant       HIGH  IID_H158632_T01_01_WG01   \n",
       "2                        stop_gained       HIGH  IID_H158632_T01_01_WG01   \n",
       "3                        stop_gained       HIGH  IID_H158632_T01_01_WG01   \n",
       "4                        stop_gained       HIGH  IID_H158632_T01_01_WG01   \n",
       "5                        stop_gained       HIGH  IID_H158635_T01_01_WG01   \n",
       "6                        stop_gained       HIGH  IID_H158641_T01_01_WG01   \n",
       "7                        stop_gained       HIGH  IID_H158644_T01_01_WG01   \n",
       "8                        stop_gained       HIGH  IID_H158669_T01_01_WG01   \n",
       "\n",
       "    Matched  \n",
       "0  Detected  \n",
       "1  Detected  \n",
       "2  Detected  \n",
       "3  Detected  \n",
       "4  Detected  \n",
       "5  Detected  \n",
       "6  Detected  \n",
       "7  Detected  \n",
       "8  Detected  "
      ]
     },
     "execution_count": 71,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "stop_total_matched[['CHR', 'START', 'REF', 'ALT','VAG_GENE', 'VAG_cDNA_CHANGE','VAG_PROTEIN_CHANGE', 'VEP_Consequence', 'VEP_IMPACT', 'Sample','Matched']].reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# - Annotation: OncoKBs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "58\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158604_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158605_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158606_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158607_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158608_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158609_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158610_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158611_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158612_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158613_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158614_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158615_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158616_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158617_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158618_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158619_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158620_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158621_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158622_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158623_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158624_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158625_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158626_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158627_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158628_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158629_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158630_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158631_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158632_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158633_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158634_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158635_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158636_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158637_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158638_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158639_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158640_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158641_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158642_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158643_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158644_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158645_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158646_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158647_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158648_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158649_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158650_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158651_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158662_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158663_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158664_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158665_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158666_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158667_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158668_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158669_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158670_T01_01_WG01_2.vcf.gz',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158671_T01_01_WG01_2.vcf.gz']"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vcfs_list='/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/*_2.vcf.gz'\n",
    "vcfs_list=!ls {vcfs_list}\n",
    "print(len(vcfs_list))\n",
    "vcfs_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158604_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158604_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158605_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158605_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158606_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158606_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158607_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158607_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158608_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158608_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158609_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158609_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158610_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158610_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158611_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158611_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158612_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158612_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158613_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158613_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158614_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158614_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158615_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158615_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158616_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158616_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158617_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158617_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158618_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158618_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158619_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158619_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158620_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158620_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158621_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158621_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158622_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158622_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158623_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158623_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158624_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158624_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158625_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158625_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158626_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158626_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158627_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158627_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158628_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158628_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158629_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158629_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158630_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158630_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158631_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158631_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158632_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158632_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158633_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158633_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158634_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158634_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158635_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158635_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158636_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158636_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158637_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158637_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158638_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158638_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158639_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158639_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158640_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158640_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158641_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158641_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158642_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158642_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158643_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158643_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158644_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158644_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158645_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158645_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158646_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158646_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158647_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158647_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158648_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158648_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158649_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158649_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158650_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158650_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158651_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158651_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158662_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158662_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158663_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158663_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158664_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158664_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158665_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158665_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158666_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158666_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158667_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158667_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158668_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158668_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158669_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158669_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158670_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158670_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n",
      "click_annotvcf annotvcf     --input_vcf /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158671_T01_01_WG01_2.vcf.gz     --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB     --output_prefix IID_H158671_T01_01_WG01_2     --assembly GRCH37D5     --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta     --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz     --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/     --ensembl-version 91     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female     --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP     --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 \n",
      "\n"
     ]
    }
   ],
   "source": [
    "for vcf in vcfs_list:\n",
    "    sample=vcf.split('/')[9].split('.vcf')[0]\n",
    "    print('click_annotvcf annotvcf \\\n",
    "    --input_vcf ',vcf,' \\\n",
    "    --outdir /work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB \\\n",
    "    --output_prefix ',sample,' \\\n",
    "    --assembly GRCH37D5 \\\n",
    "    --reference /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/genome/gr37.fasta \\\n",
    "    --vagrent /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vagrent/Homo_sapiens_KnC.GRCh37.75.vagrent.cache.gz \\\n",
    "    --vep-dir /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/vep/cache/ \\\n",
    "    --ensembl-version 91 \\\n",
    "    --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_genomes/gnomad.genomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_genome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female \\\n",
    "    --custom /ifs/work/leukgen/home/leukbot/tests/vep/gnomad_exomes/gnomad.exomes.r2.0.1.sites.noVEP.vcf.gz gnomAD_exome AC_AFR,AC_AMR,AC_ASJ,AC_EAS,AC_FIN,AC_NFE,AC_OTH,AC_Male,AC_Female,AN_AFR,AN_AMR,AN_ASJ,AN_EAS,AN_FIN,AN_NFE,AN_OTH,AN_Male,AN_Female,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_Male,AF_Female,Hom_HomR,Hom_AMR,Hom_ASJ,Hom_EAS,Hom_FIN,Hom_NFE,Hom_OTH,Hom_Male,Hom_Female \\\n",
    "    --custom /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81/CosmicMergedVariants.vcf.gz COSMIC GENE,STRAND,CDS,AA,CNT,SNP \\\n",
    "    --cosmic /ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/cosmic/81 ','\\n',sep='')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- Filtering Annotation:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Index</th>\n",
       "      <th>Isoform</th>\n",
       "      <th>RefSeq</th>\n",
       "      <th>Entrez.Gene.ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Alteration</th>\n",
       "      <th>Protein.Change</th>\n",
       "      <th>Oncogenicity</th>\n",
       "      <th>Mutation.Effect</th>\n",
       "      <th>PMIDs.for.Mutation.Effect</th>\n",
       "      <th>aaPos</th>\n",
       "      <th>chr_gnm</th>\n",
       "      <th>start_gnm</th>\n",
       "      <th>end_gnm</th>\n",
       "      <th>Abstracts.for.Mutation.Effect</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>2111</th>\n",
       "      <td>2487</td>\n",
       "      <td>ENST00000234420</td>\n",
       "      <td>NM_000179.2</td>\n",
       "      <td>2956</td>\n",
       "      <td>MSH6</td>\n",
       "      <td>G39E</td>\n",
       "      <td>G39E</td>\n",
       "      <td>Inconclusive</td>\n",
       "      <td>Inconclusive</td>\n",
       "      <td>24362816</td>\n",
       "      <td>39</td>\n",
       "      <td>2</td>\n",
       "      <td>48010487</td>\n",
       "      <td>48010489</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>592</th>\n",
       "      <td>685</td>\n",
       "      <td>ENST00000357654</td>\n",
       "      <td>NM_007294.3</td>\n",
       "      <td>672</td>\n",
       "      <td>BRCA1</td>\n",
       "      <td>V191I</td>\n",
       "      <td>V191I</td>\n",
       "      <td>Likely Neutral</td>\n",
       "      <td>Likely Neutral</td>\n",
       "      <td>23867111</td>\n",
       "      <td>191</td>\n",
       "      <td>17</td>\n",
       "      <td>41249281</td>\n",
       "      <td>41249283</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2526</th>\n",
       "      <td>3002</td>\n",
       "      <td>ENST00000265849</td>\n",
       "      <td>NM_000535.5</td>\n",
       "      <td>5395</td>\n",
       "      <td>PMS2</td>\n",
       "      <td>T485K</td>\n",
       "      <td>T485K</td>\n",
       "      <td>Likely Neutral</td>\n",
       "      <td>Likely Neutral</td>\n",
       "      <td>15256438, 24362816, 24027009</td>\n",
       "      <td>485</td>\n",
       "      <td>7</td>\n",
       "      <td>6026941</td>\n",
       "      <td>6026943</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2521</th>\n",
       "      <td>2997</td>\n",
       "      <td>ENST00000265849</td>\n",
       "      <td>NM_000535.5</td>\n",
       "      <td>5395</td>\n",
       "      <td>PMS2</td>\n",
       "      <td>P470S</td>\n",
       "      <td>P470S</td>\n",
       "      <td>Likely Neutral</td>\n",
       "      <td>Likely Neutral</td>\n",
       "      <td>24362816</td>\n",
       "      <td>470</td>\n",
       "      <td>7</td>\n",
       "      <td>6026986</td>\n",
       "      <td>6026988</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>953</th>\n",
       "      <td>1110</td>\n",
       "      <td>ENST00000302823</td>\n",
       "      <td>NM_005214.4</td>\n",
       "      <td>1493</td>\n",
       "      <td>CTLA4</td>\n",
       "      <td>T17A</td>\n",
       "      <td>T17A</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Likely Loss-of-function</td>\n",
       "      <td>15018649, 11976786, 20064370, 10082437, 898924...</td>\n",
       "      <td>17</td>\n",
       "      <td>2</td>\n",
       "      <td>204732714</td>\n",
       "      <td>204732716</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>238</td>\n",
       "      <td>ENST00000312783</td>\n",
       "      <td>NM_003600.2</td>\n",
       "      <td>6790</td>\n",
       "      <td>AURKA</td>\n",
       "      <td>F31I</td>\n",
       "      <td>F31I</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Likely Switch-of-function</td>\n",
       "      <td>12881723, 15867347, 12208850</td>\n",
       "      <td>31</td>\n",
       "      <td>20</td>\n",
       "      <td>54961539</td>\n",
       "      <td>54961541</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1739</th>\n",
       "      <td>2058</td>\n",
       "      <td>ENST00000263923</td>\n",
       "      <td>NM_002253.2</td>\n",
       "      <td>3791</td>\n",
       "      <td>KDR</td>\n",
       "      <td>Q472H</td>\n",
       "      <td>Q472H</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Likely Gain-of-function</td>\n",
       "      <td>24793135, 24599305, 21712447</td>\n",
       "      <td>472</td>\n",
       "      <td>4</td>\n",
       "      <td>55972974</td>\n",
       "      <td>55972976</td>\n",
       "      <td>Mockus et al. Abstract 73, AACR 2015 http://ca...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     Index          Isoform       RefSeq Entrez.Gene.ID   Gene Alteration  \\\n",
       "2111  2487  ENST00000234420  NM_000179.2           2956   MSH6       G39E   \n",
       "592    685  ENST00000357654  NM_007294.3            672  BRCA1      V191I   \n",
       "2526  3002  ENST00000265849  NM_000535.5           5395   PMS2      T485K   \n",
       "2521  2997  ENST00000265849  NM_000535.5           5395   PMS2      P470S   \n",
       "953   1110  ENST00000302823  NM_005214.4           1493  CTLA4       T17A   \n",
       "194    238  ENST00000312783  NM_003600.2           6790  AURKA       F31I   \n",
       "1739  2058  ENST00000263923  NM_002253.2           3791    KDR      Q472H   \n",
       "\n",
       "     Protein.Change      Oncogenicity            Mutation.Effect  \\\n",
       "2111           G39E      Inconclusive               Inconclusive   \n",
       "592           V191I    Likely Neutral             Likely Neutral   \n",
       "2526          T485K    Likely Neutral             Likely Neutral   \n",
       "2521          P470S    Likely Neutral             Likely Neutral   \n",
       "953            T17A  Likely Oncogenic    Likely Loss-of-function   \n",
       "194            F31I  Likely Oncogenic  Likely Switch-of-function   \n",
       "1739          Q472H  Likely Oncogenic    Likely Gain-of-function   \n",
       "\n",
       "                              PMIDs.for.Mutation.Effect aaPos chr_gnm  \\\n",
       "2111                                           24362816    39       2   \n",
       "592                                            23867111   191      17   \n",
       "2526                       15256438, 24362816, 24027009   485       7   \n",
       "2521                                           24362816   470       7   \n",
       "953   15018649, 11976786, 20064370, 10082437, 898924...    17       2   \n",
       "194                        12881723, 15867347, 12208850    31      20   \n",
       "1739                       24793135, 24599305, 21712447   472       4   \n",
       "\n",
       "      start_gnm    end_gnm                      Abstracts.for.Mutation.Effect  \n",
       "2111   48010487   48010489                                                NaN  \n",
       "592    41249281   41249283                                                NaN  \n",
       "2526    6026941    6026943                                                NaN  \n",
       "2521    6026986    6026988                                                NaN  \n",
       "953   204732714  204732716                                                NaN  \n",
       "194    54961539   54961541                                                NaN  \n",
       "1739   55972974   55972976  Mockus et al. Abstract 73, AACR 2015 http://ca...  "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "oncokb_ann=pd.read_csv('/work/isabl/home/gutierj2/click_annotvcf/oncoKB/I-H-136050-T1-1-D1-1.output.annot.tsv',sep='\\t')\n",
    "oncokb_ann=oncokb_ann[['CHR','START','REF','ALT','QUAL','VAG_GENE','VAG_cDNA_CHANGE','VAG_PROTEIN_CHANGE','VEP_Consequence','VEP_IMPACT']]\n",
    "oncokb_ann=oncokb_ann.drop_duplicates(subset=None, keep='first', inplace=False)\n",
    "oncokb_ann['VAG_PROTEIN_CHANGE']=oncokb_ann['VAG_PROTEIN_CHANGE'].str.replace('p.', '', regex=False)\n",
    "oncokb=pd.read_csv('/home/gutierj2/architecture/SNVs/oncoKB/oncoKB_allAnnotatedVariants_Aug_2018_aaPos_aaGnm_2bed.txt',sep='\\t')\n",
    "oncokb_info=pd.DataFrame(columns=['Index','Isoform','RefSeq','Entrez.Gene.ID','Gene','Alteration','Protein.Change','Oncogenicity','Mutation.Effect','PMIDs.for.Mutation.Effect','aaPos','chr_gnm','start_gnm','end_gnm'])\n",
    "for index,row in oncokb_ann.iterrows():\n",
    "    for index2,row2 in oncokb.iterrows():\n",
    "        if ( (str(row['VAG_GENE'])==str(row2['Gene'])) & (str(row['VAG_PROTEIN_CHANGE'])==str(row2['Alteration'])) ):\n",
    "            #print(row2)\n",
    "            oncokb_info=oncokb_info.append(row2)\n",
    "oncokb_info.sort_values(by=['Oncogenicity'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "58\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158604_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158605_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158606_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158607_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158608_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158609_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158610_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158611_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158612_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158613_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158614_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158615_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158616_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158617_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158618_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158619_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158620_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158621_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158622_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158623_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158624_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158625_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158626_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158627_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158628_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158629_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158630_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158631_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158632_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158633_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158634_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158635_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158636_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158637_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158638_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158639_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158640_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158641_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158642_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158643_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158644_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158645_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158646_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158647_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158648_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158649_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158650_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158651_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158662_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158663_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158664_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158665_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158666_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158667_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158668_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158669_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158670_T01_01_WG01_2.output.annot.tsv',\n",
       " '/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/IID_H158671_T01_01_WG01_2.output.annot.tsv']"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "oncos_list='/work/isabl/home/gutierj2/uk_all/notebooks/SNVs/oncoKB/*.output.annot.tsv'\n",
    "oncos_list=!ls {oncos_list}\n",
    "print(len(oncos_list))\n",
    "oncos_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158604_T01_01_WG01\n",
      "IID_H158605_T01_01_WG01\n",
      "IID_H158606_T01_01_WG01\n",
      "IID_H158607_T01_01_WG01\n",
      "IID_H158608_T01_01_WG01\n",
      "IID_H158609_T01_01_WG01\n",
      "IID_H158610_T01_01_WG01\n",
      "IID_H158611_T01_01_WG01\n",
      "IID_H158612_T01_01_WG01\n",
      "IID_H158613_T01_01_WG01\n",
      "IID_H158614_T01_01_WG01\n",
      "IID_H158615_T01_01_WG01\n",
      "IID_H158616_T01_01_WG01\n",
      "IID_H158617_T01_01_WG01\n",
      "IID_H158618_T01_01_WG01\n",
      "IID_H158619_T01_01_WG01\n",
      "IID_H158620_T01_01_WG01\n",
      "IID_H158621_T01_01_WG01\n",
      "IID_H158622_T01_01_WG01\n",
      "IID_H158623_T01_01_WG01\n",
      "IID_H158624_T01_01_WG01\n",
      "IID_H158625_T01_01_WG01\n",
      "IID_H158626_T01_01_WG01\n",
      "IID_H158627_T01_01_WG01\n",
      "IID_H158628_T01_01_WG01\n",
      "IID_H158629_T01_01_WG01\n",
      "IID_H158630_T01_01_WG01\n",
      "IID_H158631_T01_01_WG01\n",
      "IID_H158632_T01_01_WG01\n",
      "IID_H158633_T01_01_WG01\n",
      "IID_H158634_T01_01_WG01\n",
      "IID_H158635_T01_01_WG01\n",
      "IID_H158636_T01_01_WG01\n",
      "IID_H158637_T01_01_WG01\n",
      "IID_H158638_T01_01_WG01\n",
      "IID_H158639_T01_01_WG01\n",
      "IID_H158640_T01_01_WG01\n",
      "IID_H158641_T01_01_WG01\n",
      "IID_H158642_T01_01_WG01\n",
      "IID_H158643_T01_01_WG01\n",
      "IID_H158644_T01_01_WG01\n",
      "IID_H158645_T01_01_WG01\n",
      "IID_H158646_T01_01_WG01\n",
      "IID_H158647_T01_01_WG01\n",
      "IID_H158648_T01_01_WG01\n",
      "IID_H158649_T01_01_WG01\n",
      "IID_H158650_T01_01_WG01\n",
      "IID_H158651_T01_01_WG01\n",
      "IID_H158662_T01_01_WG01\n",
      "IID_H158663_T01_01_WG01\n",
      "IID_H158664_T01_01_WG01\n",
      "IID_H158665_T01_01_WG01\n",
      "IID_H158666_T01_01_WG01\n",
      "IID_H158667_T01_01_WG01\n",
      "IID_H158668_T01_01_WG01\n",
      "IID_H158669_T01_01_WG01\n",
      "IID_H158670_T01_01_WG01\n",
      "IID_H158671_T01_01_WG01\n"
     ]
    }
   ],
   "source": [
    "oncokb=pd.read_csv('/home/gutierj2/architecture/SNVs/oncoKB/oncoKB_allAnnotatedVariants_Aug_2018_aaPos_aaGnm_2bed.txt',sep='\\t')\n",
    "oncokb_info=pd.DataFrame(columns=['Index','Isoform','RefSeq','Entrez.Gene.ID','Gene','Alteration','Protein.Change','Oncogenicity','Mutation.Effect','PMIDs.for.Mutation.Effect','aaPos','chr_gnm','start_gnm','end_gnm'])\n",
    "\n",
    "for n in oncos_list:\n",
    "    sample=n.split('/')[9].split('_2')[0]\n",
    "    oncokb_ann=pd.read_csv(n,sep='\\t',comment='#')\n",
    "    oncokb_ann=oncokb_ann[['CHR','START','REF','ALT','QUAL','VAG_GENE','VAG_cDNA_CHANGE','VAG_PROTEIN_CHANGE','VEP_Consequence','VEP_IMPACT','VEP_gnomAD_genome']]\n",
    "    oncokb_ann=oncokb_ann[~oncokb_ann[\"VEP_gnomAD_genome\"].str.contains('rs',na=False)]\n",
    "    oncokb_ann=oncokb_ann.drop_duplicates(subset=None, keep='first', inplace=False)\n",
    "    oncokb_ann['VAG_PROTEIN_CHANGE']=oncokb_ann['VAG_PROTEIN_CHANGE'].str.replace('p.', '', regex=False)\n",
    "    print(sample)\n",
    "    for index,row in oncokb_ann.iterrows():\n",
    "        for index2,row2 in oncokb.iterrows():\n",
    "            if ( (str(row['VAG_GENE'])==str(row2['Gene'])) & (str(row['VAG_PROTEIN_CHANGE'])==str(row2['Alteration'])) ):\n",
    "                row2['Sample']=sample\n",
    "                #print(row2)\n",
    "                oncokb_info=oncokb_info.append(row2)\n",
    "#oncokb_info.sort_values(by=['Oncogenicity'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>Index</th>\n",
       "      <th>Isoform</th>\n",
       "      <th>RefSeq</th>\n",
       "      <th>Entrez.Gene.ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Alteration</th>\n",
       "      <th>Protein.Change</th>\n",
       "      <th>Oncogenicity</th>\n",
       "      <th>Mutation.Effect</th>\n",
       "      <th>PMIDs.for.Mutation.Effect</th>\n",
       "      <th>aaPos</th>\n",
       "      <th>chr_gnm</th>\n",
       "      <th>start_gnm</th>\n",
       "      <th>end_gnm</th>\n",
       "      <th>Abstracts.for.Mutation.Effect</th>\n",
       "      <th>Sample</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2281</td>\n",
       "      <td>2690</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>26037647, 21993244, 24651010, 22722830</td>\n",
       "      <td>13</td>\n",
       "      <td>1</td>\n",
       "      <td>115258743</td>\n",
       "      <td>115258745</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158605_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1684</td>\n",
       "      <td>1987</td>\n",
       "      <td>ENST00000342505</td>\n",
       "      <td>NM_002227.2</td>\n",
       "      <td>3716</td>\n",
       "      <td>JAK1</td>\n",
       "      <td>S703I</td>\n",
       "      <td>S703I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>24735922, 24825865, 23788652</td>\n",
       "      <td>703</td>\n",
       "      <td>1</td>\n",
       "      <td>65311202</td>\n",
       "      <td>65311204</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1724</td>\n",
       "      <td>2036</td>\n",
       "      <td>ENST00000458235</td>\n",
       "      <td>NM_000215.3</td>\n",
       "      <td>3718</td>\n",
       "      <td>JAK3</td>\n",
       "      <td>M511I</td>\n",
       "      <td>M511I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25193870</td>\n",
       "      <td>511</td>\n",
       "      <td>19</td>\n",
       "      <td>17949108</td>\n",
       "      <td>17949110</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>2726</td>\n",
       "      <td>3211</td>\n",
       "      <td>ENST00000351677</td>\n",
       "      <td>NM_002834.3</td>\n",
       "      <td>5781</td>\n",
       "      <td>PTPN11</td>\n",
       "      <td>T73I</td>\n",
       "      <td>T73I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>15834506</td>\n",
       "      <td>73</td>\n",
       "      <td>12</td>\n",
       "      <td>112888201</td>\n",
       "      <td>112888203</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158610_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1899</td>\n",
       "      <td>2236</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G12V</td>\n",
       "      <td>G12V</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>20516123, 20949621, 25359494, 20570890, 129572...</td>\n",
       "      <td>12</td>\n",
       "      <td>12</td>\n",
       "      <td>25398283</td>\n",
       "      <td>25398285</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158614_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>1598</td>\n",
       "      <td>1887</td>\n",
       "      <td>ENST00000378609</td>\n",
       "      <td>NM_001282539.1</td>\n",
       "      <td>2782</td>\n",
       "      <td>GNB1</td>\n",
       "      <td>K89E</td>\n",
       "      <td>K89E</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25485910</td>\n",
       "      <td>89</td>\n",
       "      <td>1</td>\n",
       "      <td>1737914</td>\n",
       "      <td>1737916</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158619_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158621_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>1900</td>\n",
       "      <td>2237</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13C</td>\n",
       "      <td>G13C</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25705018, 21079152</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158627_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>2287</td>\n",
       "      <td>2696</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>Q61R</td>\n",
       "      <td>Q61R</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25277205, 21993244, 20194776, 23515407, 246510...</td>\n",
       "      <td>61</td>\n",
       "      <td>1</td>\n",
       "      <td>115256528</td>\n",
       "      <td>115256530</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158631_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158634_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>2280</td>\n",
       "      <td>2689</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12V</td>\n",
       "      <td>G12V</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>18952898, 21993244, 21478863, 24651010</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158637_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>1901</td>\n",
       "      <td>2238</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>26037647, 22722830, 20147967</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158637_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>1521</td>\n",
       "      <td>1799</td>\n",
       "      <td>ENST00000241453</td>\n",
       "      <td>NM_004119.2</td>\n",
       "      <td>2322</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835H</td>\n",
       "      <td>D835H</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>11290608, 15256420, 26108694</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158640_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158642_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158646_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158664_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>1880</td>\n",
       "      <td>2216</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>A146T</td>\n",
       "      <td>A146T</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>19075190, 20570890, 20147967, 2403644</td>\n",
       "      <td>146</td>\n",
       "      <td>12</td>\n",
       "      <td>25378560</td>\n",
       "      <td>25378562</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158666_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1901</td>\n",
       "      <td>2238</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>26037647, 22722830, 20147967</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158666_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>1524</td>\n",
       "      <td>1802</td>\n",
       "      <td>ENST00000241453</td>\n",
       "      <td>NM_004119.2</td>\n",
       "      <td>2322</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>11290608, 23631653, 15256420, 26108694, 24255108</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158669_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>3332</td>\n",
       "      <td>3923</td>\n",
       "      <td>ENST00000269305</td>\n",
       "      <td>NM_000546.5</td>\n",
       "      <td>7157</td>\n",
       "      <td>TP53</td>\n",
       "      <td>Y220C</td>\n",
       "      <td>Y220C</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Loss-of-function</td>\n",
       "      <td>23315175, 15037740, 23630318</td>\n",
       "      <td>220</td>\n",
       "      <td>17</td>\n",
       "      <td>7578189</td>\n",
       "      <td>7578191</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158670_T01_01_WG01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>1524</td>\n",
       "      <td>1802</td>\n",
       "      <td>ENST00000241453</td>\n",
       "      <td>NM_004119.2</td>\n",
       "      <td>2322</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>11290608, 23631653, 15256420, 26108694, 24255108</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158671_T01_01_WG01</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    index Index          Isoform          RefSeq Entrez.Gene.ID    Gene  \\\n",
       "0    2281  2690  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "1    1684  1987  ENST00000342505     NM_002227.2           3716    JAK1   \n",
       "2    2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "3    1724  2036  ENST00000458235     NM_000215.3           3718    JAK3   \n",
       "4    2726  3211  ENST00000351677     NM_002834.3           5781  PTPN11   \n",
       "5    1899  2236  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "6    1598  1887  ENST00000378609  NM_001282539.1           2782    GNB1   \n",
       "7    2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "8    1900  2237  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "9    2287  2696  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "10   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "11   2280  2689  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "12   1901  2238  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "13   1521  1799  ENST00000241453     NM_004119.2           2322    FLT3   \n",
       "14   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "15   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "16   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "17   1880  2216  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "18   1901  2238  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "19   1524  1802  ENST00000241453     NM_004119.2           2322    FLT3   \n",
       "20   3332  3923  ENST00000269305     NM_000546.5           7157    TP53   \n",
       "21   1524  1802  ENST00000241453     NM_004119.2           2322    FLT3   \n",
       "\n",
       "   Alteration Protein.Change Oncogenicity   Mutation.Effect  \\\n",
       "0        G13D           G13D    Oncogenic  Gain-of-function   \n",
       "1       S703I          S703I    Oncogenic  Gain-of-function   \n",
       "2        G12D           G12D    Oncogenic  Gain-of-function   \n",
       "3       M511I          M511I    Oncogenic  Gain-of-function   \n",
       "4        T73I           T73I    Oncogenic  Gain-of-function   \n",
       "5        G12V           G12V    Oncogenic  Gain-of-function   \n",
       "6        K89E           K89E    Oncogenic  Gain-of-function   \n",
       "7        G12D           G12D    Oncogenic  Gain-of-function   \n",
       "8        G13C           G13C    Oncogenic  Gain-of-function   \n",
       "9        Q61R           Q61R    Oncogenic  Gain-of-function   \n",
       "10       G12D           G12D    Oncogenic  Gain-of-function   \n",
       "11       G12V           G12V    Oncogenic  Gain-of-function   \n",
       "12       G13D           G13D    Oncogenic  Gain-of-function   \n",
       "13      D835H          D835H    Oncogenic  Gain-of-function   \n",
       "14       G12D           G12D    Oncogenic  Gain-of-function   \n",
       "15       G12D           G12D    Oncogenic  Gain-of-function   \n",
       "16       G12D           G12D    Oncogenic  Gain-of-function   \n",
       "17      A146T          A146T    Oncogenic  Gain-of-function   \n",
       "18       G13D           G13D    Oncogenic  Gain-of-function   \n",
       "19      D835Y          D835Y    Oncogenic  Gain-of-function   \n",
       "20      Y220C          Y220C    Oncogenic  Loss-of-function   \n",
       "21      D835Y          D835Y    Oncogenic  Gain-of-function   \n",
       "\n",
       "                            PMIDs.for.Mutation.Effect aaPos chr_gnm  \\\n",
       "0              26037647, 21993244, 24651010, 22722830    13       1   \n",
       "1                        24735922, 24825865, 23788652   703       1   \n",
       "2    21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "3                                            25193870   511      19   \n",
       "4                                            15834506    73      12   \n",
       "5   20516123, 20949621, 25359494, 20570890, 129572...    12      12   \n",
       "6                                            25485910    89       1   \n",
       "7    21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "8                                  25705018, 21079152    13      12   \n",
       "9   25277205, 21993244, 20194776, 23515407, 246510...    61       1   \n",
       "10   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "11             18952898, 21993244, 21478863, 24651010    12       1   \n",
       "12                       26037647, 22722830, 20147967    13      12   \n",
       "13                       11290608, 15256420, 26108694   835      13   \n",
       "14   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "15   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "16   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "17              19075190, 20570890, 20147967, 2403644   146      12   \n",
       "18                       26037647, 22722830, 20147967    13      12   \n",
       "19   11290608, 23631653, 15256420, 26108694, 24255108   835      13   \n",
       "20                       23315175, 15037740, 23630318   220      17   \n",
       "21   11290608, 23631653, 15256420, 26108694, 24255108   835      13   \n",
       "\n",
       "    start_gnm    end_gnm  Abstracts.for.Mutation.Effect  \\\n",
       "0   115258743  115258745                            NaN   \n",
       "1    65311202   65311204                            NaN   \n",
       "2   115258746  115258748                            NaN   \n",
       "3    17949108   17949110                            NaN   \n",
       "4   112888201  112888203                            NaN   \n",
       "5    25398283   25398285                            NaN   \n",
       "6     1737914    1737916                            NaN   \n",
       "7   115258746  115258748                            NaN   \n",
       "8    25398280   25398282                            NaN   \n",
       "9   115256528  115256530                            NaN   \n",
       "10  115258746  115258748                            NaN   \n",
       "11  115258746  115258748                            NaN   \n",
       "12   25398280   25398282                            NaN   \n",
       "13   28592640   28592642                            NaN   \n",
       "14  115258746  115258748                            NaN   \n",
       "15  115258746  115258748                            NaN   \n",
       "16  115258746  115258748                            NaN   \n",
       "17   25378560   25378562                            NaN   \n",
       "18   25398280   25398282                            NaN   \n",
       "19   28592640   28592642                            NaN   \n",
       "20    7578189    7578191                            NaN   \n",
       "21   28592640   28592642                            NaN   \n",
       "\n",
       "                     Sample  \n",
       "0   IID_H158605_T01_01_WG01  \n",
       "1   IID_H158608_T01_01_WG01  \n",
       "2   IID_H158608_T01_01_WG01  \n",
       "3   IID_H158608_T01_01_WG01  \n",
       "4   IID_H158610_T01_01_WG01  \n",
       "5   IID_H158614_T01_01_WG01  \n",
       "6   IID_H158619_T01_01_WG01  \n",
       "7   IID_H158621_T01_01_WG01  \n",
       "8   IID_H158627_T01_01_WG01  \n",
       "9   IID_H158631_T01_01_WG01  \n",
       "10  IID_H158634_T01_01_WG01  \n",
       "11  IID_H158637_T01_01_WG01  \n",
       "12  IID_H158637_T01_01_WG01  \n",
       "13  IID_H158640_T01_01_WG01  \n",
       "14  IID_H158642_T01_01_WG01  \n",
       "15  IID_H158646_T01_01_WG01  \n",
       "16  IID_H158664_T01_01_WG01  \n",
       "17  IID_H158666_T01_01_WG01  \n",
       "18  IID_H158666_T01_01_WG01  \n",
       "19  IID_H158669_T01_01_WG01  \n",
       "20  IID_H158670_T01_01_WG01  \n",
       "21  IID_H158671_T01_01_WG01  "
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "oncokb_info[oncokb_info['Oncogenicity']=='Oncogenic'].reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(28, 16)"
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "oncokb_info.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# - Check Isabl: "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "##Tables:\n",
    "#oncokb_info\n",
    "#stop_total"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "keys=[]\n",
    "for index,row in oncokb_info.iterrows():\n",
    "    key=row['chr_gnm']+'_'+row['start_gnm']+'_'+row['end_gnm']+'_'+row['Sample']\n",
    "    keys.append(key)\n",
    "oncokb_info['keys']=keys"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "oncokb_info_check=oncokb_info[~(oncokb_info['Sample']=='IID_H158647_T01_01_WG01')]\n",
    "oncokb_info_check=oncokb_info_check.sort_values(by=['Sample'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- Drivers:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158605_T01_01_WG01\n",
      "IID_H158608_T01_01_WG01\n",
      "IID_H158610_T01_01_WG01\n",
      "IID_H158614_T01_01_WG01\n",
      "IID_H158618_T01_01_WG01\n",
      "IID_H158619_T01_01_WG01\n",
      "IID_H158621_T01_01_WG01\n",
      "IID_H158624_T01_01_WG01\n",
      "IID_H158625_T01_01_WG01\n",
      "IID_H158627_T01_01_WG01\n",
      "IID_H158631_T01_01_WG01\n",
      "IID_H158632_T01_01_WG01\n",
      "IID_H158634_T01_01_WG01\n",
      "IID_H158637_T01_01_WG01\n",
      "IID_H158640_T01_01_WG01\n",
      "IID_H158642_T01_01_WG01\n",
      "IID_H158646_T01_01_WG01\n",
      "IID_H158664_T01_01_WG01\n",
      "IID_H158666_T01_01_WG01\n",
      "IID_H158669_T01_01_WG01\n",
      "IID_H158670_T01_01_WG01\n",
      "IID_H158671_T01_01_WG01\n",
      "SVsupport_drivers:\n",
      "25\n",
      "No SVsupport_drivers:\n",
      "3\n"
     ]
    }
   ],
   "source": [
    "samples_noDetect=list(oncokb_info_check['Sample'].unique())\n",
    "SV_support_drivers=list()\n",
    "SV_no_support_drivers=list()\n",
    "\n",
    "for n in samples_noDetect:\n",
    "    #isabl\n",
    "    list_annotSVs=!isabl get-outdirs -fi application__name ANNOT_SNVS -fi targets__system_id {n}\n",
    "    path=list_annotSVs[1]+'/drivers/*.snvs.output.annot.coding.tsv.gz'\n",
    "    print(n)\n",
    "\n",
    "    svs_pass=!ls {path}\n",
    "    \n",
    "    #iterate df\n",
    "    df=oncokb_info[oncokb_info['Sample']==n]\n",
    "    df=df.drop_duplicates(subset=['keys'])\n",
    "    #print(df.shape)\n",
    "    #print('- ',n,':','\\n',df[['chr_seg','start_seg','end_seg','CNV','Sample','Fraction']],'\\n')\n",
    "    chrom_list=list()\n",
    "    \n",
    "    #SV_support=list()\n",
    "    for index,row in df.iterrows():\n",
    "        gene=row['Gene']\n",
    "        alteration=row['Alteration']\n",
    "\n",
    "        \n",
    "        #Awk command + print\n",
    "        awk_output=!zless {svs_pass[0]} | grep {gene} | grep {alteration}\n",
    "        awk_output=list(set(awk_output))\n",
    "        #print(awk_output)\n",
    "        #if (not chrom in chrom_list):\n",
    "            #for n2 in awk_output:\n",
    "                #print(n2)\n",
    "            #print('\\n')\n",
    "            #chrom_list.append(chrom)\n",
    "        \n",
    "        #SV support\n",
    "        \n",
    "        if(len(awk_output)==0):\n",
    "            SV_no_support_drivers.append(row['keys'])\n",
    "        else:            \n",
    "            SV_support_drivers.append(row['keys'])\n",
    "\n",
    "print('SVsupport_drivers:')\n",
    "SV_support_drivers=list(set(SV_support_drivers))\n",
    "print(len(SV_support_drivers))\n",
    "#for n3 in sorted(SV_support):\n",
    "    #print(n3)\n",
    "print('No SVsupport_drivers:')\n",
    "SV_no_support=list(set(SV_no_support_drivers))\n",
    "print(len(SV_no_support_drivers))\n",
    "#for n3 in sorted(SV_no_support):\n",
    "    #print(n3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "oncokb_info_matched=oncokb_info.copy()\n",
    "\n",
    "match_support=[]\n",
    "for index,row in oncokb_info_matched.iterrows():\n",
    "    if(row['keys'] in SV_support_drivers):\n",
    "        match_support.append('Detected')\n",
    "    else:\n",
    "        match_support.append('Non Detected')\n",
    "oncokb_info_matched['Matched']=match_support"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>Index</th>\n",
       "      <th>Isoform</th>\n",
       "      <th>RefSeq</th>\n",
       "      <th>Entrez.Gene.ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Alteration</th>\n",
       "      <th>Protein.Change</th>\n",
       "      <th>Oncogenicity</th>\n",
       "      <th>Mutation.Effect</th>\n",
       "      <th>PMIDs.for.Mutation.Effect</th>\n",
       "      <th>aaPos</th>\n",
       "      <th>chr_gnm</th>\n",
       "      <th>start_gnm</th>\n",
       "      <th>end_gnm</th>\n",
       "      <th>Abstracts.for.Mutation.Effect</th>\n",
       "      <th>Sample</th>\n",
       "      <th>keys</th>\n",
       "      <th>Matched</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2281</td>\n",
       "      <td>2690</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>26037647, 21993244, 24651010, 22722830</td>\n",
       "      <td>13</td>\n",
       "      <td>1</td>\n",
       "      <td>115258743</td>\n",
       "      <td>115258745</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158605_T01_01_WG01</td>\n",
       "      <td>1_115258743_115258745_IID_H158605_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1684</td>\n",
       "      <td>1987</td>\n",
       "      <td>ENST00000342505</td>\n",
       "      <td>NM_002227.2</td>\n",
       "      <td>3716</td>\n",
       "      <td>JAK1</td>\n",
       "      <td>S703I</td>\n",
       "      <td>S703I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>24735922, 24825865, 23788652</td>\n",
       "      <td>703</td>\n",
       "      <td>1</td>\n",
       "      <td>65311202</td>\n",
       "      <td>65311204</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>1_65311202_65311204_IID_H158608_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158608_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1724</td>\n",
       "      <td>2036</td>\n",
       "      <td>ENST00000458235</td>\n",
       "      <td>NM_000215.3</td>\n",
       "      <td>3718</td>\n",
       "      <td>JAK3</td>\n",
       "      <td>M511I</td>\n",
       "      <td>M511I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25193870</td>\n",
       "      <td>511</td>\n",
       "      <td>19</td>\n",
       "      <td>17949108</td>\n",
       "      <td>17949110</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>19_17949108_17949110_IID_H158608_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>2726</td>\n",
       "      <td>3211</td>\n",
       "      <td>ENST00000351677</td>\n",
       "      <td>NM_002834.3</td>\n",
       "      <td>5781</td>\n",
       "      <td>PTPN11</td>\n",
       "      <td>T73I</td>\n",
       "      <td>T73I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>15834506</td>\n",
       "      <td>73</td>\n",
       "      <td>12</td>\n",
       "      <td>112888201</td>\n",
       "      <td>112888203</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158610_T01_01_WG01</td>\n",
       "      <td>12_112888201_112888203_IID_H158610_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1899</td>\n",
       "      <td>2236</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G12V</td>\n",
       "      <td>G12V</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>20516123, 20949621, 25359494, 20570890, 129572...</td>\n",
       "      <td>12</td>\n",
       "      <td>12</td>\n",
       "      <td>25398283</td>\n",
       "      <td>25398285</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158614_T01_01_WG01</td>\n",
       "      <td>12_25398283_25398285_IID_H158614_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>3337</td>\n",
       "      <td>3928</td>\n",
       "      <td>ENST00000269305</td>\n",
       "      <td>NM_000546.5</td>\n",
       "      <td>7157</td>\n",
       "      <td>TP53</td>\n",
       "      <td>Y236D</td>\n",
       "      <td>Y236D</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Loss-of-function</td>\n",
       "      <td>8633021</td>\n",
       "      <td>236</td>\n",
       "      <td>17</td>\n",
       "      <td>7577573</td>\n",
       "      <td>7577575</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158618_T01_01_WG01</td>\n",
       "      <td>17_7577573_7577575_IID_H158618_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>1598</td>\n",
       "      <td>1887</td>\n",
       "      <td>ENST00000378609</td>\n",
       "      <td>NM_001282539.1</td>\n",
       "      <td>2782</td>\n",
       "      <td>GNB1</td>\n",
       "      <td>K89E</td>\n",
       "      <td>K89E</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25485910</td>\n",
       "      <td>89</td>\n",
       "      <td>1</td>\n",
       "      <td>1737914</td>\n",
       "      <td>1737916</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158619_T01_01_WG01</td>\n",
       "      <td>1_1737914_1737916_IID_H158619_T01_01_WG01</td>\n",
       "      <td>Non Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158621_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158621_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>711</td>\n",
       "      <td>805</td>\n",
       "      <td>ENST00000380152</td>\n",
       "      <td>NM_000059.3</td>\n",
       "      <td>675</td>\n",
       "      <td>BRCA2</td>\n",
       "      <td>R2784W</td>\n",
       "      <td>R2784W</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Likely Loss-of-function</td>\n",
       "      <td>18451181</td>\n",
       "      <td>2784</td>\n",
       "      <td>13</td>\n",
       "      <td>32944557</td>\n",
       "      <td>32944559</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158624_T01_01_WG01</td>\n",
       "      <td>13_32944557_32944559_IID_H158624_T01_01_WG01</td>\n",
       "      <td>Non Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>616</td>\n",
       "      <td>709</td>\n",
       "      <td>ENST00000380152</td>\n",
       "      <td>NM_000059.3</td>\n",
       "      <td>675</td>\n",
       "      <td>BRCA2</td>\n",
       "      <td>C1365Y</td>\n",
       "      <td>C1365Y</td>\n",
       "      <td>Inconclusive</td>\n",
       "      <td>Inconclusive</td>\n",
       "      <td>17924331</td>\n",
       "      <td>1365</td>\n",
       "      <td>13</td>\n",
       "      <td>32912585</td>\n",
       "      <td>32912587</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158625_T01_01_WG01</td>\n",
       "      <td>13_32912585_32912587_IID_H158625_T01_01_WG01</td>\n",
       "      <td>Non Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>1900</td>\n",
       "      <td>2237</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13C</td>\n",
       "      <td>G13C</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25705018, 21079152</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158627_T01_01_WG01</td>\n",
       "      <td>12_25398280_25398282_IID_H158627_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>2287</td>\n",
       "      <td>2696</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>Q61R</td>\n",
       "      <td>Q61R</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25277205, 21993244, 20194776, 23515407, 246510...</td>\n",
       "      <td>61</td>\n",
       "      <td>1</td>\n",
       "      <td>115256528</td>\n",
       "      <td>115256530</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158631_T01_01_WG01</td>\n",
       "      <td>1_115256528_115256530_IID_H158631_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>3279</td>\n",
       "      <td>3869</td>\n",
       "      <td>ENST00000269305</td>\n",
       "      <td>NM_000546.5</td>\n",
       "      <td>7157</td>\n",
       "      <td>TP53</td>\n",
       "      <td>R248Q</td>\n",
       "      <td>R248Q</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Likely Loss-of-function</td>\n",
       "      <td>24814347, 21445056, 16778209</td>\n",
       "      <td>248</td>\n",
       "      <td>17</td>\n",
       "      <td>7577537</td>\n",
       "      <td>7577539</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158631_T01_01_WG01</td>\n",
       "      <td>17_7577537_7577539_IID_H158631_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>3279</td>\n",
       "      <td>3869</td>\n",
       "      <td>ENST00000269305</td>\n",
       "      <td>NM_000546.5</td>\n",
       "      <td>7157</td>\n",
       "      <td>TP53</td>\n",
       "      <td>R248Q</td>\n",
       "      <td>R248Q</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Likely Loss-of-function</td>\n",
       "      <td>24814347, 21445056, 16778209</td>\n",
       "      <td>248</td>\n",
       "      <td>17</td>\n",
       "      <td>7577537</td>\n",
       "      <td>7577539</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>17_7577537_7577539_IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158634_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158634_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>2280</td>\n",
       "      <td>2689</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12V</td>\n",
       "      <td>G12V</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>18952898, 21993244, 21478863, 24651010</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158637_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158637_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>1901</td>\n",
       "      <td>2238</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>26037647, 22722830, 20147967</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158637_T01_01_WG01</td>\n",
       "      <td>12_25398280_25398282_IID_H158637_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1521</td>\n",
       "      <td>1799</td>\n",
       "      <td>ENST00000241453</td>\n",
       "      <td>NM_004119.2</td>\n",
       "      <td>2322</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835H</td>\n",
       "      <td>D835H</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>11290608, 15256420, 26108694</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158640_T01_01_WG01</td>\n",
       "      <td>13_28592640_28592642_IID_H158640_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158642_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158642_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>921</td>\n",
       "      <td>1066</td>\n",
       "      <td>ENST00000575354</td>\n",
       "      <td>NM_015125.3</td>\n",
       "      <td>23152</td>\n",
       "      <td>CIC</td>\n",
       "      <td>R215W</td>\n",
       "      <td>R215W</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Loss-of-function</td>\n",
       "      <td>28278156</td>\n",
       "      <td>215</td>\n",
       "      <td>19</td>\n",
       "      <td>42791757</td>\n",
       "      <td>42791759</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158642_T01_01_WG01</td>\n",
       "      <td>19_42791757_42791759_IID_H158642_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158646_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158646_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158664_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158664_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>1880</td>\n",
       "      <td>2216</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>A146T</td>\n",
       "      <td>A146T</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>19075190, 20570890, 20147967, 2403644</td>\n",
       "      <td>146</td>\n",
       "      <td>12</td>\n",
       "      <td>25378560</td>\n",
       "      <td>25378562</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158666_T01_01_WG01</td>\n",
       "      <td>12_25378560_25378562_IID_H158666_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>1901</td>\n",
       "      <td>2238</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>26037647, 22722830, 20147967</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158666_T01_01_WG01</td>\n",
       "      <td>12_25398280_25398282_IID_H158666_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>1524</td>\n",
       "      <td>1802</td>\n",
       "      <td>ENST00000241453</td>\n",
       "      <td>NM_004119.2</td>\n",
       "      <td>2322</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>11290608, 23631653, 15256420, 26108694, 24255108</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158669_T01_01_WG01</td>\n",
       "      <td>13_28592640_28592642_IID_H158669_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>3332</td>\n",
       "      <td>3923</td>\n",
       "      <td>ENST00000269305</td>\n",
       "      <td>NM_000546.5</td>\n",
       "      <td>7157</td>\n",
       "      <td>TP53</td>\n",
       "      <td>Y220C</td>\n",
       "      <td>Y220C</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Loss-of-function</td>\n",
       "      <td>23315175, 15037740, 23630318</td>\n",
       "      <td>220</td>\n",
       "      <td>17</td>\n",
       "      <td>7578189</td>\n",
       "      <td>7578191</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158670_T01_01_WG01</td>\n",
       "      <td>17_7578189_7578191_IID_H158670_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>1524</td>\n",
       "      <td>1802</td>\n",
       "      <td>ENST00000241453</td>\n",
       "      <td>NM_004119.2</td>\n",
       "      <td>2322</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>11290608, 23631653, 15256420, 26108694, 24255108</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158671_T01_01_WG01</td>\n",
       "      <td>13_28592640_28592642_IID_H158671_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    index Index          Isoform          RefSeq Entrez.Gene.ID    Gene  \\\n",
       "0    2281  2690  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "1    1684  1987  ENST00000342505     NM_002227.2           3716    JAK1   \n",
       "2    2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "3    1724  2036  ENST00000458235     NM_000215.3           3718    JAK3   \n",
       "4    2726  3211  ENST00000351677     NM_002834.3           5781  PTPN11   \n",
       "5    1899  2236  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "6    3337  3928  ENST00000269305     NM_000546.5           7157    TP53   \n",
       "7    1598  1887  ENST00000378609  NM_001282539.1           2782    GNB1   \n",
       "8    2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "9     711   805  ENST00000380152     NM_000059.3            675   BRCA2   \n",
       "10    616   709  ENST00000380152     NM_000059.3            675   BRCA2   \n",
       "11   1900  2237  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "12   2287  2696  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "13   3279  3869  ENST00000269305     NM_000546.5           7157    TP53   \n",
       "14   3279  3869  ENST00000269305     NM_000546.5           7157    TP53   \n",
       "15   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "16   2280  2689  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "17   1901  2238  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "18   1521  1799  ENST00000241453     NM_004119.2           2322    FLT3   \n",
       "19   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "20    921  1066  ENST00000575354     NM_015125.3          23152     CIC   \n",
       "21   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "22   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "23   1880  2216  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "24   1901  2238  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "25   1524  1802  ENST00000241453     NM_004119.2           2322    FLT3   \n",
       "26   3332  3923  ENST00000269305     NM_000546.5           7157    TP53   \n",
       "27   1524  1802  ENST00000241453     NM_004119.2           2322    FLT3   \n",
       "\n",
       "   Alteration Protein.Change      Oncogenicity          Mutation.Effect  \\\n",
       "0        G13D           G13D         Oncogenic         Gain-of-function   \n",
       "1       S703I          S703I         Oncogenic         Gain-of-function   \n",
       "2        G12D           G12D         Oncogenic         Gain-of-function   \n",
       "3       M511I          M511I         Oncogenic         Gain-of-function   \n",
       "4        T73I           T73I         Oncogenic         Gain-of-function   \n",
       "5        G12V           G12V         Oncogenic         Gain-of-function   \n",
       "6       Y236D          Y236D  Likely Oncogenic         Loss-of-function   \n",
       "7        K89E           K89E         Oncogenic         Gain-of-function   \n",
       "8        G12D           G12D         Oncogenic         Gain-of-function   \n",
       "9      R2784W         R2784W  Likely Oncogenic  Likely Loss-of-function   \n",
       "10     C1365Y         C1365Y      Inconclusive             Inconclusive   \n",
       "11       G13C           G13C         Oncogenic         Gain-of-function   \n",
       "12       Q61R           Q61R         Oncogenic         Gain-of-function   \n",
       "13      R248Q          R248Q  Likely Oncogenic  Likely Loss-of-function   \n",
       "14      R248Q          R248Q  Likely Oncogenic  Likely Loss-of-function   \n",
       "15       G12D           G12D         Oncogenic         Gain-of-function   \n",
       "16       G12V           G12V         Oncogenic         Gain-of-function   \n",
       "17       G13D           G13D         Oncogenic         Gain-of-function   \n",
       "18      D835H          D835H         Oncogenic         Gain-of-function   \n",
       "19       G12D           G12D         Oncogenic         Gain-of-function   \n",
       "20      R215W          R215W  Likely Oncogenic         Loss-of-function   \n",
       "21       G12D           G12D         Oncogenic         Gain-of-function   \n",
       "22       G12D           G12D         Oncogenic         Gain-of-function   \n",
       "23      A146T          A146T         Oncogenic         Gain-of-function   \n",
       "24       G13D           G13D         Oncogenic         Gain-of-function   \n",
       "25      D835Y          D835Y         Oncogenic         Gain-of-function   \n",
       "26      Y220C          Y220C         Oncogenic         Loss-of-function   \n",
       "27      D835Y          D835Y         Oncogenic         Gain-of-function   \n",
       "\n",
       "                            PMIDs.for.Mutation.Effect aaPos chr_gnm  \\\n",
       "0              26037647, 21993244, 24651010, 22722830    13       1   \n",
       "1                        24735922, 24825865, 23788652   703       1   \n",
       "2    21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "3                                            25193870   511      19   \n",
       "4                                            15834506    73      12   \n",
       "5   20516123, 20949621, 25359494, 20570890, 129572...    12      12   \n",
       "6                                             8633021   236      17   \n",
       "7                                            25485910    89       1   \n",
       "8    21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "9                                            18451181  2784      13   \n",
       "10                                           17924331  1365      13   \n",
       "11                                 25705018, 21079152    13      12   \n",
       "12  25277205, 21993244, 20194776, 23515407, 246510...    61       1   \n",
       "13                       24814347, 21445056, 16778209   248      17   \n",
       "14                       24814347, 21445056, 16778209   248      17   \n",
       "15   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "16             18952898, 21993244, 21478863, 24651010    12       1   \n",
       "17                       26037647, 22722830, 20147967    13      12   \n",
       "18                       11290608, 15256420, 26108694   835      13   \n",
       "19   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "20                                           28278156   215      19   \n",
       "21   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "22   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "23              19075190, 20570890, 20147967, 2403644   146      12   \n",
       "24                       26037647, 22722830, 20147967    13      12   \n",
       "25   11290608, 23631653, 15256420, 26108694, 24255108   835      13   \n",
       "26                       23315175, 15037740, 23630318   220      17   \n",
       "27   11290608, 23631653, 15256420, 26108694, 24255108   835      13   \n",
       "\n",
       "    start_gnm    end_gnm  Abstracts.for.Mutation.Effect  \\\n",
       "0   115258743  115258745                            NaN   \n",
       "1    65311202   65311204                            NaN   \n",
       "2   115258746  115258748                            NaN   \n",
       "3    17949108   17949110                            NaN   \n",
       "4   112888201  112888203                            NaN   \n",
       "5    25398283   25398285                            NaN   \n",
       "6     7577573    7577575                            NaN   \n",
       "7     1737914    1737916                            NaN   \n",
       "8   115258746  115258748                            NaN   \n",
       "9    32944557   32944559                            NaN   \n",
       "10   32912585   32912587                            NaN   \n",
       "11   25398280   25398282                            NaN   \n",
       "12  115256528  115256530                            NaN   \n",
       "13    7577537    7577539                            NaN   \n",
       "14    7577537    7577539                            NaN   \n",
       "15  115258746  115258748                            NaN   \n",
       "16  115258746  115258748                            NaN   \n",
       "17   25398280   25398282                            NaN   \n",
       "18   28592640   28592642                            NaN   \n",
       "19  115258746  115258748                            NaN   \n",
       "20   42791757   42791759                            NaN   \n",
       "21  115258746  115258748                            NaN   \n",
       "22  115258746  115258748                            NaN   \n",
       "23   25378560   25378562                            NaN   \n",
       "24   25398280   25398282                            NaN   \n",
       "25   28592640   28592642                            NaN   \n",
       "26    7578189    7578191                            NaN   \n",
       "27   28592640   28592642                            NaN   \n",
       "\n",
       "                     Sample                                            keys  \\\n",
       "0   IID_H158605_T01_01_WG01   1_115258743_115258745_IID_H158605_T01_01_WG01   \n",
       "1   IID_H158608_T01_01_WG01     1_65311202_65311204_IID_H158608_T01_01_WG01   \n",
       "2   IID_H158608_T01_01_WG01   1_115258746_115258748_IID_H158608_T01_01_WG01   \n",
       "3   IID_H158608_T01_01_WG01    19_17949108_17949110_IID_H158608_T01_01_WG01   \n",
       "4   IID_H158610_T01_01_WG01  12_112888201_112888203_IID_H158610_T01_01_WG01   \n",
       "5   IID_H158614_T01_01_WG01    12_25398283_25398285_IID_H158614_T01_01_WG01   \n",
       "6   IID_H158618_T01_01_WG01      17_7577573_7577575_IID_H158618_T01_01_WG01   \n",
       "7   IID_H158619_T01_01_WG01       1_1737914_1737916_IID_H158619_T01_01_WG01   \n",
       "8   IID_H158621_T01_01_WG01   1_115258746_115258748_IID_H158621_T01_01_WG01   \n",
       "9   IID_H158624_T01_01_WG01    13_32944557_32944559_IID_H158624_T01_01_WG01   \n",
       "10  IID_H158625_T01_01_WG01    13_32912585_32912587_IID_H158625_T01_01_WG01   \n",
       "11  IID_H158627_T01_01_WG01    12_25398280_25398282_IID_H158627_T01_01_WG01   \n",
       "12  IID_H158631_T01_01_WG01   1_115256528_115256530_IID_H158631_T01_01_WG01   \n",
       "13  IID_H158631_T01_01_WG01      17_7577537_7577539_IID_H158631_T01_01_WG01   \n",
       "14  IID_H158632_T01_01_WG01      17_7577537_7577539_IID_H158632_T01_01_WG01   \n",
       "15  IID_H158634_T01_01_WG01   1_115258746_115258748_IID_H158634_T01_01_WG01   \n",
       "16  IID_H158637_T01_01_WG01   1_115258746_115258748_IID_H158637_T01_01_WG01   \n",
       "17  IID_H158637_T01_01_WG01    12_25398280_25398282_IID_H158637_T01_01_WG01   \n",
       "18  IID_H158640_T01_01_WG01    13_28592640_28592642_IID_H158640_T01_01_WG01   \n",
       "19  IID_H158642_T01_01_WG01   1_115258746_115258748_IID_H158642_T01_01_WG01   \n",
       "20  IID_H158642_T01_01_WG01    19_42791757_42791759_IID_H158642_T01_01_WG01   \n",
       "21  IID_H158646_T01_01_WG01   1_115258746_115258748_IID_H158646_T01_01_WG01   \n",
       "22  IID_H158664_T01_01_WG01   1_115258746_115258748_IID_H158664_T01_01_WG01   \n",
       "23  IID_H158666_T01_01_WG01    12_25378560_25378562_IID_H158666_T01_01_WG01   \n",
       "24  IID_H158666_T01_01_WG01    12_25398280_25398282_IID_H158666_T01_01_WG01   \n",
       "25  IID_H158669_T01_01_WG01    13_28592640_28592642_IID_H158669_T01_01_WG01   \n",
       "26  IID_H158670_T01_01_WG01      17_7578189_7578191_IID_H158670_T01_01_WG01   \n",
       "27  IID_H158671_T01_01_WG01    13_28592640_28592642_IID_H158671_T01_01_WG01   \n",
       "\n",
       "         Matched  \n",
       "0       Detected  \n",
       "1       Detected  \n",
       "2       Detected  \n",
       "3       Detected  \n",
       "4       Detected  \n",
       "5       Detected  \n",
       "6       Detected  \n",
       "7   Non Detected  \n",
       "8       Detected  \n",
       "9   Non Detected  \n",
       "10  Non Detected  \n",
       "11      Detected  \n",
       "12      Detected  \n",
       "13      Detected  \n",
       "14      Detected  \n",
       "15      Detected  \n",
       "16      Detected  \n",
       "17      Detected  \n",
       "18      Detected  \n",
       "19      Detected  \n",
       "20      Detected  \n",
       "21      Detected  \n",
       "22      Detected  \n",
       "23      Detected  \n",
       "24      Detected  \n",
       "25      Detected  \n",
       "26      Detected  \n",
       "27      Detected  "
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "oncokb_info_matched.reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>Index</th>\n",
       "      <th>Isoform</th>\n",
       "      <th>RefSeq</th>\n",
       "      <th>Entrez.Gene.ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Alteration</th>\n",
       "      <th>Protein.Change</th>\n",
       "      <th>Oncogenicity</th>\n",
       "      <th>Mutation.Effect</th>\n",
       "      <th>PMIDs.for.Mutation.Effect</th>\n",
       "      <th>aaPos</th>\n",
       "      <th>chr_gnm</th>\n",
       "      <th>start_gnm</th>\n",
       "      <th>end_gnm</th>\n",
       "      <th>Abstracts.for.Mutation.Effect</th>\n",
       "      <th>Sample</th>\n",
       "      <th>keys</th>\n",
       "      <th>Matched</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2281</td>\n",
       "      <td>2690</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>26037647, 21993244, 24651010, 22722830</td>\n",
       "      <td>13</td>\n",
       "      <td>1</td>\n",
       "      <td>115258743</td>\n",
       "      <td>115258745</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158605_T01_01_WG01</td>\n",
       "      <td>1_115258743_115258745_IID_H158605_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1684</td>\n",
       "      <td>1987</td>\n",
       "      <td>ENST00000342505</td>\n",
       "      <td>NM_002227.2</td>\n",
       "      <td>3716</td>\n",
       "      <td>JAK1</td>\n",
       "      <td>S703I</td>\n",
       "      <td>S703I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>24735922, 24825865, 23788652</td>\n",
       "      <td>703</td>\n",
       "      <td>1</td>\n",
       "      <td>65311202</td>\n",
       "      <td>65311204</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>1_65311202_65311204_IID_H158608_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158608_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1724</td>\n",
       "      <td>2036</td>\n",
       "      <td>ENST00000458235</td>\n",
       "      <td>NM_000215.3</td>\n",
       "      <td>3718</td>\n",
       "      <td>JAK3</td>\n",
       "      <td>M511I</td>\n",
       "      <td>M511I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25193870</td>\n",
       "      <td>511</td>\n",
       "      <td>19</td>\n",
       "      <td>17949108</td>\n",
       "      <td>17949110</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>19_17949108_17949110_IID_H158608_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>2726</td>\n",
       "      <td>3211</td>\n",
       "      <td>ENST00000351677</td>\n",
       "      <td>NM_002834.3</td>\n",
       "      <td>5781</td>\n",
       "      <td>PTPN11</td>\n",
       "      <td>T73I</td>\n",
       "      <td>T73I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>15834506</td>\n",
       "      <td>73</td>\n",
       "      <td>12</td>\n",
       "      <td>112888201</td>\n",
       "      <td>112888203</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158610_T01_01_WG01</td>\n",
       "      <td>12_112888201_112888203_IID_H158610_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1899</td>\n",
       "      <td>2236</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G12V</td>\n",
       "      <td>G12V</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>20516123, 20949621, 25359494, 20570890, 129572...</td>\n",
       "      <td>12</td>\n",
       "      <td>12</td>\n",
       "      <td>25398283</td>\n",
       "      <td>25398285</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158614_T01_01_WG01</td>\n",
       "      <td>12_25398283_25398285_IID_H158614_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>1598</td>\n",
       "      <td>1887</td>\n",
       "      <td>ENST00000378609</td>\n",
       "      <td>NM_001282539.1</td>\n",
       "      <td>2782</td>\n",
       "      <td>GNB1</td>\n",
       "      <td>K89E</td>\n",
       "      <td>K89E</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25485910</td>\n",
       "      <td>89</td>\n",
       "      <td>1</td>\n",
       "      <td>1737914</td>\n",
       "      <td>1737916</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158619_T01_01_WG01</td>\n",
       "      <td>1_1737914_1737916_IID_H158619_T01_01_WG01</td>\n",
       "      <td>Non Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158621_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158621_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>1900</td>\n",
       "      <td>2237</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13C</td>\n",
       "      <td>G13C</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25705018, 21079152</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158627_T01_01_WG01</td>\n",
       "      <td>12_25398280_25398282_IID_H158627_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>2287</td>\n",
       "      <td>2696</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>Q61R</td>\n",
       "      <td>Q61R</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>25277205, 21993244, 20194776, 23515407, 246510...</td>\n",
       "      <td>61</td>\n",
       "      <td>1</td>\n",
       "      <td>115256528</td>\n",
       "      <td>115256530</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158631_T01_01_WG01</td>\n",
       "      <td>1_115256528_115256530_IID_H158631_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158634_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158634_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>2280</td>\n",
       "      <td>2689</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12V</td>\n",
       "      <td>G12V</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>18952898, 21993244, 21478863, 24651010</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158637_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158637_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>1901</td>\n",
       "      <td>2238</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>26037647, 22722830, 20147967</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158637_T01_01_WG01</td>\n",
       "      <td>12_25398280_25398282_IID_H158637_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>1521</td>\n",
       "      <td>1799</td>\n",
       "      <td>ENST00000241453</td>\n",
       "      <td>NM_004119.2</td>\n",
       "      <td>2322</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835H</td>\n",
       "      <td>D835H</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>11290608, 15256420, 26108694</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158640_T01_01_WG01</td>\n",
       "      <td>13_28592640_28592642_IID_H158640_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158642_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158642_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158646_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158646_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>2278</td>\n",
       "      <td>2687</td>\n",
       "      <td>ENST00000369535</td>\n",
       "      <td>NM_002524.4</td>\n",
       "      <td>4893</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>21993244, 23687087, 24651010, 25252692, 18372904</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158664_T01_01_WG01</td>\n",
       "      <td>1_115258746_115258748_IID_H158664_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>1880</td>\n",
       "      <td>2216</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>A146T</td>\n",
       "      <td>A146T</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>19075190, 20570890, 20147967, 2403644</td>\n",
       "      <td>146</td>\n",
       "      <td>12</td>\n",
       "      <td>25378560</td>\n",
       "      <td>25378562</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158666_T01_01_WG01</td>\n",
       "      <td>12_25378560_25378562_IID_H158666_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1901</td>\n",
       "      <td>2238</td>\n",
       "      <td>ENST00000256078</td>\n",
       "      <td>NM_033360.2</td>\n",
       "      <td>3845</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>26037647, 22722830, 20147967</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158666_T01_01_WG01</td>\n",
       "      <td>12_25398280_25398282_IID_H158666_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>1524</td>\n",
       "      <td>1802</td>\n",
       "      <td>ENST00000241453</td>\n",
       "      <td>NM_004119.2</td>\n",
       "      <td>2322</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>11290608, 23631653, 15256420, 26108694, 24255108</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158669_T01_01_WG01</td>\n",
       "      <td>13_28592640_28592642_IID_H158669_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>3332</td>\n",
       "      <td>3923</td>\n",
       "      <td>ENST00000269305</td>\n",
       "      <td>NM_000546.5</td>\n",
       "      <td>7157</td>\n",
       "      <td>TP53</td>\n",
       "      <td>Y220C</td>\n",
       "      <td>Y220C</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Loss-of-function</td>\n",
       "      <td>23315175, 15037740, 23630318</td>\n",
       "      <td>220</td>\n",
       "      <td>17</td>\n",
       "      <td>7578189</td>\n",
       "      <td>7578191</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158670_T01_01_WG01</td>\n",
       "      <td>17_7578189_7578191_IID_H158670_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>1524</td>\n",
       "      <td>1802</td>\n",
       "      <td>ENST00000241453</td>\n",
       "      <td>NM_004119.2</td>\n",
       "      <td>2322</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>11290608, 23631653, 15256420, 26108694, 24255108</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>NaN</td>\n",
       "      <td>IID_H158671_T01_01_WG01</td>\n",
       "      <td>13_28592640_28592642_IID_H158671_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    index Index          Isoform          RefSeq Entrez.Gene.ID    Gene  \\\n",
       "0    2281  2690  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "1    1684  1987  ENST00000342505     NM_002227.2           3716    JAK1   \n",
       "2    2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "3    1724  2036  ENST00000458235     NM_000215.3           3718    JAK3   \n",
       "4    2726  3211  ENST00000351677     NM_002834.3           5781  PTPN11   \n",
       "5    1899  2236  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "6    1598  1887  ENST00000378609  NM_001282539.1           2782    GNB1   \n",
       "7    2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "8    1900  2237  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "9    2287  2696  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "10   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "11   2280  2689  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "12   1901  2238  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "13   1521  1799  ENST00000241453     NM_004119.2           2322    FLT3   \n",
       "14   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "15   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "16   2278  2687  ENST00000369535     NM_002524.4           4893    NRAS   \n",
       "17   1880  2216  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "18   1901  2238  ENST00000256078     NM_033360.2           3845    KRAS   \n",
       "19   1524  1802  ENST00000241453     NM_004119.2           2322    FLT3   \n",
       "20   3332  3923  ENST00000269305     NM_000546.5           7157    TP53   \n",
       "21   1524  1802  ENST00000241453     NM_004119.2           2322    FLT3   \n",
       "\n",
       "   Alteration Protein.Change Oncogenicity   Mutation.Effect  \\\n",
       "0        G13D           G13D    Oncogenic  Gain-of-function   \n",
       "1       S703I          S703I    Oncogenic  Gain-of-function   \n",
       "2        G12D           G12D    Oncogenic  Gain-of-function   \n",
       "3       M511I          M511I    Oncogenic  Gain-of-function   \n",
       "4        T73I           T73I    Oncogenic  Gain-of-function   \n",
       "5        G12V           G12V    Oncogenic  Gain-of-function   \n",
       "6        K89E           K89E    Oncogenic  Gain-of-function   \n",
       "7        G12D           G12D    Oncogenic  Gain-of-function   \n",
       "8        G13C           G13C    Oncogenic  Gain-of-function   \n",
       "9        Q61R           Q61R    Oncogenic  Gain-of-function   \n",
       "10       G12D           G12D    Oncogenic  Gain-of-function   \n",
       "11       G12V           G12V    Oncogenic  Gain-of-function   \n",
       "12       G13D           G13D    Oncogenic  Gain-of-function   \n",
       "13      D835H          D835H    Oncogenic  Gain-of-function   \n",
       "14       G12D           G12D    Oncogenic  Gain-of-function   \n",
       "15       G12D           G12D    Oncogenic  Gain-of-function   \n",
       "16       G12D           G12D    Oncogenic  Gain-of-function   \n",
       "17      A146T          A146T    Oncogenic  Gain-of-function   \n",
       "18       G13D           G13D    Oncogenic  Gain-of-function   \n",
       "19      D835Y          D835Y    Oncogenic  Gain-of-function   \n",
       "20      Y220C          Y220C    Oncogenic  Loss-of-function   \n",
       "21      D835Y          D835Y    Oncogenic  Gain-of-function   \n",
       "\n",
       "                            PMIDs.for.Mutation.Effect aaPos chr_gnm  \\\n",
       "0              26037647, 21993244, 24651010, 22722830    13       1   \n",
       "1                        24735922, 24825865, 23788652   703       1   \n",
       "2    21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "3                                            25193870   511      19   \n",
       "4                                            15834506    73      12   \n",
       "5   20516123, 20949621, 25359494, 20570890, 129572...    12      12   \n",
       "6                                            25485910    89       1   \n",
       "7    21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "8                                  25705018, 21079152    13      12   \n",
       "9   25277205, 21993244, 20194776, 23515407, 246510...    61       1   \n",
       "10   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "11             18952898, 21993244, 21478863, 24651010    12       1   \n",
       "12                       26037647, 22722830, 20147967    13      12   \n",
       "13                       11290608, 15256420, 26108694   835      13   \n",
       "14   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "15   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "16   21993244, 23687087, 24651010, 25252692, 18372904    12       1   \n",
       "17              19075190, 20570890, 20147967, 2403644   146      12   \n",
       "18                       26037647, 22722830, 20147967    13      12   \n",
       "19   11290608, 23631653, 15256420, 26108694, 24255108   835      13   \n",
       "20                       23315175, 15037740, 23630318   220      17   \n",
       "21   11290608, 23631653, 15256420, 26108694, 24255108   835      13   \n",
       "\n",
       "    start_gnm    end_gnm  Abstracts.for.Mutation.Effect  \\\n",
       "0   115258743  115258745                            NaN   \n",
       "1    65311202   65311204                            NaN   \n",
       "2   115258746  115258748                            NaN   \n",
       "3    17949108   17949110                            NaN   \n",
       "4   112888201  112888203                            NaN   \n",
       "5    25398283   25398285                            NaN   \n",
       "6     1737914    1737916                            NaN   \n",
       "7   115258746  115258748                            NaN   \n",
       "8    25398280   25398282                            NaN   \n",
       "9   115256528  115256530                            NaN   \n",
       "10  115258746  115258748                            NaN   \n",
       "11  115258746  115258748                            NaN   \n",
       "12   25398280   25398282                            NaN   \n",
       "13   28592640   28592642                            NaN   \n",
       "14  115258746  115258748                            NaN   \n",
       "15  115258746  115258748                            NaN   \n",
       "16  115258746  115258748                            NaN   \n",
       "17   25378560   25378562                            NaN   \n",
       "18   25398280   25398282                            NaN   \n",
       "19   28592640   28592642                            NaN   \n",
       "20    7578189    7578191                            NaN   \n",
       "21   28592640   28592642                            NaN   \n",
       "\n",
       "                     Sample                                            keys  \\\n",
       "0   IID_H158605_T01_01_WG01   1_115258743_115258745_IID_H158605_T01_01_WG01   \n",
       "1   IID_H158608_T01_01_WG01     1_65311202_65311204_IID_H158608_T01_01_WG01   \n",
       "2   IID_H158608_T01_01_WG01   1_115258746_115258748_IID_H158608_T01_01_WG01   \n",
       "3   IID_H158608_T01_01_WG01    19_17949108_17949110_IID_H158608_T01_01_WG01   \n",
       "4   IID_H158610_T01_01_WG01  12_112888201_112888203_IID_H158610_T01_01_WG01   \n",
       "5   IID_H158614_T01_01_WG01    12_25398283_25398285_IID_H158614_T01_01_WG01   \n",
       "6   IID_H158619_T01_01_WG01       1_1737914_1737916_IID_H158619_T01_01_WG01   \n",
       "7   IID_H158621_T01_01_WG01   1_115258746_115258748_IID_H158621_T01_01_WG01   \n",
       "8   IID_H158627_T01_01_WG01    12_25398280_25398282_IID_H158627_T01_01_WG01   \n",
       "9   IID_H158631_T01_01_WG01   1_115256528_115256530_IID_H158631_T01_01_WG01   \n",
       "10  IID_H158634_T01_01_WG01   1_115258746_115258748_IID_H158634_T01_01_WG01   \n",
       "11  IID_H158637_T01_01_WG01   1_115258746_115258748_IID_H158637_T01_01_WG01   \n",
       "12  IID_H158637_T01_01_WG01    12_25398280_25398282_IID_H158637_T01_01_WG01   \n",
       "13  IID_H158640_T01_01_WG01    13_28592640_28592642_IID_H158640_T01_01_WG01   \n",
       "14  IID_H158642_T01_01_WG01   1_115258746_115258748_IID_H158642_T01_01_WG01   \n",
       "15  IID_H158646_T01_01_WG01   1_115258746_115258748_IID_H158646_T01_01_WG01   \n",
       "16  IID_H158664_T01_01_WG01   1_115258746_115258748_IID_H158664_T01_01_WG01   \n",
       "17  IID_H158666_T01_01_WG01    12_25378560_25378562_IID_H158666_T01_01_WG01   \n",
       "18  IID_H158666_T01_01_WG01    12_25398280_25398282_IID_H158666_T01_01_WG01   \n",
       "19  IID_H158669_T01_01_WG01    13_28592640_28592642_IID_H158669_T01_01_WG01   \n",
       "20  IID_H158670_T01_01_WG01      17_7578189_7578191_IID_H158670_T01_01_WG01   \n",
       "21  IID_H158671_T01_01_WG01    13_28592640_28592642_IID_H158671_T01_01_WG01   \n",
       "\n",
       "         Matched  \n",
       "0       Detected  \n",
       "1       Detected  \n",
       "2       Detected  \n",
       "3       Detected  \n",
       "4       Detected  \n",
       "5       Detected  \n",
       "6   Non Detected  \n",
       "7       Detected  \n",
       "8       Detected  \n",
       "9       Detected  \n",
       "10      Detected  \n",
       "11      Detected  \n",
       "12      Detected  \n",
       "13      Detected  \n",
       "14      Detected  \n",
       "15      Detected  \n",
       "16      Detected  \n",
       "17      Detected  \n",
       "18      Detected  \n",
       "19      Detected  \n",
       "20      Detected  \n",
       "21      Detected  "
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "oncokb_info_matched[oncokb_info_matched['Oncogenicity']=='Oncogenic'].reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['Index', 'Isoform', 'RefSeq', 'Entrez.Gene.ID', 'Gene', 'Alteration',\n",
       "       'Protein.Change', 'Oncogenicity', 'Mutation.Effect',\n",
       "       'PMIDs.for.Mutation.Effect', 'aaPos', 'chr_gnm', 'start_gnm', 'end_gnm',\n",
       "       'Abstracts.for.Mutation.Effect', 'Sample', 'keys', 'Matched'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 65,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "oncokb_info_matched.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Alteration</th>\n",
       "      <th>Protein.Change</th>\n",
       "      <th>Oncogenicity</th>\n",
       "      <th>Mutation.Effect</th>\n",
       "      <th>aaPos</th>\n",
       "      <th>chr_gnm</th>\n",
       "      <th>start_gnm</th>\n",
       "      <th>end_gnm</th>\n",
       "      <th>Sample</th>\n",
       "      <th>Matched</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2281</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>13</td>\n",
       "      <td>1</td>\n",
       "      <td>115258743</td>\n",
       "      <td>115258745</td>\n",
       "      <td>IID_H158605_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1684</td>\n",
       "      <td>JAK1</td>\n",
       "      <td>S703I</td>\n",
       "      <td>S703I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>703</td>\n",
       "      <td>1</td>\n",
       "      <td>65311202</td>\n",
       "      <td>65311204</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2278</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1724</td>\n",
       "      <td>JAK3</td>\n",
       "      <td>M511I</td>\n",
       "      <td>M511I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>511</td>\n",
       "      <td>19</td>\n",
       "      <td>17949108</td>\n",
       "      <td>17949110</td>\n",
       "      <td>IID_H158608_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>2726</td>\n",
       "      <td>PTPN11</td>\n",
       "      <td>T73I</td>\n",
       "      <td>T73I</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>73</td>\n",
       "      <td>12</td>\n",
       "      <td>112888201</td>\n",
       "      <td>112888203</td>\n",
       "      <td>IID_H158610_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1899</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G12V</td>\n",
       "      <td>G12V</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>12</td>\n",
       "      <td>12</td>\n",
       "      <td>25398283</td>\n",
       "      <td>25398285</td>\n",
       "      <td>IID_H158614_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>3337</td>\n",
       "      <td>TP53</td>\n",
       "      <td>Y236D</td>\n",
       "      <td>Y236D</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Loss-of-function</td>\n",
       "      <td>236</td>\n",
       "      <td>17</td>\n",
       "      <td>7577573</td>\n",
       "      <td>7577575</td>\n",
       "      <td>IID_H158618_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>1598</td>\n",
       "      <td>GNB1</td>\n",
       "      <td>K89E</td>\n",
       "      <td>K89E</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>89</td>\n",
       "      <td>1</td>\n",
       "      <td>1737914</td>\n",
       "      <td>1737916</td>\n",
       "      <td>IID_H158619_T01_01_WG01</td>\n",
       "      <td>Non Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>2278</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>IID_H158621_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>711</td>\n",
       "      <td>BRCA2</td>\n",
       "      <td>R2784W</td>\n",
       "      <td>R2784W</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Likely Loss-of-function</td>\n",
       "      <td>2784</td>\n",
       "      <td>13</td>\n",
       "      <td>32944557</td>\n",
       "      <td>32944559</td>\n",
       "      <td>IID_H158624_T01_01_WG01</td>\n",
       "      <td>Non Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>616</td>\n",
       "      <td>BRCA2</td>\n",
       "      <td>C1365Y</td>\n",
       "      <td>C1365Y</td>\n",
       "      <td>Inconclusive</td>\n",
       "      <td>Inconclusive</td>\n",
       "      <td>1365</td>\n",
       "      <td>13</td>\n",
       "      <td>32912585</td>\n",
       "      <td>32912587</td>\n",
       "      <td>IID_H158625_T01_01_WG01</td>\n",
       "      <td>Non Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>1900</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13C</td>\n",
       "      <td>G13C</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>IID_H158627_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>2287</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>Q61R</td>\n",
       "      <td>Q61R</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>61</td>\n",
       "      <td>1</td>\n",
       "      <td>115256528</td>\n",
       "      <td>115256530</td>\n",
       "      <td>IID_H158631_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>3279</td>\n",
       "      <td>TP53</td>\n",
       "      <td>R248Q</td>\n",
       "      <td>R248Q</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Likely Loss-of-function</td>\n",
       "      <td>248</td>\n",
       "      <td>17</td>\n",
       "      <td>7577537</td>\n",
       "      <td>7577539</td>\n",
       "      <td>IID_H158631_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>3279</td>\n",
       "      <td>TP53</td>\n",
       "      <td>R248Q</td>\n",
       "      <td>R248Q</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Likely Loss-of-function</td>\n",
       "      <td>248</td>\n",
       "      <td>17</td>\n",
       "      <td>7577537</td>\n",
       "      <td>7577539</td>\n",
       "      <td>IID_H158632_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>2278</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>IID_H158634_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>2280</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12V</td>\n",
       "      <td>G12V</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>IID_H158637_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>1901</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>IID_H158637_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1521</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835H</td>\n",
       "      <td>D835H</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>IID_H158640_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>2278</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>IID_H158642_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>921</td>\n",
       "      <td>CIC</td>\n",
       "      <td>R215W</td>\n",
       "      <td>R215W</td>\n",
       "      <td>Likely Oncogenic</td>\n",
       "      <td>Loss-of-function</td>\n",
       "      <td>215</td>\n",
       "      <td>19</td>\n",
       "      <td>42791757</td>\n",
       "      <td>42791759</td>\n",
       "      <td>IID_H158642_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>2278</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>IID_H158646_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>2278</td>\n",
       "      <td>NRAS</td>\n",
       "      <td>G12D</td>\n",
       "      <td>G12D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>115258746</td>\n",
       "      <td>115258748</td>\n",
       "      <td>IID_H158664_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>1880</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>A146T</td>\n",
       "      <td>A146T</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>146</td>\n",
       "      <td>12</td>\n",
       "      <td>25378560</td>\n",
       "      <td>25378562</td>\n",
       "      <td>IID_H158666_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>1901</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G13D</td>\n",
       "      <td>G13D</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>13</td>\n",
       "      <td>12</td>\n",
       "      <td>25398280</td>\n",
       "      <td>25398282</td>\n",
       "      <td>IID_H158666_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>1524</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>IID_H158669_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>3332</td>\n",
       "      <td>TP53</td>\n",
       "      <td>Y220C</td>\n",
       "      <td>Y220C</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Loss-of-function</td>\n",
       "      <td>220</td>\n",
       "      <td>17</td>\n",
       "      <td>7578189</td>\n",
       "      <td>7578191</td>\n",
       "      <td>IID_H158670_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>1524</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>D835Y</td>\n",
       "      <td>Oncogenic</td>\n",
       "      <td>Gain-of-function</td>\n",
       "      <td>835</td>\n",
       "      <td>13</td>\n",
       "      <td>28592640</td>\n",
       "      <td>28592642</td>\n",
       "      <td>IID_H158671_T01_01_WG01</td>\n",
       "      <td>Detected</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    index    Gene Alteration Protein.Change      Oncogenicity  \\\n",
       "0    2281    NRAS       G13D           G13D         Oncogenic   \n",
       "1    1684    JAK1      S703I          S703I         Oncogenic   \n",
       "2    2278    NRAS       G12D           G12D         Oncogenic   \n",
       "3    1724    JAK3      M511I          M511I         Oncogenic   \n",
       "4    2726  PTPN11       T73I           T73I         Oncogenic   \n",
       "5    1899    KRAS       G12V           G12V         Oncogenic   \n",
       "6    3337    TP53      Y236D          Y236D  Likely Oncogenic   \n",
       "7    1598    GNB1       K89E           K89E         Oncogenic   \n",
       "8    2278    NRAS       G12D           G12D         Oncogenic   \n",
       "9     711   BRCA2     R2784W         R2784W  Likely Oncogenic   \n",
       "10    616   BRCA2     C1365Y         C1365Y      Inconclusive   \n",
       "11   1900    KRAS       G13C           G13C         Oncogenic   \n",
       "12   2287    NRAS       Q61R           Q61R         Oncogenic   \n",
       "13   3279    TP53      R248Q          R248Q  Likely Oncogenic   \n",
       "14   3279    TP53      R248Q          R248Q  Likely Oncogenic   \n",
       "15   2278    NRAS       G12D           G12D         Oncogenic   \n",
       "16   2280    NRAS       G12V           G12V         Oncogenic   \n",
       "17   1901    KRAS       G13D           G13D         Oncogenic   \n",
       "18   1521    FLT3      D835H          D835H         Oncogenic   \n",
       "19   2278    NRAS       G12D           G12D         Oncogenic   \n",
       "20    921     CIC      R215W          R215W  Likely Oncogenic   \n",
       "21   2278    NRAS       G12D           G12D         Oncogenic   \n",
       "22   2278    NRAS       G12D           G12D         Oncogenic   \n",
       "23   1880    KRAS      A146T          A146T         Oncogenic   \n",
       "24   1901    KRAS       G13D           G13D         Oncogenic   \n",
       "25   1524    FLT3      D835Y          D835Y         Oncogenic   \n",
       "26   3332    TP53      Y220C          Y220C         Oncogenic   \n",
       "27   1524    FLT3      D835Y          D835Y         Oncogenic   \n",
       "\n",
       "            Mutation.Effect aaPos chr_gnm  start_gnm    end_gnm  \\\n",
       "0          Gain-of-function    13       1  115258743  115258745   \n",
       "1          Gain-of-function   703       1   65311202   65311204   \n",
       "2          Gain-of-function    12       1  115258746  115258748   \n",
       "3          Gain-of-function   511      19   17949108   17949110   \n",
       "4          Gain-of-function    73      12  112888201  112888203   \n",
       "5          Gain-of-function    12      12   25398283   25398285   \n",
       "6          Loss-of-function   236      17    7577573    7577575   \n",
       "7          Gain-of-function    89       1    1737914    1737916   \n",
       "8          Gain-of-function    12       1  115258746  115258748   \n",
       "9   Likely Loss-of-function  2784      13   32944557   32944559   \n",
       "10             Inconclusive  1365      13   32912585   32912587   \n",
       "11         Gain-of-function    13      12   25398280   25398282   \n",
       "12         Gain-of-function    61       1  115256528  115256530   \n",
       "13  Likely Loss-of-function   248      17    7577537    7577539   \n",
       "14  Likely Loss-of-function   248      17    7577537    7577539   \n",
       "15         Gain-of-function    12       1  115258746  115258748   \n",
       "16         Gain-of-function    12       1  115258746  115258748   \n",
       "17         Gain-of-function    13      12   25398280   25398282   \n",
       "18         Gain-of-function   835      13   28592640   28592642   \n",
       "19         Gain-of-function    12       1  115258746  115258748   \n",
       "20         Loss-of-function   215      19   42791757   42791759   \n",
       "21         Gain-of-function    12       1  115258746  115258748   \n",
       "22         Gain-of-function    12       1  115258746  115258748   \n",
       "23         Gain-of-function   146      12   25378560   25378562   \n",
       "24         Gain-of-function    13      12   25398280   25398282   \n",
       "25         Gain-of-function   835      13   28592640   28592642   \n",
       "26         Loss-of-function   220      17    7578189    7578191   \n",
       "27         Gain-of-function   835      13   28592640   28592642   \n",
       "\n",
       "                     Sample       Matched  \n",
       "0   IID_H158605_T01_01_WG01      Detected  \n",
       "1   IID_H158608_T01_01_WG01      Detected  \n",
       "2   IID_H158608_T01_01_WG01      Detected  \n",
       "3   IID_H158608_T01_01_WG01      Detected  \n",
       "4   IID_H158610_T01_01_WG01      Detected  \n",
       "5   IID_H158614_T01_01_WG01      Detected  \n",
       "6   IID_H158618_T01_01_WG01      Detected  \n",
       "7   IID_H158619_T01_01_WG01  Non Detected  \n",
       "8   IID_H158621_T01_01_WG01      Detected  \n",
       "9   IID_H158624_T01_01_WG01  Non Detected  \n",
       "10  IID_H158625_T01_01_WG01  Non Detected  \n",
       "11  IID_H158627_T01_01_WG01      Detected  \n",
       "12  IID_H158631_T01_01_WG01      Detected  \n",
       "13  IID_H158631_T01_01_WG01      Detected  \n",
       "14  IID_H158632_T01_01_WG01      Detected  \n",
       "15  IID_H158634_T01_01_WG01      Detected  \n",
       "16  IID_H158637_T01_01_WG01      Detected  \n",
       "17  IID_H158637_T01_01_WG01      Detected  \n",
       "18  IID_H158640_T01_01_WG01      Detected  \n",
       "19  IID_H158642_T01_01_WG01      Detected  \n",
       "20  IID_H158642_T01_01_WG01      Detected  \n",
       "21  IID_H158646_T01_01_WG01      Detected  \n",
       "22  IID_H158664_T01_01_WG01      Detected  \n",
       "23  IID_H158666_T01_01_WG01      Detected  \n",
       "24  IID_H158666_T01_01_WG01      Detected  \n",
       "25  IID_H158669_T01_01_WG01      Detected  \n",
       "26  IID_H158670_T01_01_WG01      Detected  \n",
       "27  IID_H158671_T01_01_WG01      Detected  "
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "oncokb_info_matched[['Gene','Alteration','Protein.Change', 'Oncogenicity', 'Mutation.Effect','aaPos', 'chr_gnm', 'start_gnm', 'end_gnm','Sample','Matched']].reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
