{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('/juno/work/isabl/home/zhouy1/mergeSVvcf')\n",
    "import mergesvvcf\n",
    "import gzip\n",
    "import pysam\n",
    "import pandas as pd\n",
    "import subprocess"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/samples.tsv\",'r') as f:\n",
    "    samples = f.readlines()\n",
    "    samples = [i.strip('\\n') for i in samples]\n",
    "\n",
    "    \n",
    "for sample in samples:\n",
    "    print(sample)\n",
    "    brass = f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/uk_all_sv_1213/{sample}/{sample}_vs_I-H-108298-N1-1-D1-1.annotated.flagged.output.tsv\"\n",
    "    brass_vcf = f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/uk_all_sv_1213/{sample}/merged_svs.all.vcf.gz\"\n",
    "    svaba = f\"/work/isabl/home/gutierj2/uk_all/svABA_unmatch/no_germline/{sample}.svaba.sv.vcf.gz\"\n",
    "\n",
    "    # filter svaba calls by # supporting reads, mapping quality\n",
    "    svaba = pysam.VariantFile(svaba, 'r')\n",
    "    header = svaba.header.__str__()\n",
    "\n",
    "    variants_to_keep = []\n",
    "    for v in svaba:\n",
    "        if (v.info['MAPQ']>30) & (v.samples[0]['AD']>10):\n",
    "            variants_to_keep += [v]\n",
    "    svaba_filtered = f'/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/{sample}.svaba.filtered.vcf'\n",
    "    with open(svaba_filtered, 'a') as f:\n",
    "        f.write(header)\n",
    "        for i in variants_to_keep:\n",
    "            f.write(i.__str__())\n",
    "            \n",
    "    # filter brass calls by cat1, convert tsv to vcf\n",
    "    brass = pd.read_csv(brass, sep='\\t')\n",
    "    brass = brass[brass['AnnotSV type'] == 'full']\n",
    "    brass = brass[brass['cat1_filters'] == 0]\n",
    "    brass = brass[brass['brass_score'] > 10]\n",
    "    \n",
    "    \n",
    "    with gzip.open(brass_vcf,'rb') as f:\n",
    "        content = f.readlines()\n",
    "    content = [i.decode('ascii') for i in content]\n",
    "    variants = [i for i in content if not i.startswith('#')]\n",
    "    header = [i for i in content if i.startswith('#')]\n",
    "    \n",
    "    filtered_brass = f'/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/{sample}.brass.filtered.vcf'\n",
    "    with open(filtered_brass,'w') as out:\n",
    "        out.write(''.join(header))\n",
    "\n",
    "        counter = 0\n",
    "        for index, row in brass.iterrows():\n",
    "            start = row['end1']\n",
    "            end = row['end2']\n",
    "            for v in variants:\n",
    "                v_end = v.split('\\t')[4].split(':')[1].strip('N').strip('[').strip(']')\n",
    "                v_start = v.split('\\t')[1]\n",
    "                if (int(v_end) == int(end)) & (int(start) == int(v_start)):\n",
    "                    out.write(v)\n",
    "                if (int(v_end) == int(start)) & (int(end) == int(v_start)):\n",
    "                    out.write(v)\n",
    "    \n",
    "    sorted_brass = f'/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/{sample}.brass.sorted.vcf'\n",
    "    cmd = ['vcf-sort', filtered_brass]\n",
    "    f = open(sorted_brass, \"w\")\n",
    "    subprocess.check_call(cmd, stdout=f)\n",
    "\n",
    "    # intersection between two vcfs\n",
    "    out = f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/merged/{sample}.merged.vcf\"\n",
    "    mergesvvcf.mergedfile.merge(\n",
    "        filenames=[sorted_brass, svaba_filtered],\n",
    "        programs=['brass','svaba'],\n",
    "        forceSV=False,\n",
    "        outfile=out,\n",
    "        slop=100,\n",
    "        min_num_callers=2,\n",
    "        verbose=False\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for sample in samples:\n",
    "    with open(f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/merged/{sample}.merged.vcf\", 'r') as fin:\n",
    "        with open(f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/merged/two_callers/{sample}.consensus.vcf\",'a') as fout:\n",
    "            \n",
    "            content = fin.readlines()\n",
    "            variants = [i for i in content if not i.startswith('#')]\n",
    "            header = [i for i in content if i.startswith('#')]\n",
    "            \n",
    "            fout.write(''.join(header))\n",
    "\n",
    "            for v in variants:\n",
    "                if ('Callers=svaba,brass' in v) & ('PASS' in v):\n",
    "                    fout.write(v)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "    sample = \"IID_H158671_T01_01_WG01\"\n",
    "    brass = f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/uk_all_sv_1213/{sample}/{sample}_vs_I-H-108298-N1-1-D1-1.annotated.flagged.output.tsv\"\n",
    "    brass_vcf = f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/uk_all_sv_1213/{sample}/merged_svs.all.vcf.gz\"\n",
    "    svaba = f\"/work/isabl/home/gutierj2/uk_all/svABA_unmatch/no_germline/{sample}.svaba.sv.vcf.gz\"\n",
    "\n",
    "    # filter svaba calls by # supporting reads, mapping quality\n",
    "    svaba = pysam.VariantFile(svaba, 'r')\n",
    "    header = svaba.header.__str__()\n",
    "\n",
    "    variants_to_keep = []\n",
    "    for v in svaba:\n",
    "        if (v.info['MAPQ']>30) & (v.samples[0]['AD']>10):\n",
    "            variants_to_keep += [v]\n",
    "    svaba_filtered = f'/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/{sample}.svaba.filtered.vcf'\n",
    "    with open(svaba_filtered, 'a') as f:\n",
    "        f.write(header)\n",
    "        for i in variants_to_keep:\n",
    "            f.write(i.__str__())\n",
    "            \n",
    "    # filter brass calls by cat1, convert tsv to vcf\n",
    "    brass = pd.read_csv(brass, sep='\\t')\n",
    "    brass = brass[brass['AnnotSV type'] == 'full']\n",
    "    brass = brass[brass['cat1_filters'] == 0]\n",
    "    \n",
    "    brass"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.options.display.max_columns=None\n",
    "filtered_brass"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# intersection between two vcfs\n",
    "for sample in samples:\n",
    "    \n",
    "    matched_brass = \"/work/isabl/data/analyses/21/37/262137/IID_H158671_T01_01_WG01_vs_IID_H158671_N01_01_WG01.annot.vcf.gz\"\n",
    "    unmatched_consensus = \"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/merged/two_callers/IID_H158671_T01_01_WG01.consensus.vcf\"\n",
    "    out = \"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/test.vcf\"\n",
    "    mergesvvcf.mergedfile.merge(\n",
    "        filenames=[matched_brass, unmatched_consensus],\n",
    "        programs=['matched_brass','unmatched_consensus'],\n",
    "        forceSV=False,\n",
    "        outfile=out,\n",
    "        slop=100,\n",
    "        min_num_callers=2, \n",
    "        verbose=False\n",
    "    )\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import isabl_cli as ii"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ii.get_analyses(targets_system__id='IID_H158671_T01_01_WG01', application__name='BRASS' )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/samples.tsv\",'r') as f:\n",
    "    samples = f.readlines()\n",
    "    samples = [i.strip('\\n') for i in samples]\n",
    "\n",
    "for sample in samples:\n",
    "    print(sample)\n",
    "    gridss = f\"/work/isabl/home/zhouy1/gridss/{sample}_unmatched.sv.vcf\"\n",
    "    brass = f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/uk_all_sv_1213/{sample}/merged_svs.all.vcf.gz\"\n",
    "    svaba = f\"/work/isabl/home/gutierj2/uk_all/svABA_unmatch/no_germline/{sample}.svaba.sv.vcf.gz\"\n",
    "\n",
    "    out = f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/{sample}.unmatched.merged.vcf\"\n",
    "    mergesvvcf.mergedfile.merge(\n",
    "        filenames=[gridss, brass, svaba],\n",
    "        programs=['gridss', 'brass', 'svaba'],\n",
    "        forceSV=False,\n",
    "        outfile=out,\n",
    "        slop=100,\n",
    "        min_num_callers=2, \n",
    "        verbose=False\n",
    "    )\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception ignored in: <bound method _TemporaryFileCloser.__del__ of <tempfile._TemporaryFileCloser object at 0x7f3c95815c18>>\n",
      "Traceback (most recent call last):\n",
      "  File \"/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/tempfile.py\", line 450, in __del__\n",
      "    self.close()\n",
      "  File \"/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/tempfile.py\", line 446, in close\n",
      "    unlink(self.name)\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/unsortedvyy6pz45.vcf'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_UDIRc4.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_sAWukP.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_r6DBGo.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_fNrHFH.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_FYad1X.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_TLvvcD.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_yZiFpY.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_C6sclc.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_dwxyLs.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_47az1_.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_Wh4PVr.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_Y9bH_p.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_TEcbHA.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_rgJOdD.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158640_T01_01_WG01/split_SqYR9G.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_nd6Ch2.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_4tY6og.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_CbjNQE.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_oaHnSz.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_jYubpM.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_rGWXGG.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_Q1vxZX.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_KBkRK8.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_TvwVHU.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_hxOKyc.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_90Dxfc.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_FEBiiU.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_nu4g8f.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_RZZ8v6.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_7WbscM.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158628_T01_01_WG01/split_jQ6e6s.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_g6SUBH.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_93hQ91.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_jZvl6L.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_9YCbRa.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_t_kznA.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_aB8RcY.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_Fz8FGX.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_4Hlbmp.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_vNqX8D.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_01Hvk2.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split__i5eQQ.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_ko8cHL.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split__rPRIB.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_zCipu4.flagged.vcf.gz\n",
      "/work/isabl/home/zhouy1/tmp/tmp_flag/IID_H158625_T01_01_WG01/split_azBGjz.flagged.vcf.gz\n"
     ]
    }
   ],
   "source": [
    "\n",
    "import tempfile\n",
    "import os\n",
    "import glob\n",
    "import subprocess\n",
    "import shutil\n",
    "import gzip\n",
    "\n",
    "tmps = {\n",
    "    \"IID_H158636_T01_01_WG01\": \"/work/isabl/tmp/gutierj2/2020/02/10/tmp6zlaq2/\",\n",
    "    \"IID_H158609_T01_01_WG01\": \"/work/isabl/tmp/gutierj2/2020/02/10/tmpmxZiK9/\",\n",
    "    \"IID_H158646_T01_01_WG01\": \"/work/isabl/tmp/gutierj2/2020/02/10/tmpj5lcMC/\",\n",
    "    \"IID_H158671_T01_01_WG01\": \"/work/isabl/tmp/gutierj2/2020/02/10/tmprgxw9P/\",\n",
    "    \"IID_H158640_T01_01_WG01\": \"/work/isabl/tmp/gutierj2/2020/02/10/tmpU5uN8W/\",\n",
    "    \"IID_H158628_T01_01_WG01\": \"/work/isabl/tmp/gutierj2/2020/02/10/tmpfDbKJX/\",\n",
    "    \"IID_H158625_T01_01_WG01\": \"/work/isabl/tmp/gutierj2/2020/02/10/tmpEBt4Go/\"\n",
    "}\n",
    "\n",
    "root_working_dir = \"/work/isabl/home/zhouy1/tmp/tmp_flag\"\n",
    "\n",
    "for sample, tmp in tmps.items():\n",
    "    working_dir = os.path.join(root_working_dir, sample)\n",
    "\n",
    "    unsorted_merged_vcf = tempfile.NamedTemporaryFile(\n",
    "        prefix=\"unsorted\", suffix=\".vcf\", dir=working_dir, delete=True\n",
    "    )\n",
    "\n",
    "    result = []\n",
    "\n",
    "    for vcf in glob.glob(os.path.join(working_dir, \"*.gz\")):\n",
    "        print(vcf)\n",
    "        with gzip.open(vcf, \"r\") as f:\n",
    "            lines = f.readlines()\n",
    "            headers = [l.decode() for l in lines if l.startswith(b\"#\")]\n",
    "            variants = [l.decode() for l in lines if not l.startswith(b\"#\")]\n",
    "        result.extend(variants)\n",
    "\n",
    "    with open(unsorted_merged_vcf.name, \"w\") as f:\n",
    "        f.write(\"\".join(headers))\n",
    "        f.write(\"\".join(result))\n",
    "\n",
    "\n",
    "    out_vcf = os.path.join(working_dir, \"merged.vcf\")\n",
    "    cmd = [\"bcftools\", 'sort', '-o', out_vcf, unsorted_merged_vcf.name]\n",
    "    subprocess.check_call(cmd)\n",
    "    subprocess.check_call([\"bgzip\", \"-f\", out_vcf])\n",
    "    subprocess.check_call([\"tabix\", \"-f\", \"-p\", \"vcf\", out_vcf + \".gz\"])\n",
    "    unsorted_merged_vcf.close()\n",
    "    # for vcf in os.listdir(working_dir):\n",
    "    #     os.remove(vcf)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "print(1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pysam import VariantFile\n",
    "\n",
    "vcf = pysam.VariantFile(\"/work/isabl/home/zhouy1/dbsnp_138.b37_common.vcf.gz\")\n",
    "\n",
    "v = vcf.fetch('1', 10582, 10584)\n",
    "for i in v:\n",
    "    print(i)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pysam import VariantFile\n",
    "\n",
    "\n",
    "dbvcf_in=VariantFile(\"/ifs/work/leukgen/ref/homo_sapiens/GRCh37d5/dbsnp/138/dbsnp_138.b37.vcf.gz\", 'rb')\n",
    "dbvcf_out=VariantFile('/work/isabl/home/zhouy1/dbsnp_138.b37_common.vcf', 'w', header=dbvcf_in.header)\n",
    "for row in dbvcf_in:\n",
    "    split = row.__str__().split('\\t')\n",
    "    if('COMMON=1' in split[7]):\n",
    "        if('<' not in split[4]):\n",
    "            #print(row)\n",
    "            dbvcf_out.write(row)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "with open(\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/samples.tsv\",'r') as f:\n",
    "    samples = f.readlines()\n",
    "    samples = [i.strip('\\n') for i in samples]\n",
    "    \n",
    "for sample in samples:\n",
    "    print(sample)\n",
    "    df = pd.read_csv(f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/uk_all_sv_1213/{sample}/{sample}_vs_I-H-108298-N1-1-D1-1.annotated.flagged.output.tsv\", sep='\\t')\n",
    "    df[df['cat1_filters']!=0].to_csv(f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/matthieu_eliminated/{sample}.matthieu.filtered.tsv\", sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "x-1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "with open(\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/samples.tsv\",'r') as f:\n",
    "    samples = f.readlines()\n",
    "    samples = [i.strip('\\n') for i in samples]\n",
    "    \n",
    "outdir = \"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge\"\n",
    "for sample in samples:\n",
    "    cmd = [\n",
    "        'bsub',\n",
    "        '-e', os.path.join(outdir, f\"{sample}_err\"),\n",
    "        '-o', os.path.join(outdir, f\"{sample}_out\"),\n",
    "        \"python\", f\"/work/isabl/home/zhouy1/notebook/Unmatched_genome/merge_triple_callers_and_annotate.py\", sample\n",
    "    ]\n",
    "    subprocess.check_output(cmd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "from bsub import bsub"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "bsub.bsub.bsub"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bsub"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>chr1</th>\n",
       "      <th>start1</th>\n",
       "      <th>end1</th>\n",
       "      <th>chr2</th>\n",
       "      <th>start2</th>\n",
       "      <th>end2</th>\n",
       "      <th>SV length</th>\n",
       "      <th>SV type</th>\n",
       "      <th>ID</th>\n",
       "      <th>REF</th>\n",
       "      <th>...</th>\n",
       "      <th>delly_hgsvc_hg38_filtered</th>\n",
       "      <th>DGV_filtered</th>\n",
       "      <th>not_MAPQ1&gt;6</th>\n",
       "      <th>not_MAPQ2&gt;6</th>\n",
       "      <th>not_MATCH1&gt;50</th>\n",
       "      <th>not_MATCH2&gt;50</th>\n",
       "      <th>not_SUPP1&lt;1.75*depth</th>\n",
       "      <th>not_SUPP2&lt;1.75*depth</th>\n",
       "      <th>cat2_filters</th>\n",
       "      <th>cat1_filters</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>4480433</td>\n",
       "      <td>4480434</td>\n",
       "      <td>10</td>\n",
       "      <td>7011114</td>\n",
       "      <td>7011115</td>\n",
       "      <td>NaN</td>\n",
       "      <td>TRA</td>\n",
       "      <td>1_4480434_10_7011115_TRA_++</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>4480433</td>\n",
       "      <td>4480434</td>\n",
       "      <td>10</td>\n",
       "      <td>7011114</td>\n",
       "      <td>7011115</td>\n",
       "      <td>NaN</td>\n",
       "      <td>TRA</td>\n",
       "      <td>1_4480434_10_7011115_TRA_++</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>10877921</td>\n",
       "      <td>10877922</td>\n",
       "      <td>6</td>\n",
       "      <td>134717443</td>\n",
       "      <td>134717444</td>\n",
       "      <td>NaN</td>\n",
       "      <td>TRA</td>\n",
       "      <td>1_10877922_6_134717444_TRA_+-</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>yes</td>\n",
       "      <td>no</td>\n",
       "      <td>yes</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1</td>\n",
       "      <td>11773065</td>\n",
       "      <td>11773066</td>\n",
       "      <td>1</td>\n",
       "      <td>27357132</td>\n",
       "      <td>27357133</td>\n",
       "      <td>15584067.0</td>\n",
       "      <td>INV</td>\n",
       "      <td>1_11773066_1_27357133_INV_--</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>1</td>\n",
       "      <td>11773065</td>\n",
       "      <td>11773066</td>\n",
       "      <td>1</td>\n",
       "      <td>27357132</td>\n",
       "      <td>27357133</td>\n",
       "      <td>15584067.0</td>\n",
       "      <td>INV</td>\n",
       "      <td>1_11773066_1_27357133_INV_--</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6624</th>\n",
       "      <td>X</td>\n",
       "      <td>134030783</td>\n",
       "      <td>134030784</td>\n",
       "      <td>2</td>\n",
       "      <td>34643858</td>\n",
       "      <td>34643859</td>\n",
       "      <td>NaN</td>\n",
       "      <td>TRA</td>\n",
       "      <td>2_34643859_X_134030784_TRA_+-</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>yes</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6625</th>\n",
       "      <td>X</td>\n",
       "      <td>155090038</td>\n",
       "      <td>155090039</td>\n",
       "      <td>6</td>\n",
       "      <td>20168886</td>\n",
       "      <td>20168887</td>\n",
       "      <td>NaN</td>\n",
       "      <td>TRA</td>\n",
       "      <td>6_20168887_X_155090039_TRA_+-</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>yes</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>2.0</td>\n",
       "      <td>3.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6626</th>\n",
       "      <td>Y</td>\n",
       "      <td>21204135</td>\n",
       "      <td>21204136</td>\n",
       "      <td>11</td>\n",
       "      <td>65523675</td>\n",
       "      <td>65523676</td>\n",
       "      <td>NaN</td>\n",
       "      <td>TRA</td>\n",
       "      <td>11_65523676_Y_21204136_TRA_--</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>yes</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6627</th>\n",
       "      <td>Y</td>\n",
       "      <td>21204135</td>\n",
       "      <td>21204136</td>\n",
       "      <td>11</td>\n",
       "      <td>65523675</td>\n",
       "      <td>65523676</td>\n",
       "      <td>NaN</td>\n",
       "      <td>TRA</td>\n",
       "      <td>11_65523676_Y_21204136_TRA_--</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>yes</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6628</th>\n",
       "      <td>Y</td>\n",
       "      <td>21204135</td>\n",
       "      <td>21204136</td>\n",
       "      <td>11</td>\n",
       "      <td>65523675</td>\n",
       "      <td>65523676</td>\n",
       "      <td>NaN</td>\n",
       "      <td>TRA</td>\n",
       "      <td>11_65523676_Y_21204136_TRA_--</td>\n",
       "      <td>N</td>\n",
       "      <td>...</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>yes</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>no</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4391 rows Ã— 124 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     chr1     start1       end1 chr2     start2       end2   SV length  \\\n",
       "0       1    4480433    4480434   10    7011114    7011115         NaN   \n",
       "1       1    4480433    4480434   10    7011114    7011115         NaN   \n",
       "4       1   10877921   10877922    6  134717443  134717444         NaN   \n",
       "5       1   11773065   11773066    1   27357132   27357133  15584067.0   \n",
       "6       1   11773065   11773066    1   27357132   27357133  15584067.0   \n",
       "...   ...        ...        ...  ...        ...        ...         ...   \n",
       "6624    X  134030783  134030784    2   34643858   34643859         NaN   \n",
       "6625    X  155090038  155090039    6   20168886   20168887         NaN   \n",
       "6626    Y   21204135   21204136   11   65523675   65523676         NaN   \n",
       "6627    Y   21204135   21204136   11   65523675   65523676         NaN   \n",
       "6628    Y   21204135   21204136   11   65523675   65523676         NaN   \n",
       "\n",
       "     SV type                             ID REF  ...  \\\n",
       "0        TRA    1_4480434_10_7011115_TRA_++   N  ...   \n",
       "1        TRA    1_4480434_10_7011115_TRA_++   N  ...   \n",
       "4        TRA  1_10877922_6_134717444_TRA_+-   N  ...   \n",
       "5        INV   1_11773066_1_27357133_INV_--   N  ...   \n",
       "6        INV   1_11773066_1_27357133_INV_--   N  ...   \n",
       "...      ...                            ...  ..  ...   \n",
       "6624     TRA  2_34643859_X_134030784_TRA_+-   N  ...   \n",
       "6625     TRA  6_20168887_X_155090039_TRA_+-   N  ...   \n",
       "6626     TRA  11_65523676_Y_21204136_TRA_--   N  ...   \n",
       "6627     TRA  11_65523676_Y_21204136_TRA_--   N  ...   \n",
       "6628     TRA  11_65523676_Y_21204136_TRA_--   N  ...   \n",
       "\n",
       "     delly_hgsvc_hg38_filtered DGV_filtered  not_MAPQ1>6 not_MAPQ2>6  \\\n",
       "0                           no           no           no          no   \n",
       "1                           no           no           no          no   \n",
       "4                           no           no          yes          no   \n",
       "5                           no           no           no          no   \n",
       "6                           no           no           no          no   \n",
       "...                        ...          ...          ...         ...   \n",
       "6624                        no           no           no         yes   \n",
       "6625                        no           no          yes          no   \n",
       "6626                        no           no           no         yes   \n",
       "6627                        no           no           no         yes   \n",
       "6628                        no           no           no         yes   \n",
       "\n",
       "     not_MATCH1>50 not_MATCH2>50 not_SUPP1<1.75*depth not_SUPP2<1.75*depth  \\\n",
       "0               no            no                   no                   no   \n",
       "1               no            no                   no                   no   \n",
       "4              yes            no                   no                   no   \n",
       "5               no            no                   no                   no   \n",
       "6               no            no                   no                   no   \n",
       "...            ...           ...                  ...                  ...   \n",
       "6624            no            no                   no                   no   \n",
       "6625            no            no                   no                   no   \n",
       "6626            no            no                   no                   no   \n",
       "6627            no            no                   no                   no   \n",
       "6628            no            no                   no                   no   \n",
       "\n",
       "     cat2_filters  cat1_filters  \n",
       "0             1.0           1.0  \n",
       "1             1.0           1.0  \n",
       "4             2.0           2.0  \n",
       "5             0.0           1.0  \n",
       "6             0.0           1.0  \n",
       "...           ...           ...  \n",
       "6624          1.0           1.0  \n",
       "6625          2.0           3.0  \n",
       "6626          1.0           1.0  \n",
       "6627          1.0           1.0  \n",
       "6628          1.0           1.0  \n",
       "\n",
       "[4391 rows x 124 columns]"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "sample = \"IID_H158671_T01_01_WG01\"\n",
    "tsv = f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/uk_all_sv_1213/{sample}/{sample}_vs_I-H-108298-N1-1-D1-1.annotated.flagged.output.tsv\"\n",
    "df = pd.read_csv(tsv, sep='\\t')\n",
    "filtered = df[df['cat1_filters']!=0]\n",
    "filtered"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/merged/IID_H158671_T01_01_WG01.merged.vcf\", 'r') as f:\n",
    "    content = f.readlines()\n",
    "    sg = [i for i in content if \"Callers=svaba,gridss\" in i]\n",
    "    sg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158671_T01_01_WG01\n",
      "IID_H158670_T01_01_WG01\n",
      "IID_H158669_T01_01_WG01\n",
      "IID_H158668_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,61) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158667_T01_01_WG01\n",
      "IID_H158666_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,38,63) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158665_T01_01_WG01\n",
      "IID_H158664_T01_01_WG01\n",
      "IID_H158663_T01_01_WG01\n",
      "IID_H158662_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,38,55,56,63) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158651_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,35,61) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158650_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,55,56) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158649_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,35,38,61,63,78) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158648_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,63) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158647_T01_01_WG01\n",
      "IID_H158646_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,35,55,56,61) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158645_T01_01_WG01\n",
      "IID_H158644_T01_01_WG01\n",
      "IID_H158643_T01_01_WG01\n",
      "IID_H158642_T01_01_WG01\n",
      "IID_H158641_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,35,38,61,63) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158640_T01_01_WG01\n",
      "IID_H158639_T01_01_WG01\n",
      "IID_H158638_T01_01_WG01\n",
      "IID_H158637_T01_01_WG01\n",
      "IID_H158636_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,38,55,56,61,63) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158635_T01_01_WG01\n",
      "IID_H158634_T01_01_WG01\n",
      "IID_H158633_T01_01_WG01\n",
      "IID_H158632_T01_01_WG01\n",
      "IID_H158631_T01_01_WG01\n",
      "IID_H158630_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,35,38,63) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158629_T01_01_WG01\n",
      "IID_H158628_T01_01_WG01\n",
      "IID_H158627_T01_01_WG01\n",
      "IID_H158626_T01_01_WG01\n",
      "IID_H158625_T01_01_WG01\n",
      "IID_H158624_T01_01_WG01\n",
      "IID_H158623_T01_01_WG01\n",
      "IID_H158622_T01_01_WG01\n",
      "IID_H158621_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,38,61,63) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158620_T01_01_WG01\n",
      "IID_H158619_T01_01_WG01\n",
      "IID_H158618_T01_01_WG01\n",
      "IID_H158617_T01_01_WG01\n",
      "IID_H158616_T01_01_WG01\n",
      "IID_H158615_T01_01_WG01\n",
      "IID_H158614_T01_01_WG01\n",
      "IID_H158613_T01_01_WG01\n",
      "IID_H158612_T01_01_WG01\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1,38,61,63,78) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158611_T01_01_WG01\n",
      "IID_H158610_T01_01_WG01\n",
      "IID_H158609_T01_01_WG01\n",
      "IID_H158608_T01_01_WG01\n",
      "IID_H158607_T01_01_WG01\n",
      "IID_H158606_T01_01_WG01\n",
      "IID_H158605_T01_01_WG01\n",
      "IID_H158604_T01_01_WG01\n"
     ]
    }
   ],
   "source": [
    "# Integrate CNV into SV\n",
    "sample = \"IID_H158606_T01_01_WG01\"\n",
    "\n",
    "with open(\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/svaba/samples.tsv\",'r') as f:\n",
    "    samples = f.readlines()\n",
    "    samples = [i.strip('\\n') for i in samples]\n",
    "\n",
    "for sample in samples:\n",
    "    print(sample)\n",
    "    # sv = pd.read_csv(\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/uk_all_sv_1213/IID_H158604_T01_01_WG01/IID_H158604_T01_01_WG01_vs_I-H-108298-N1-1-D1-1.annotated.flagged.output.tsv\", sep='\\t')\n",
    "    sv = pd.read_csv(f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/annotated/{sample}/{sample}.merged.annotated.tsv\", sep='\\t')\n",
    "    sv = sv[sv['AnnotSV type']=='full']\n",
    "    sv['SV chrom'] = sv['SV chrom'].astype(str)\n",
    "    sv = sv[(sv['SV type']=='DEL') | (sv['SV type']=='DUP')]\n",
    "    for index, row in sv.iterrows():\n",
    "        INFO = row['INFO']\n",
    "        callers = INFO.split(';')[1].replace('Callers=', '').split(',')\n",
    "        for i in callers:\n",
    "            sv.loc[index, 'num_callers'] = len(callers)\n",
    "            sv.loc[index, i] = 1\n",
    "    sv = sv[(sv['num_callers']>=2) & (sv['brass'] == 1)]\n",
    "\n",
    "    cnv = pd.read_csv(f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/uk_all_0115/{sample}/{sample}.subclone.final.cns\",sep='\\t')\n",
    "    cnv['chromosome'] = cnv['chromosome'].astype(str)\n",
    "    cnv = cnv[(cnv['cn1']!=1) | (cnv['cn2']!=1)]\n",
    "\n",
    "    window = 1000\n",
    "\n",
    "    for index, row in sv.iterrows():\n",
    "        sv_chr = row['SV chrom']\n",
    "        sv_start = row['SV start']\n",
    "        sv_end = row['SV end']\n",
    "        sv_length = row['SV length']\n",
    "        for i, r in cnv.iterrows():\n",
    "            cnv_chr = r['chromosome']\n",
    "            cnv_start = r['start']\n",
    "            cnv_end = r['end']\n",
    "            if sv_chr == cnv_chr:\n",
    "                if abs(sv_start - cnv_start)<window:\n",
    "                    overlap = (min(cnv_end, sv_end) - max(cnv_start, sv_start))/sv_length\n",
    "                    sv.loc[index, 'cnv'] = f\"{cnv_chr}:{cnv_start}-{cnv_end} {overlap}\"\n",
    "    sv.to_csv(f\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/integrate_cnv_in_sv/{sample}.tsv\",sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>AnnotSV ID</th>\n",
       "      <th>SV chrom</th>\n",
       "      <th>SV start</th>\n",
       "      <th>SV end</th>\n",
       "      <th>SV length</th>\n",
       "      <th>SV type</th>\n",
       "      <th>ID</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>FILTER</th>\n",
       "      <th>INFO</th>\n",
       "      <th>FORMAT</th>\n",
       "      <th>I-H-108298-N1-1-D1-1</th>\n",
       "      <th>IID_H158606_T01_01_WG01</th>\n",
       "      <th>AnnotSV type</th>\n",
       "      <th>Gene name</th>\n",
       "      <th>NM</th>\n",
       "      <th>CDS length</th>\n",
       "      <th>tx length</th>\n",
       "      <th>location</th>\n",
       "      <th>intersectStart</th>\n",
       "      <th>intersectEnd</th>\n",
       "      <th>DGV_GAIN_IDs</th>\n",
       "      <th>DGV_GAIN_n_samples_with_SV</th>\n",
       "      <th>DGV_GAIN_n_samples_tested</th>\n",
       "      <th>DGV_GAIN_Frequency</th>\n",
       "      <th>DGV_LOSS_IDs</th>\n",
       "      <th>DGV_LOSS_n_samples_with_SV</th>\n",
       "      <th>DGV_LOSS_n_samples_tested</th>\n",
       "      <th>DGV_LOSS_Frequency</th>\n",
       "      <th>GD_ID</th>\n",
       "      <th>GD_AN</th>\n",
       "      <th>GD_N_HET</th>\n",
       "      <th>GD_N_HOMALT</th>\n",
       "      <th>GD_AF</th>\n",
       "      <th>GD_POPMAX_AF</th>\n",
       "      <th>GD_ID_others</th>\n",
       "      <th>DDD_SV</th>\n",
       "      <th>DDD_DUP_n_samples_with_SV</th>\n",
       "      <th>DDD_DUP_Frequency</th>\n",
       "      <th>DDD_DEL_n_samples_with_SV</th>\n",
       "      <th>DDD_DEL_Frequency</th>\n",
       "      <th>1000g_event</th>\n",
       "      <th>1000g_AF</th>\n",
       "      <th>1000g_max_AF</th>\n",
       "      <th>IMH_ID</th>\n",
       "      <th>IMH_AF</th>\n",
       "      <th>IMH_ID_others</th>\n",
       "      <th>promoters</th>\n",
       "      <th>dbVar_event</th>\n",
       "      <th>dbVar_variant</th>\n",
       "      <th>dbVar_status</th>\n",
       "      <th>TADcoordinates</th>\n",
       "      <th>ENCODEexperiments</th>\n",
       "      <th>GCcontent_left</th>\n",
       "      <th>GCcontent_right</th>\n",
       "      <th>Repeats_coord_left</th>\n",
       "      <th>Repeats_type_left</th>\n",
       "      <th>Repeats_coord_right</th>\n",
       "      <th>Repeats_type_right</th>\n",
       "      <th>ACMG</th>\n",
       "      <th>HI_CGscore</th>\n",
       "      <th>TriS_CGscore</th>\n",
       "      <th>DDD_status</th>\n",
       "      <th>DDD_mode</th>\n",
       "      <th>DDD_consequence</th>\n",
       "      <th>DDD_disease</th>\n",
       "      <th>DDD_pmids</th>\n",
       "      <th>HI_DDDpercent</th>\n",
       "      <th>synZ_ExAC</th>\n",
       "      <th>misZ_ExAC</th>\n",
       "      <th>pLI_ExAC</th>\n",
       "      <th>delZ_ExAC</th>\n",
       "      <th>dupZ_ExAC</th>\n",
       "      <th>cnvZ_ExAC</th>\n",
       "      <th>morbidGenes</th>\n",
       "      <th>morbidGenesCandidates</th>\n",
       "      <th>Mim Number</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Inheritance</th>\n",
       "      <th>AnnotSV ranking</th>\n",
       "      <th>num_callers</th>\n",
       "      <th>gridss</th>\n",
       "      <th>svaba</th>\n",
       "      <th>brass</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>15532</th>\n",
       "      <td>10_28821260_28848026_DEL</td>\n",
       "      <td>10</td>\n",
       "      <td>28821260</td>\n",
       "      <td>28848026</td>\n",
       "      <td>26766.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[10:28848026[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=28848026;Callers=brass,svaba,gridss;CHR2=1...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>48:0:0.578947:0.571429:0.571429:51:-71.56:0:24...</td>\n",
       "      <td>full</td>\n",
       "      <td>WAC/WAC-AS1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>10:8738346-108102195_INV;10:15431941-70315351_INV</td>\n",
       "      <td>WAC/WAC-AS1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.650000</td>\n",
       "      <td>0.415000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10:28848065-28848094/10:28848118-28848410</td>\n",
       "      <td>(TG)n/L2a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25374</th>\n",
       "      <td>11_68782074_68787123_DEL</td>\n",
       "      <td>11</td>\n",
       "      <td>68782074</td>\n",
       "      <td>68787123</td>\n",
       "      <td>5049.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[11:68787123[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=68787123;Callers=brass,svaba,gridss;CHR2=1...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>57:20:0.480519:0.5:0.45679:64:-89.38:0:31:88.4...</td>\n",
       "      <td>full</td>\n",
       "      <td>MRGPRF-AS1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>11:1642919-71249653_INV;11:49233127-89368291_I...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.515000</td>\n",
       "      <td>0.610000</td>\n",
       "      <td>11:68782152-68782459</td>\n",
       "      <td>MLT1A0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25495</th>\n",
       "      <td>11_77932829_77939621_DUP</td>\n",
       "      <td>11</td>\n",
       "      <td>77932829</td>\n",
       "      <td>77939621</td>\n",
       "      <td>6792.0</td>\n",
       "      <td>DUP</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>]11:77939621]N</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=77939621;Callers=brass,svaba,gridss;CHR2=1...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>21:7:1.19481:1.22667:1.17949:0:-73.68:21:10:72...</td>\n",
       "      <td>full</td>\n",
       "      <td>GAB2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>11:77550802-78000858_DUP</td>\n",
       "      <td>0.000059</td>\n",
       "      <td>11:49233127-89368291_INV;11:49425031-89029881_...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.575000</td>\n",
       "      <td>0.470000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>38.04</td>\n",
       "      <td>0.355310</td>\n",
       "      <td>-1.040969</td>\n",
       "      <td>0.997629</td>\n",
       "      <td>0.920190</td>\n",
       "      <td>0.780302</td>\n",
       "      <td>1.033942</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35276</th>\n",
       "      <td>13_83435494_83439889_DEL</td>\n",
       "      <td>13</td>\n",
       "      <td>83435494</td>\n",
       "      <td>83439889</td>\n",
       "      <td>4395.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[13:83439889[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=83439889;Callers=brass,svaba,gridss;CHR2=1...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>48:25:0.493506:0.487179:0.520548:66:-113.3:0:3...</td>\n",
       "      <td>full</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>gnomAD_v2_DEL_13_161665;gnomAD_v2_DEL_13_161668</td>\n",
       "      <td>21476;18384</td>\n",
       "      <td>1;2</td>\n",
       "      <td>0;0</td>\n",
       "      <td>0.00010900000052060932</td>\n",
       "      <td>0.000720</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>13:83249177-83483973_DEL</td>\n",
       "      <td>0.000059</td>\n",
       "      <td>13:32957269-100763564_INV;13:33672321-10440911...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.220000</td>\n",
       "      <td>0.280000</td>\n",
       "      <td>13:83435564-83435614</td>\n",
       "      <td>A-rich</td>\n",
       "      <td>13:83439503-83439802/13:83439899-83439941</td>\n",
       "      <td>AluSg/(TATG)n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38617</th>\n",
       "      <td>14_106330472_107113741_DEL</td>\n",
       "      <td>14</td>\n",
       "      <td>106330472</td>\n",
       "      <td>107113741</td>\n",
       "      <td>783269.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[14:107113741[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=107113741;Callers=brass,gridss;CHR2=14;SVT...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>13:0:0.157895:0.16:0.218182:15:61:249.99:23:1:...</td>\n",
       "      <td>full</td>\n",
       "      <td>ADAM6/FAM30A/LINC00221/LINC00226</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>gssvG12029,gssvG12030</td>\n",
       "      <td>29</td>\n",
       "      <td>13250</td>\n",
       "      <td>0.002189</td>\n",
       "      <td>gssvL37119,gssvL37121,gssvL37120</td>\n",
       "      <td>539</td>\n",
       "      <td>15951</td>\n",
       "      <td>0.033791</td>\n",
       "      <td>gnomAD_v2_DEL_14_170979;gnomAD_v2_DEL_14_17098...</td>\n",
       "      <td>21388;21474;21422;21476;21474;21476;21422;2147...</td>\n",
       "      <td>11;4;6;4;5;1;1;2;12;1;1;1;4;4;12;10;7</td>\n",
       "      <td>0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0</td>\n",
       "      <td>0.0005590000073425472</td>\n",
       "      <td>0.006734</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>14:106326045-107178888_DEL;14:106329421-106926...</td>\n",
       "      <td>0.011460</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ADAM6/FAM30A/LINC00221/LINC00226</td>\n",
       "      <td>copy_number_gain;copy_number_loss</td>\n",
       "      <td>nssv1602987;nssv3396033;nssv576460;nssv582489</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.575000</td>\n",
       "      <td>0.535000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>2.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38633</th>\n",
       "      <td>14_106357063_106518398_DEL</td>\n",
       "      <td>14</td>\n",
       "      <td>106357063</td>\n",
       "      <td>106518398</td>\n",
       "      <td>161335.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[14:106518398[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=106518398;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>14:0:0.171053:0.178082:0.419355:16:-49.51:16:4...</td>\n",
       "      <td>full</td>\n",
       "      <td>ADAM6/FAM30A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>gssvG12029,gssvG12033,gssvG12051,gssvG12043</td>\n",
       "      <td>33</td>\n",
       "      <td>13284</td>\n",
       "      <td>0.002484</td>\n",
       "      <td>gssvL37119,gssvL37122,gssvL37121,gssvL37131,gs...</td>\n",
       "      <td>563</td>\n",
       "      <td>15858</td>\n",
       "      <td>0.035503</td>\n",
       "      <td>gnomAD_v2_DEL_14_170862;gnomAD_v2_DEL_14_17088...</td>\n",
       "      <td>21476;21448;21474;21474;21474;21474;21446;2144...</td>\n",
       "      <td>31;9;6;3;3;1;2;12;8;11;4;6;4;6;5;1;136;4;1;1;4...</td>\n",
       "      <td>0;1;0;0;0;2;0;1;0;0;0;0;0;0;0;0;11;0;0;0;0;0;0...</td>\n",
       "      <td>0.00735800014808774</td>\n",
       "      <td>0.034193</td>\n",
       "      <td>gnomAD_v2_DUP_14_40112;gnomAD_v2_MCNV_14_755;g...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>14:106326045-107178888_DEL;14:106329399-106471...</td>\n",
       "      <td>0.023430</td>\n",
       "      <td>14:106350702-106552313_INV;14:106350727-106725...</td>\n",
       "      <td>ADAM6/FAM30A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.645000</td>\n",
       "      <td>0.580000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38641</th>\n",
       "      <td>14_106551444_107034728_DEL</td>\n",
       "      <td>14</td>\n",
       "      <td>106551444</td>\n",
       "      <td>107034728</td>\n",
       "      <td>483284.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[14:107034728[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=107034728;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>49:15:0.131579:0.12987:0.25:52:-92.42:0:31:92....</td>\n",
       "      <td>full</td>\n",
       "      <td>LINC00221/LINC00226</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>gssvG12029,gssvG12030</td>\n",
       "      <td>29</td>\n",
       "      <td>13250</td>\n",
       "      <td>0.002189</td>\n",
       "      <td>gssvL37119,gssvL37121,gssvL37124,gssvL37120,gs...</td>\n",
       "      <td>875</td>\n",
       "      <td>25389</td>\n",
       "      <td>0.034464</td>\n",
       "      <td>gnomAD_v2_DEL_14_170979;gnomAD_v2_DEL_14_17098...</td>\n",
       "      <td>21388;21474;21422;21476;21474;21476;21422;2147...</td>\n",
       "      <td>11;4;6;4;5;1;1;2;12;1;1;1;4;4;12;10;7;1</td>\n",
       "      <td>0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0</td>\n",
       "      <td>0.0005590000073425472</td>\n",
       "      <td>0.006734</td>\n",
       "      <td>gnomAD_v2_DUP_14_40201</td>\n",
       "      <td>14:106544103-106916718</td>\n",
       "      <td>18.0</td>\n",
       "      <td>0.021302</td>\n",
       "      <td>24.0</td>\n",
       "      <td>0.028402</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>14:106326045-107178888_DEL;14:106329421-106926...</td>\n",
       "      <td>0.011460</td>\n",
       "      <td>14:106590902-107013925_DUP;14:106624500-107125...</td>\n",
       "      <td>LINC00221/LINC00226</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.505000</td>\n",
       "      <td>0.590000</td>\n",
       "      <td>14:106551142-106551601</td>\n",
       "      <td>LTR16B2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53658</th>\n",
       "      <td>2_89203502_89475818_DEL</td>\n",
       "      <td>2</td>\n",
       "      <td>89203502</td>\n",
       "      <td>89475818</td>\n",
       "      <td>272316.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[2:89475818[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=89475818;Callers=brass,svaba,gridss;CHR2=2...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>23:10:0.402597:0.402597:0.815789:31:-62.72:0:2...</td>\n",
       "      <td>full</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>gssvG20566,gssvG20563,gssvG20577,gssvG20562</td>\n",
       "      <td>242</td>\n",
       "      <td>20739</td>\n",
       "      <td>0.011669</td>\n",
       "      <td>gssvL64379,gssvL64384,gssvL64387,gssvL64389</td>\n",
       "      <td>944</td>\n",
       "      <td>21959</td>\n",
       "      <td>0.042989</td>\n",
       "      <td>gnomAD_v2_DEL_2_23514;gnomAD_v2_DEL_2_23520;gn...</td>\n",
       "      <td>1064;21458;21458;21408;21470;21372;21474;21468...</td>\n",
       "      <td>40;3;3;1;2;1;3;2;3;1;1;1;3</td>\n",
       "      <td>4;0;0;0;1;0;0;0;0;0;0;0;0</td>\n",
       "      <td>0.04511300101876259</td>\n",
       "      <td>0.107623</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2:89198438-89590982</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.003550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>2:89132234-89399420_DEL;2:89132244-89495574_DE...</td>\n",
       "      <td>0.021740</td>\n",
       "      <td>2:26280500-162970618_INV;2:33078871-153818960_...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.505000</td>\n",
       "      <td>0.505000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54969</th>\n",
       "      <td>2_203281243_203286840_DUP</td>\n",
       "      <td>2</td>\n",
       "      <td>203281243</td>\n",
       "      <td>203286840</td>\n",
       "      <td>5597.0</td>\n",
       "      <td>DUP</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>]2:203286840]N</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=203286840;Callers=brass,svaba;CHR2=2;SVTYP...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::..:.:.:.</td>\n",
       "      <td>60:0:1.42857:1.46667:1.46667:0:-193.1:61:4:185...</td>\n",
       "      <td>full</td>\n",
       "      <td>BMPR2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>gssvL68773</td>\n",
       "      <td>2</td>\n",
       "      <td>12365</td>\n",
       "      <td>0.000162</td>\n",
       "      <td>gnomAD_v2_DUP_2_7194</td>\n",
       "      <td>20400</td>\n",
       "      <td>10</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0004900000058114529</td>\n",
       "      <td>0.000940</td>\n",
       "      <td>gnomAD_v2_DEL_2_30788;gnomAD_v2_INV_2_117</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>2:203281078-203287109_DUP</td>\n",
       "      <td>0.000178</td>\n",
       "      <td>2:37574354-210299893_INV;2:122235797-235423496...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.490000</td>\n",
       "      <td>0.560000</td>\n",
       "      <td>2:203280989-203281150/2:203281150-203281442</td>\n",
       "      <td>AluJr/AluSc</td>\n",
       "      <td>2:203286646-203286929</td>\n",
       "      <td>AluSc5</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.48</td>\n",
       "      <td>0.303008</td>\n",
       "      <td>1.476221</td>\n",
       "      <td>0.999600</td>\n",
       "      <td>0.805782</td>\n",
       "      <td>0.357465</td>\n",
       "      <td>0.654177</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>2.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66679</th>\n",
       "      <td>6_25909421_25913762_DUP</td>\n",
       "      <td>6</td>\n",
       "      <td>25909421</td>\n",
       "      <td>25913762</td>\n",
       "      <td>4341.0</td>\n",
       "      <td>DUP</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>]6:25913762]N</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=25913762;Callers=brass,gridss;CHR2=6;SVTYP...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>8:0:1.09091:1.09091:1.15068:0:70:254.63:13:6:0...</td>\n",
       "      <td>full</td>\n",
       "      <td>SLC17A2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>6:18151823-168174514_INV;6:20232222-26808015_I...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.425000</td>\n",
       "      <td>0.370000</td>\n",
       "      <td>6:25909396-25909703</td>\n",
       "      <td>AluSx</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>64.34</td>\n",
       "      <td>0.086779</td>\n",
       "      <td>0.715960</td>\n",
       "      <td>0.013546</td>\n",
       "      <td>-1.282744</td>\n",
       "      <td>0.393160</td>\n",
       "      <td>-0.311826</td>\n",
       "      <td>yes</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>2.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73561</th>\n",
       "      <td>8_143850649_143929135_DUP</td>\n",
       "      <td>8</td>\n",
       "      <td>143850649</td>\n",
       "      <td>143929135</td>\n",
       "      <td>78486.0</td>\n",
       "      <td>DUP</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>]8:143929135]N</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=143929135;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>25:0:1.57143:1.63514:1.49383:0:-91.25:0:32:90....</td>\n",
       "      <td>full</td>\n",
       "      <td>GML/LY6D/LYNX1/LYNX1-SLURP2/SLURP2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>gssvL127967,gssvL127968</td>\n",
       "      <td>3</td>\n",
       "      <td>12408</td>\n",
       "      <td>0.000242</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>gnomAD_v2_INV_8_448</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GML/LY6D/LYNX1/LYNX1-SLURP2/SLURP2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8:143840001-144400000</td>\n",
       "      <td>ENCFF531PAV:ENCFF931RKD</td>\n",
       "      <td>0.660000</td>\n",
       "      <td>0.320000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8:143928389-143929557</td>\n",
       "      <td>L1M3f</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75156</th>\n",
       "      <td>9_121152838_121295287_DEL</td>\n",
       "      <td>9</td>\n",
       "      <td>121152838</td>\n",
       "      <td>121295287</td>\n",
       "      <td>142449.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[9:121295287[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=121295287;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>41:0:0.592105:0.592105:0.6:44:-122.1:40:0:122....</td>\n",
       "      <td>full</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>9:120401415-122017859_DUP;9:120865318-12138613...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9:121160001-122240000;9:121200001-122160000;9:...</td>\n",
       "      <td>ENCFF306YQN:ENCFF373MCF:ENCFF444XGA:ENCFF558RG...</td>\n",
       "      <td>0.175000</td>\n",
       "      <td>0.420000</td>\n",
       "      <td>9:121152631-121152796/9:121152878-121152940</td>\n",
       "      <td>(TA)n/(TA)n</td>\n",
       "      <td>9:121295249-121295398</td>\n",
       "      <td>MIR3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78325</th>\n",
       "      <td>X_68458648_68461623_DUP</td>\n",
       "      <td>X</td>\n",
       "      <td>68458648</td>\n",
       "      <td>68461623</td>\n",
       "      <td>2975.0</td>\n",
       "      <td>DUP</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>]X:68461623]N</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=68461623;Callers=brass,svaba,gridss;CHR2=X...</td>\n",
       "      <td>brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::..:.:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>44:28:1.42105:1.5:1.35:0:-146.9:53:0:146.6,0,1...</td>\n",
       "      <td>full</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>gnomAD_v2_DUP_X_55241</td>\n",
       "      <td>21430</td>\n",
       "      <td>34</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0015869999770075083</td>\n",
       "      <td>0.004090</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>X:68458604-68461670_DUP</td>\n",
       "      <td>0.000475</td>\n",
       "      <td>X:46769177-84009828_INV</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.540000</td>\n",
       "      <td>0.530000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                       AnnotSV ID SV chrom   SV start     SV end  SV length  \\\n",
       "15532    10_28821260_28848026_DEL       10   28821260   28848026    26766.0   \n",
       "25374    11_68782074_68787123_DEL       11   68782074   68787123     5049.0   \n",
       "25495    11_77932829_77939621_DUP       11   77932829   77939621     6792.0   \n",
       "35276    13_83435494_83439889_DEL       13   83435494   83439889     4395.0   \n",
       "38617  14_106330472_107113741_DEL       14  106330472  107113741   783269.0   \n",
       "38633  14_106357063_106518398_DEL       14  106357063  106518398   161335.0   \n",
       "38641  14_106551444_107034728_DEL       14  106551444  107034728   483284.0   \n",
       "53658     2_89203502_89475818_DEL        2   89203502   89475818   272316.0   \n",
       "54969   2_203281243_203286840_DUP        2  203281243  203286840     5597.0   \n",
       "66679     6_25909421_25913762_DUP        6   25909421   25913762     4341.0   \n",
       "73561   8_143850649_143929135_DUP        8  143850649  143929135    78486.0   \n",
       "75156   9_121152838_121295287_DEL        9  121152838  121295287   142449.0   \n",
       "78325     X_68458648_68461623_DUP        X   68458648   68461623     2975.0   \n",
       "\n",
       "      SV type ID REF              ALT QUAL FILTER  \\\n",
       "15532     DEL  .   N   N[10:28848026[    .   PASS   \n",
       "25374     DEL  .   N   N[11:68787123[    .   PASS   \n",
       "25495     DUP  .   N   ]11:77939621]N    .   PASS   \n",
       "35276     DEL  .   N   N[13:83439889[    .   PASS   \n",
       "38617     DEL  .   N  N[14:107113741[    .   PASS   \n",
       "38633     DEL  .   N  N[14:106518398[    .   PASS   \n",
       "38641     DEL  .   N  N[14:107034728[    .   PASS   \n",
       "53658     DEL  .   N    N[2:89475818[    .   PASS   \n",
       "54969     DUP  .   N   ]2:203286840]N    .   PASS   \n",
       "66679     DUP  .   N    ]6:25913762]N    .   PASS   \n",
       "73561     DUP  .   N   ]8:143929135]N    .   PASS   \n",
       "75156     DEL  .   N   N[9:121295287[    .   PASS   \n",
       "78325     DUP  .   N    ]X:68461623]N    .   PASS   \n",
       "\n",
       "                                                    INFO  \\\n",
       "15532  END=28848026;Callers=brass,svaba,gridss;CHR2=1...   \n",
       "25374  END=68787123;Callers=brass,svaba,gridss;CHR2=1...   \n",
       "25495  END=77939621;Callers=brass,svaba,gridss;CHR2=1...   \n",
       "35276  END=83439889;Callers=brass,svaba,gridss;CHR2=1...   \n",
       "38617  END=107113741;Callers=brass,gridss;CHR2=14;SVT...   \n",
       "38633  END=106518398;Callers=brass,svaba,gridss;CHR2=...   \n",
       "38641  END=107034728;Callers=brass,svaba,gridss;CHR2=...   \n",
       "53658  END=89475818;Callers=brass,svaba,gridss;CHR2=2...   \n",
       "54969  END=203286840;Callers=brass,svaba;CHR2=2;SVTYP...   \n",
       "66679  END=25913762;Callers=brass,gridss;CHR2=6;SVTYP...   \n",
       "73561  END=143929135;Callers=brass,svaba,gridss;CHR2=...   \n",
       "75156  END=121295287;Callers=brass,svaba,gridss;CHR2=...   \n",
       "78325  END=68461623;Callers=brass,svaba,gridss;CHR2=X...   \n",
       "\n",
       "                                                  FORMAT  \\\n",
       "15532  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "25374  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "25495  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "35276  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "38617  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "38633  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "38641  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "53658  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "54969  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "66679  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "73561  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "75156  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "78325  brass_IID_H158606_T01_01_WG01_PS:brass_IID_H15...   \n",
       "\n",
       "                                    I-H-108298-N1-1-D1-1  \\\n",
       "15532  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "25374  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "25495  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "35276  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "38617  0:0:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "38633  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "38641  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "53658  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "54969                        0:0:.:.:.:.:.:.:.::..:.:.:.   \n",
       "66679  0:0:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "73561  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "75156  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "78325  0:0:.:.:.:.:.:.:.::..:.:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "\n",
       "                                 IID_H158606_T01_01_WG01 AnnotSV type  \\\n",
       "15532  48:0:0.578947:0.571429:0.571429:51:-71.56:0:24...         full   \n",
       "25374  57:20:0.480519:0.5:0.45679:64:-89.38:0:31:88.4...         full   \n",
       "25495  21:7:1.19481:1.22667:1.17949:0:-73.68:21:10:72...         full   \n",
       "35276  48:25:0.493506:0.487179:0.520548:66:-113.3:0:3...         full   \n",
       "38617  13:0:0.157895:0.16:0.218182:15:61:249.99:23:1:...         full   \n",
       "38633  14:0:0.171053:0.178082:0.419355:16:-49.51:16:4...         full   \n",
       "38641  49:15:0.131579:0.12987:0.25:52:-92.42:0:31:92....         full   \n",
       "53658  23:10:0.402597:0.402597:0.815789:31:-62.72:0:2...         full   \n",
       "54969  60:0:1.42857:1.46667:1.46667:0:-193.1:61:4:185...         full   \n",
       "66679  8:0:1.09091:1.09091:1.15068:0:70:254.63:13:6:0...         full   \n",
       "73561  25:0:1.57143:1.63514:1.49383:0:-91.25:0:32:90....         full   \n",
       "75156  41:0:0.592105:0.592105:0.6:44:-122.1:40:0:122....         full   \n",
       "78325  44:28:1.42105:1.5:1.35:0:-146.9:53:0:146.6,0,1...         full   \n",
       "\n",
       "                                Gene name   NM  CDS length  tx length  \\\n",
       "15532                         WAC/WAC-AS1  NaN         NaN        NaN   \n",
       "25374                          MRGPRF-AS1  NaN         NaN        NaN   \n",
       "25495                                GAB2  NaN         NaN        NaN   \n",
       "35276                                 NaN  NaN         NaN        NaN   \n",
       "38617    ADAM6/FAM30A/LINC00221/LINC00226  NaN         NaN        NaN   \n",
       "38633                        ADAM6/FAM30A  NaN         NaN        NaN   \n",
       "38641                 LINC00221/LINC00226  NaN         NaN        NaN   \n",
       "53658                                 NaN  NaN         NaN        NaN   \n",
       "54969                               BMPR2  NaN         NaN        NaN   \n",
       "66679                             SLC17A2  NaN         NaN        NaN   \n",
       "73561  GML/LY6D/LYNX1/LYNX1-SLURP2/SLURP2  NaN         NaN        NaN   \n",
       "75156                                 NaN  NaN         NaN        NaN   \n",
       "78325                                 NaN  NaN         NaN        NaN   \n",
       "\n",
       "      location  intersectStart  intersectEnd  \\\n",
       "15532      NaN             NaN           NaN   \n",
       "25374      NaN             NaN           NaN   \n",
       "25495      NaN             NaN           NaN   \n",
       "35276      NaN             NaN           NaN   \n",
       "38617      NaN             NaN           NaN   \n",
       "38633      NaN             NaN           NaN   \n",
       "38641      NaN             NaN           NaN   \n",
       "53658      NaN             NaN           NaN   \n",
       "54969      NaN             NaN           NaN   \n",
       "66679      NaN             NaN           NaN   \n",
       "73561      NaN             NaN           NaN   \n",
       "75156      NaN             NaN           NaN   \n",
       "78325      NaN             NaN           NaN   \n",
       "\n",
       "                                      DGV_GAIN_IDs  \\\n",
       "15532                                          NaN   \n",
       "25374                                          NaN   \n",
       "25495                                          NaN   \n",
       "35276                                          NaN   \n",
       "38617                        gssvG12029,gssvG12030   \n",
       "38633  gssvG12029,gssvG12033,gssvG12051,gssvG12043   \n",
       "38641                        gssvG12029,gssvG12030   \n",
       "53658  gssvG20566,gssvG20563,gssvG20577,gssvG20562   \n",
       "54969                                          NaN   \n",
       "66679                                          NaN   \n",
       "73561                                          NaN   \n",
       "75156                                          NaN   \n",
       "78325                                          NaN   \n",
       "\n",
       "       DGV_GAIN_n_samples_with_SV  DGV_GAIN_n_samples_tested  \\\n",
       "15532                           0                          0   \n",
       "25374                           0                          0   \n",
       "25495                           0                          0   \n",
       "35276                           0                          0   \n",
       "38617                          29                      13250   \n",
       "38633                          33                      13284   \n",
       "38641                          29                      13250   \n",
       "53658                         242                      20739   \n",
       "54969                           0                          0   \n",
       "66679                           0                          0   \n",
       "73561                           0                          0   \n",
       "75156                           0                          0   \n",
       "78325                           0                          0   \n",
       "\n",
       "       DGV_GAIN_Frequency                                       DGV_LOSS_IDs  \\\n",
       "15532           -1.000000                                                NaN   \n",
       "25374           -1.000000                                                NaN   \n",
       "25495           -1.000000                                                NaN   \n",
       "35276           -1.000000                                                NaN   \n",
       "38617            0.002189                   gssvL37119,gssvL37121,gssvL37120   \n",
       "38633            0.002484  gssvL37119,gssvL37122,gssvL37121,gssvL37131,gs...   \n",
       "38641            0.002189  gssvL37119,gssvL37121,gssvL37124,gssvL37120,gs...   \n",
       "53658            0.011669        gssvL64379,gssvL64384,gssvL64387,gssvL64389   \n",
       "54969           -1.000000                                         gssvL68773   \n",
       "66679           -1.000000                                                NaN   \n",
       "73561           -1.000000                            gssvL127967,gssvL127968   \n",
       "75156           -1.000000                                                NaN   \n",
       "78325           -1.000000                                                NaN   \n",
       "\n",
       "       DGV_LOSS_n_samples_with_SV  DGV_LOSS_n_samples_tested  \\\n",
       "15532                           0                          0   \n",
       "25374                           0                          0   \n",
       "25495                           0                          0   \n",
       "35276                           0                          0   \n",
       "38617                         539                      15951   \n",
       "38633                         563                      15858   \n",
       "38641                         875                      25389   \n",
       "53658                         944                      21959   \n",
       "54969                           2                      12365   \n",
       "66679                           0                          0   \n",
       "73561                           3                      12408   \n",
       "75156                           0                          0   \n",
       "78325                           0                          0   \n",
       "\n",
       "       DGV_LOSS_Frequency                                              GD_ID  \\\n",
       "15532           -1.000000                                                NaN   \n",
       "25374           -1.000000                                                NaN   \n",
       "25495           -1.000000                                                NaN   \n",
       "35276           -1.000000    gnomAD_v2_DEL_13_161665;gnomAD_v2_DEL_13_161668   \n",
       "38617            0.033791  gnomAD_v2_DEL_14_170979;gnomAD_v2_DEL_14_17098...   \n",
       "38633            0.035503  gnomAD_v2_DEL_14_170862;gnomAD_v2_DEL_14_17088...   \n",
       "38641            0.034464  gnomAD_v2_DEL_14_170979;gnomAD_v2_DEL_14_17098...   \n",
       "53658            0.042989  gnomAD_v2_DEL_2_23514;gnomAD_v2_DEL_2_23520;gn...   \n",
       "54969            0.000162                               gnomAD_v2_DUP_2_7194   \n",
       "66679           -1.000000                                                NaN   \n",
       "73561            0.000242                                                NaN   \n",
       "75156           -1.000000                                                NaN   \n",
       "78325           -1.000000                              gnomAD_v2_DUP_X_55241   \n",
       "\n",
       "                                                   GD_AN  \\\n",
       "15532                                                NaN   \n",
       "25374                                                NaN   \n",
       "25495                                                NaN   \n",
       "35276                                        21476;18384   \n",
       "38617  21388;21474;21422;21476;21474;21476;21422;2147...   \n",
       "38633  21476;21448;21474;21474;21474;21474;21446;2144...   \n",
       "38641  21388;21474;21422;21476;21474;21476;21422;2147...   \n",
       "53658  1064;21458;21458;21408;21470;21372;21474;21468...   \n",
       "54969                                              20400   \n",
       "66679                                                NaN   \n",
       "73561                                                NaN   \n",
       "75156                                                NaN   \n",
       "78325                                              21430   \n",
       "\n",
       "                                                GD_N_HET  \\\n",
       "15532                                                NaN   \n",
       "25374                                                NaN   \n",
       "25495                                                NaN   \n",
       "35276                                                1;2   \n",
       "38617              11;4;6;4;5;1;1;2;12;1;1;1;4;4;12;10;7   \n",
       "38633  31;9;6;3;3;1;2;12;8;11;4;6;4;6;5;1;136;4;1;1;4...   \n",
       "38641            11;4;6;4;5;1;1;2;12;1;1;1;4;4;12;10;7;1   \n",
       "53658                         40;3;3;1;2;1;3;2;3;1;1;1;3   \n",
       "54969                                                 10   \n",
       "66679                                                NaN   \n",
       "73561                                                NaN   \n",
       "75156                                                NaN   \n",
       "78325                                                 34   \n",
       "\n",
       "                                             GD_N_HOMALT  \\\n",
       "15532                                                NaN   \n",
       "25374                                                NaN   \n",
       "25495                                                NaN   \n",
       "35276                                                0;0   \n",
       "38617                  0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0   \n",
       "38633  0;1;0;0;0;2;0;1;0;0;0;0;0;0;0;0;11;0;0;0;0;0;0...   \n",
       "38641                0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0   \n",
       "53658                          4;0;0;0;1;0;0;0;0;0;0;0;0   \n",
       "54969                                                  0   \n",
       "66679                                                NaN   \n",
       "73561                                                NaN   \n",
       "75156                                                NaN   \n",
       "78325                                                  0   \n",
       "\n",
       "                        GD_AF  GD_POPMAX_AF  \\\n",
       "15532                      -1     -1.000000   \n",
       "25374                      -1     -1.000000   \n",
       "25495                      -1     -1.000000   \n",
       "35276  0.00010900000052060932      0.000720   \n",
       "38617   0.0005590000073425472      0.006734   \n",
       "38633     0.00735800014808774      0.034193   \n",
       "38641   0.0005590000073425472      0.006734   \n",
       "53658     0.04511300101876259      0.107623   \n",
       "54969   0.0004900000058114529      0.000940   \n",
       "66679                      -1     -1.000000   \n",
       "73561                      -1     -1.000000   \n",
       "75156                      -1     -1.000000   \n",
       "78325   0.0015869999770075083      0.004090   \n",
       "\n",
       "                                            GD_ID_others  \\\n",
       "15532                                                NaN   \n",
       "25374                                                NaN   \n",
       "25495                                                NaN   \n",
       "35276                                                NaN   \n",
       "38617                                                NaN   \n",
       "38633  gnomAD_v2_DUP_14_40112;gnomAD_v2_MCNV_14_755;g...   \n",
       "38641                             gnomAD_v2_DUP_14_40201   \n",
       "53658                                                NaN   \n",
       "54969          gnomAD_v2_DEL_2_30788;gnomAD_v2_INV_2_117   \n",
       "66679                                                NaN   \n",
       "73561                                gnomAD_v2_INV_8_448   \n",
       "75156                                                NaN   \n",
       "78325                                                NaN   \n",
       "\n",
       "                       DDD_SV  DDD_DUP_n_samples_with_SV  DDD_DUP_Frequency  \\\n",
       "15532                     NaN                        NaN          -1.000000   \n",
       "25374                     NaN                        NaN          -1.000000   \n",
       "25495                     NaN                        NaN          -1.000000   \n",
       "35276                     NaN                        NaN          -1.000000   \n",
       "38617                     NaN                        NaN          -1.000000   \n",
       "38633                     NaN                        NaN          -1.000000   \n",
       "38641  14:106544103-106916718                       18.0           0.021302   \n",
       "53658     2:89198438-89590982                        0.0           0.000000   \n",
       "54969                     NaN                        NaN          -1.000000   \n",
       "66679                     NaN                        NaN          -1.000000   \n",
       "73561                     NaN                        NaN          -1.000000   \n",
       "75156                     NaN                        NaN          -1.000000   \n",
       "78325                     NaN                        NaN          -1.000000   \n",
       "\n",
       "       DDD_DEL_n_samples_with_SV  DDD_DEL_Frequency 1000g_event  1000g_AF  \\\n",
       "15532                        NaN          -1.000000         NaN      -1.0   \n",
       "25374                        NaN          -1.000000         NaN      -1.0   \n",
       "25495                        NaN          -1.000000         NaN      -1.0   \n",
       "35276                        NaN          -1.000000         NaN      -1.0   \n",
       "38617                        NaN          -1.000000         NaN      -1.0   \n",
       "38633                        NaN          -1.000000         NaN      -1.0   \n",
       "38641                       24.0           0.028402         NaN      -1.0   \n",
       "53658                        3.0           0.003550         NaN      -1.0   \n",
       "54969                        NaN          -1.000000         NaN      -1.0   \n",
       "66679                        NaN          -1.000000         NaN      -1.0   \n",
       "73561                        NaN          -1.000000         NaN      -1.0   \n",
       "75156                        NaN          -1.000000         NaN      -1.0   \n",
       "78325                        NaN          -1.000000         NaN      -1.0   \n",
       "\n",
       "       1000g_max_AF                                             IMH_ID  \\\n",
       "15532          -1.0                                                NaN   \n",
       "25374          -1.0                                                NaN   \n",
       "25495          -1.0                           11:77550802-78000858_DUP   \n",
       "35276          -1.0                           13:83249177-83483973_DEL   \n",
       "38617          -1.0  14:106326045-107178888_DEL;14:106329421-106926...   \n",
       "38633          -1.0  14:106326045-107178888_DEL;14:106329399-106471...   \n",
       "38641          -1.0  14:106326045-107178888_DEL;14:106329421-106926...   \n",
       "53658          -1.0  2:89132234-89399420_DEL;2:89132244-89495574_DE...   \n",
       "54969          -1.0                          2:203281078-203287109_DUP   \n",
       "66679          -1.0                                                NaN   \n",
       "73561          -1.0                                                NaN   \n",
       "75156          -1.0                                                NaN   \n",
       "78325          -1.0                            X:68458604-68461670_DUP   \n",
       "\n",
       "         IMH_AF                                      IMH_ID_others  \\\n",
       "15532 -1.000000  10:8738346-108102195_INV;10:15431941-70315351_INV   \n",
       "25374 -1.000000  11:1642919-71249653_INV;11:49233127-89368291_I...   \n",
       "25495  0.000059  11:49233127-89368291_INV;11:49425031-89029881_...   \n",
       "35276  0.000059  13:32957269-100763564_INV;13:33672321-10440911...   \n",
       "38617  0.011460                                                NaN   \n",
       "38633  0.023430  14:106350702-106552313_INV;14:106350727-106725...   \n",
       "38641  0.011460  14:106590902-107013925_DUP;14:106624500-107125...   \n",
       "53658  0.021740  2:26280500-162970618_INV;2:33078871-153818960_...   \n",
       "54969  0.000178  2:37574354-210299893_INV;2:122235797-235423496...   \n",
       "66679 -1.000000  6:18151823-168174514_INV;6:20232222-26808015_I...   \n",
       "73561 -1.000000                                                NaN   \n",
       "75156 -1.000000  9:120401415-122017859_DUP;9:120865318-12138613...   \n",
       "78325  0.000475                            X:46769177-84009828_INV   \n",
       "\n",
       "                                promoters                        dbVar_event  \\\n",
       "15532                         WAC/WAC-AS1                                NaN   \n",
       "25374                                 NaN                                NaN   \n",
       "25495                                 NaN                                NaN   \n",
       "35276                                 NaN                                NaN   \n",
       "38617    ADAM6/FAM30A/LINC00221/LINC00226  copy_number_gain;copy_number_loss   \n",
       "38633                        ADAM6/FAM30A                                NaN   \n",
       "38641                 LINC00221/LINC00226                                NaN   \n",
       "53658                                 NaN                                NaN   \n",
       "54969                                 NaN                                NaN   \n",
       "66679                                 NaN                                NaN   \n",
       "73561  GML/LY6D/LYNX1/LYNX1-SLURP2/SLURP2                                NaN   \n",
       "75156                                 NaN                                NaN   \n",
       "78325                                 NaN                                NaN   \n",
       "\n",
       "                                       dbVar_variant dbVar_status  \\\n",
       "15532                                            NaN          NaN   \n",
       "25374                                            NaN          NaN   \n",
       "25495                                            NaN          NaN   \n",
       "35276                                            NaN          NaN   \n",
       "38617  nssv1602987;nssv3396033;nssv576460;nssv582489   Pathogenic   \n",
       "38633                                            NaN          NaN   \n",
       "38641                                            NaN          NaN   \n",
       "53658                                            NaN          NaN   \n",
       "54969                                            NaN          NaN   \n",
       "66679                                            NaN          NaN   \n",
       "73561                                            NaN          NaN   \n",
       "75156                                            NaN          NaN   \n",
       "78325                                            NaN          NaN   \n",
       "\n",
       "                                          TADcoordinates  \\\n",
       "15532                                                NaN   \n",
       "25374                                                NaN   \n",
       "25495                                                NaN   \n",
       "35276                                                NaN   \n",
       "38617                                                NaN   \n",
       "38633                                                NaN   \n",
       "38641                                                NaN   \n",
       "53658                                                NaN   \n",
       "54969                                                NaN   \n",
       "66679                                                NaN   \n",
       "73561                              8:143840001-144400000   \n",
       "75156  9:121160001-122240000;9:121200001-122160000;9:...   \n",
       "78325                                                NaN   \n",
       "\n",
       "                                       ENCODEexperiments GCcontent_left  \\\n",
       "15532                                                NaN       0.650000   \n",
       "25374                                                NaN       0.515000   \n",
       "25495                                                NaN       0.575000   \n",
       "35276                                                NaN       0.220000   \n",
       "38617                                                NaN       0.575000   \n",
       "38633                                                NaN       0.645000   \n",
       "38641                                                NaN       0.505000   \n",
       "53658                                                NaN       0.505000   \n",
       "54969                                                NaN       0.490000   \n",
       "66679                                                NaN       0.425000   \n",
       "73561                            ENCFF531PAV:ENCFF931RKD       0.660000   \n",
       "75156  ENCFF306YQN:ENCFF373MCF:ENCFF444XGA:ENCFF558RG...       0.175000   \n",
       "78325                                                NaN       0.540000   \n",
       "\n",
       "      GCcontent_right                           Repeats_coord_left  \\\n",
       "15532        0.415000                                          NaN   \n",
       "25374        0.610000                         11:68782152-68782459   \n",
       "25495        0.470000                                          NaN   \n",
       "35276        0.280000                         13:83435564-83435614   \n",
       "38617        0.535000                                          NaN   \n",
       "38633        0.580000                                          NaN   \n",
       "38641        0.590000                       14:106551142-106551601   \n",
       "53658        0.505000                                          NaN   \n",
       "54969        0.560000  2:203280989-203281150/2:203281150-203281442   \n",
       "66679        0.370000                          6:25909396-25909703   \n",
       "73561        0.320000                                          NaN   \n",
       "75156        0.420000  9:121152631-121152796/9:121152878-121152940   \n",
       "78325        0.530000                                          NaN   \n",
       "\n",
       "      Repeats_type_left                        Repeats_coord_right  \\\n",
       "15532               NaN  10:28848065-28848094/10:28848118-28848410   \n",
       "25374            MLT1A0                                        NaN   \n",
       "25495               NaN                                        NaN   \n",
       "35276            A-rich  13:83439503-83439802/13:83439899-83439941   \n",
       "38617               NaN                                        NaN   \n",
       "38633               NaN                                        NaN   \n",
       "38641           LTR16B2                                        NaN   \n",
       "53658               NaN                                        NaN   \n",
       "54969       AluJr/AluSc                      2:203286646-203286929   \n",
       "66679             AluSx                                        NaN   \n",
       "73561               NaN                      8:143928389-143929557   \n",
       "75156       (TA)n/(TA)n                      9:121295249-121295398   \n",
       "78325               NaN                                        NaN   \n",
       "\n",
       "      Repeats_type_right ACMG  HI_CGscore TriS_CGscore DDD_status DDD_mode  \\\n",
       "15532          (TG)n/L2a  NaN         3.0          NaN        NaN      NaN   \n",
       "25374                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "25495                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "35276      AluSg/(TATG)n  NaN         NaN          NaN        NaN      NaN   \n",
       "38617                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "38633                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "38641                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "53658                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "54969             AluSc5  NaN         3.0          NaN        NaN      NaN   \n",
       "66679                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "73561              L1M3f  NaN         NaN          NaN        NaN      NaN   \n",
       "75156               MIR3  NaN         NaN          NaN        NaN      NaN   \n",
       "78325                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "\n",
       "      DDD_consequence DDD_disease DDD_pmids  HI_DDDpercent  synZ_ExAC  \\\n",
       "15532             NaN         NaN       NaN            NaN        NaN   \n",
       "25374             NaN         NaN       NaN            NaN        NaN   \n",
       "25495             NaN         NaN       NaN          38.04   0.355310   \n",
       "35276             NaN         NaN       NaN            NaN        NaN   \n",
       "38617             NaN         NaN       NaN            NaN        NaN   \n",
       "38633             NaN         NaN       NaN            NaN        NaN   \n",
       "38641             NaN         NaN       NaN            NaN        NaN   \n",
       "53658             NaN         NaN       NaN            NaN        NaN   \n",
       "54969             NaN         NaN       NaN           1.48   0.303008   \n",
       "66679             NaN         NaN       NaN          64.34   0.086779   \n",
       "73561             NaN         NaN       NaN            NaN        NaN   \n",
       "75156             NaN         NaN       NaN            NaN        NaN   \n",
       "78325             NaN         NaN       NaN            NaN        NaN   \n",
       "\n",
       "       misZ_ExAC  pLI_ExAC  delZ_ExAC  dupZ_ExAC  cnvZ_ExAC morbidGenes  \\\n",
       "15532        NaN       NaN        NaN        NaN        NaN         yes   \n",
       "25374        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "25495  -1.040969  0.997629   0.920190   0.780302   1.033942         NaN   \n",
       "35276        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "38617        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "38633        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "38641        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "53658        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "54969   1.476221  0.999600   0.805782   0.357465   0.654177         yes   \n",
       "66679   0.715960  0.013546  -1.282744   0.393160  -0.311826         yes   \n",
       "73561        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "75156        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "78325        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "\n",
       "      morbidGenesCandidates Mim Number Phenotypes Inheritance  \\\n",
       "15532                   NaN        NaN        NaN         NaN   \n",
       "25374                   NaN        NaN        NaN         NaN   \n",
       "25495                   NaN        NaN        NaN         NaN   \n",
       "35276                   NaN        NaN        NaN         NaN   \n",
       "38617                   NaN        NaN        NaN         NaN   \n",
       "38633                   NaN        NaN        NaN         NaN   \n",
       "38641                   NaN        NaN        NaN         NaN   \n",
       "53658                   NaN        NaN        NaN         NaN   \n",
       "54969                   NaN        NaN        NaN         NaN   \n",
       "66679                   yes        NaN        NaN         NaN   \n",
       "73561                   NaN        NaN        NaN         NaN   \n",
       "75156                   NaN        NaN        NaN         NaN   \n",
       "78325                   NaN        NaN        NaN         NaN   \n",
       "\n",
       "       AnnotSV ranking  num_callers  gridss  svaba  brass  \n",
       "15532                4          3.0     1.0    1.0    1.0  \n",
       "25374                2          3.0     1.0    1.0    1.0  \n",
       "25495                2          3.0     1.0    1.0    1.0  \n",
       "35276                2          3.0     1.0    1.0    1.0  \n",
       "38617                5          2.0     1.0    NaN    1.0  \n",
       "38633                1          3.0     1.0    1.0    1.0  \n",
       "38641                1          3.0     1.0    1.0    1.0  \n",
       "53658                1          3.0     1.0    1.0    1.0  \n",
       "54969                4          2.0     NaN    1.0    1.0  \n",
       "66679                4          2.0     1.0    NaN    1.0  \n",
       "73561                2          3.0     1.0    1.0    1.0  \n",
       "75156                2          3.0     1.0    1.0    1.0  \n",
       "78325                2          3.0     1.0    1.0    1.0  "
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>chromosome</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>gene</th>\n",
       "      <th>log2</th>\n",
       "      <th>baf</th>\n",
       "      <th>cn</th>\n",
       "      <th>cn1</th>\n",
       "      <th>cn2</th>\n",
       "      <th>depth</th>\n",
       "      <th>probes</th>\n",
       "      <th>weight</th>\n",
       "      <th>aberrant_cell_frac</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>9</td>\n",
       "      <td>121152952</td>\n",
       "      <td>121294770</td>\n",
       "      <td>-</td>\n",
       "      <td>-0.772378</td>\n",
       "      <td>0.840909</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>45.1837</td>\n",
       "      <td>152</td>\n",
       "      <td>145.729000</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>X</td>\n",
       "      <td>1446744</td>\n",
       "      <td>2699498</td>\n",
       "      <td>-</td>\n",
       "      <td>-0.859047</td>\n",
       "      <td>0.540000</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>74.0274</td>\n",
       "      <td>1271</td>\n",
       "      <td>858.825000</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>8</td>\n",
       "      <td>143850648</td>\n",
       "      <td>143929135</td>\n",
       "      <td>-</td>\n",
       "      <td>0.583125</td>\n",
       "      <td>0.674376</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>121.6700</td>\n",
       "      <td>32</td>\n",
       "      <td>0.947261</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>9</td>\n",
       "      <td>121152837</td>\n",
       "      <td>121295287</td>\n",
       "      <td>-</td>\n",
       "      <td>-0.772284</td>\n",
       "      <td>0.842023</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>45.1754</td>\n",
       "      <td>23</td>\n",
       "      <td>0.958744</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>10</td>\n",
       "      <td>28821259</td>\n",
       "      <td>28848026</td>\n",
       "      <td>-</td>\n",
       "      <td>-0.787244</td>\n",
       "      <td>0.820922</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>44.3410</td>\n",
       "      <td>7</td>\n",
       "      <td>0.955477</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   chromosome      start        end gene      log2       baf   cn  cn1  cn2  \\\n",
       "9           9  121152952  121294770    - -0.772378  0.840909  1.0  1.0  0.0   \n",
       "24          X    1446744    2699498    - -0.859047  0.540000  1.0  1.0  0.0   \n",
       "26          8  143850648  143929135    -  0.583125  0.674376  3.0  2.0  1.0   \n",
       "27          9  121152837  121295287    - -0.772284  0.842023  1.0  1.0  0.0   \n",
       "28         10   28821259   28848026    - -0.787244  0.820922  1.0  1.0  0.0   \n",
       "\n",
       "       depth  probes      weight  aberrant_cell_frac  \n",
       "9    45.1837     152  145.729000                 1.0  \n",
       "24   74.0274    1271  858.825000                 1.0  \n",
       "26  121.6700      32    0.947261                 NaN  \n",
       "27   45.1754      23    0.958744                 NaN  \n",
       "28   44.3410       7    0.955477                 NaN  "
      ]
     },
     "execution_count": 89,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cnv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [],
   "source": [
    "window = 1000\n",
    "\n",
    "for index, row in sv.iterrows():\n",
    "    sv_chr = row['SV chrom']\n",
    "    sv_start = row['SV start']\n",
    "    sv_end = row['SV end']\n",
    "    sv_length = row['SV length']\n",
    "    for i, r in cnv.iterrows():\n",
    "        cnv_chr = r['chromosome']\n",
    "        cnv_start = r['start']\n",
    "        cnv_end = r['end']\n",
    "        if sv_chr == cnv_chr:\n",
    "            if abs(sv_start - cnv_start)<window:\n",
    "                overlap = (min(cnv_end, sv_end) - max(cnv_start, sv_start))/sv_length\n",
    "                sv.loc[index, 'cnv'] = f\"{cnv_chr}:{cnv_start}-{cnv_end} {overlap}\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>AnnotSV ID</th>\n",
       "      <th>SV chrom</th>\n",
       "      <th>SV start</th>\n",
       "      <th>SV end</th>\n",
       "      <th>SV length</th>\n",
       "      <th>SV type</th>\n",
       "      <th>ID</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>FILTER</th>\n",
       "      <th>INFO</th>\n",
       "      <th>FORMAT</th>\n",
       "      <th>I-H-108298-N1-1-D1-1</th>\n",
       "      <th>IID_H158605_T01_01_WG01</th>\n",
       "      <th>AnnotSV type</th>\n",
       "      <th>Gene name</th>\n",
       "      <th>NM</th>\n",
       "      <th>CDS length</th>\n",
       "      <th>tx length</th>\n",
       "      <th>location</th>\n",
       "      <th>intersectStart</th>\n",
       "      <th>intersectEnd</th>\n",
       "      <th>DGV_GAIN_IDs</th>\n",
       "      <th>DGV_GAIN_n_samples_with_SV</th>\n",
       "      <th>DGV_GAIN_n_samples_tested</th>\n",
       "      <th>DGV_GAIN_Frequency</th>\n",
       "      <th>DGV_LOSS_IDs</th>\n",
       "      <th>DGV_LOSS_n_samples_with_SV</th>\n",
       "      <th>DGV_LOSS_n_samples_tested</th>\n",
       "      <th>DGV_LOSS_Frequency</th>\n",
       "      <th>GD_ID</th>\n",
       "      <th>GD_AN</th>\n",
       "      <th>GD_N_HET</th>\n",
       "      <th>GD_N_HOMALT</th>\n",
       "      <th>GD_AF</th>\n",
       "      <th>GD_POPMAX_AF</th>\n",
       "      <th>GD_ID_others</th>\n",
       "      <th>DDD_SV</th>\n",
       "      <th>DDD_DUP_n_samples_with_SV</th>\n",
       "      <th>DDD_DUP_Frequency</th>\n",
       "      <th>DDD_DEL_n_samples_with_SV</th>\n",
       "      <th>DDD_DEL_Frequency</th>\n",
       "      <th>1000g_event</th>\n",
       "      <th>1000g_AF</th>\n",
       "      <th>1000g_max_AF</th>\n",
       "      <th>IMH_ID</th>\n",
       "      <th>IMH_AF</th>\n",
       "      <th>IMH_ID_others</th>\n",
       "      <th>promoters</th>\n",
       "      <th>dbVar_event</th>\n",
       "      <th>dbVar_variant</th>\n",
       "      <th>dbVar_status</th>\n",
       "      <th>TADcoordinates</th>\n",
       "      <th>ENCODEexperiments</th>\n",
       "      <th>GCcontent_left</th>\n",
       "      <th>GCcontent_right</th>\n",
       "      <th>Repeats_coord_left</th>\n",
       "      <th>Repeats_type_left</th>\n",
       "      <th>Repeats_coord_right</th>\n",
       "      <th>Repeats_type_right</th>\n",
       "      <th>ACMG</th>\n",
       "      <th>HI_CGscore</th>\n",
       "      <th>TriS_CGscore</th>\n",
       "      <th>DDD_status</th>\n",
       "      <th>DDD_mode</th>\n",
       "      <th>DDD_consequence</th>\n",
       "      <th>DDD_disease</th>\n",
       "      <th>DDD_pmids</th>\n",
       "      <th>HI_DDDpercent</th>\n",
       "      <th>synZ_ExAC</th>\n",
       "      <th>misZ_ExAC</th>\n",
       "      <th>pLI_ExAC</th>\n",
       "      <th>delZ_ExAC</th>\n",
       "      <th>dupZ_ExAC</th>\n",
       "      <th>cnvZ_ExAC</th>\n",
       "      <th>morbidGenes</th>\n",
       "      <th>morbidGenesCandidates</th>\n",
       "      <th>Mim Number</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Inheritance</th>\n",
       "      <th>AnnotSV ranking</th>\n",
       "      <th>num_callers</th>\n",
       "      <th>gridss</th>\n",
       "      <th>svaba</th>\n",
       "      <th>brass</th>\n",
       "      <th>cnv</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>50275</th>\n",
       "      <td>5_60201995_60294535_DEL</td>\n",
       "      <td>5</td>\n",
       "      <td>60201995</td>\n",
       "      <td>60294535</td>\n",
       "      <td>92540.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[5:60294535[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=60294535;Callers=gridss,svaba,brass;CHR2=5...</td>\n",
       "      <td>brass_IID_H158605_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>47:22:0.493506:0.487179:0.475:53:-102.3:0:34:1...</td>\n",
       "      <td>full</td>\n",
       "      <td>ERCC8/NDUFAF2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>gnomAD_v2_DEL_5_67953</td>\n",
       "      <td>21476</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>4.70000013592653e-05</td>\n",
       "      <td>0.000561</td>\n",
       "      <td>gnomAD_v2_DUP_5_15058</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>5:60121012-60337042_DUP;5:24758284-101693420_I...</td>\n",
       "      <td>ERCC8/NDUFAF2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5:60200001-61080000;5:60200001-61120000;5:6020...</td>\n",
       "      <td>ENCFF066NYU:ENCFF075QYD:ENCFF136AUY:ENCFF248DG...</td>\n",
       "      <td>0.400000</td>\n",
       "      <td>0.345000</td>\n",
       "      <td>5:60201501-60201986/5:60202021-60202154</td>\n",
       "      <td>LTR8A/AluJo</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>53.74</td>\n",
       "      <td>-0.648906</td>\n",
       "      <td>-0.460345</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.661223</td>\n",
       "      <td>-0.234814</td>\n",
       "      <td>-0.860460</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>5:60201994-60294535 1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56231</th>\n",
       "      <td>7_110382221_110493855_DEL</td>\n",
       "      <td>7</td>\n",
       "      <td>110382221</td>\n",
       "      <td>110493855</td>\n",
       "      <td>111634.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[7:110493855[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=110493855;Callers=gridss,svaba,brass;CHR2=...</td>\n",
       "      <td>brass_IID_H158605_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>33:27:0.506494:0.5:0.534247:38:-92.42:0:30:92....</td>\n",
       "      <td>full</td>\n",
       "      <td>IMMP2L</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>7:110382216-110493864_DEL</td>\n",
       "      <td>0.000059</td>\n",
       "      <td>7:66631328-143298938_INV;7:79317852-147370690_...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>7:110400001-111360000;7:110440001-111360000</td>\n",
       "      <td>ENCFF274VJU:ENCFF588KUZ:ENCFF701HCM:ENCFF996RJN</td>\n",
       "      <td>0.275000</td>\n",
       "      <td>0.280000</td>\n",
       "      <td>7:110382299-110382541</td>\n",
       "      <td>L1ME4a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4.25</td>\n",
       "      <td>-0.543226</td>\n",
       "      <td>-0.023467</td>\n",
       "      <td>0.034132</td>\n",
       "      <td>-2.623544</td>\n",
       "      <td>0.194114</td>\n",
       "      <td>-2.471628</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>7:110382219-110493855 1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58404</th>\n",
       "      <td>9_15424528_22380856_DEL</td>\n",
       "      <td>9</td>\n",
       "      <td>15424528</td>\n",
       "      <td>22380856</td>\n",
       "      <td>6956328.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[9:22380856[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=22380856;Callers=gridss,svaba,brass;CHR2=9...</td>\n",
       "      <td>brass_IID_H158605_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>39:15:0.618421:0.618421:0.61039:50:-107.6:0:33...</td>\n",
       "      <td>full</td>\n",
       "      <td>ACER2/ADAMTSL1/BNC2/BNC2-AS1/C9orf92/CCDC171/C...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ACER2/ADAMTSL1/BNC2/BNC2-AS1/C9orf92/CCDC171/C...</td>\n",
       "      <td>copy_number_gain;copy_number_loss</td>\n",
       "      <td>nssv13651853;nssv578326;nssv582133</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>9:15680001-17560000;9:15720001-17000000;9:1572...</td>\n",
       "      <td>ENCFF032FMN:ENCFF066NYU:ENCFF075QYD:ENCFF136AU...</td>\n",
       "      <td>0.210000</td>\n",
       "      <td>0.400000</td>\n",
       "      <td>9:15424411-15424698</td>\n",
       "      <td>MER5C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>yes</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>9:15424527-22380856 1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58477</th>\n",
       "      <td>9_21975806_22009649_DEL</td>\n",
       "      <td>9</td>\n",
       "      <td>21975806</td>\n",
       "      <td>22009649</td>\n",
       "      <td>33843.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[9:22009649[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=22009649;Callers=gridss,svaba,brass;CHR2=9...</td>\n",
       "      <td>brass_IID_H158605_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>37:15:0.210526:0.210526:0.347826:43:-79.22:0:2...</td>\n",
       "      <td>full</td>\n",
       "      <td>CDKN2A/CDKN2B/CDKN2B-AS1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>gnomAD_v2_DUP_9_27128</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CDKN2A/CDKN2B-AS1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.410000</td>\n",
       "      <td>0.485000</td>\n",
       "      <td>9:21975883-21976031</td>\n",
       "      <td>MIR</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>yes</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>9:21975805-22009649 1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                      AnnotSV ID SV chrom   SV start     SV end  SV length  \\\n",
       "50275    5_60201995_60294535_DEL        5   60201995   60294535    92540.0   \n",
       "56231  7_110382221_110493855_DEL        7  110382221  110493855   111634.0   \n",
       "58404    9_15424528_22380856_DEL        9   15424528   22380856  6956328.0   \n",
       "58477    9_21975806_22009649_DEL        9   21975806   22009649    33843.0   \n",
       "\n",
       "      SV type ID REF             ALT QUAL FILTER  \\\n",
       "50275     DEL  .   N   N[5:60294535[    .   PASS   \n",
       "56231     DEL  .   N  N[7:110493855[    .   PASS   \n",
       "58404     DEL  .   N   N[9:22380856[    .   PASS   \n",
       "58477     DEL  .   N   N[9:22009649[    .   PASS   \n",
       "\n",
       "                                                    INFO  \\\n",
       "50275  END=60294535;Callers=gridss,svaba,brass;CHR2=5...   \n",
       "56231  END=110493855;Callers=gridss,svaba,brass;CHR2=...   \n",
       "58404  END=22380856;Callers=gridss,svaba,brass;CHR2=9...   \n",
       "58477  END=22009649;Callers=gridss,svaba,brass;CHR2=9...   \n",
       "\n",
       "                                                  FORMAT  \\\n",
       "50275  brass_IID_H158605_T01_01_WG01_PS:brass_IID_H15...   \n",
       "56231  brass_IID_H158605_T01_01_WG01_PS:brass_IID_H15...   \n",
       "58404  brass_IID_H158605_T01_01_WG01_PS:brass_IID_H15...   \n",
       "58477  brass_IID_H158605_T01_01_WG01_PS:brass_IID_H15...   \n",
       "\n",
       "                                    I-H-108298-N1-1-D1-1  \\\n",
       "50275  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "56231  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "58404  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "58477  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "\n",
       "                                 IID_H158605_T01_01_WG01 AnnotSV type  \\\n",
       "50275  47:22:0.493506:0.487179:0.475:53:-102.3:0:34:1...         full   \n",
       "56231  33:27:0.506494:0.5:0.534247:38:-92.42:0:30:92....         full   \n",
       "58404  39:15:0.618421:0.618421:0.61039:50:-107.6:0:33...         full   \n",
       "58477  37:15:0.210526:0.210526:0.347826:43:-79.22:0:2...         full   \n",
       "\n",
       "                                               Gene name   NM  CDS length  \\\n",
       "50275                                      ERCC8/NDUFAF2  NaN         NaN   \n",
       "56231                                             IMMP2L  NaN         NaN   \n",
       "58404  ACER2/ADAMTSL1/BNC2/BNC2-AS1/C9orf92/CCDC171/C...  NaN         NaN   \n",
       "58477                           CDKN2A/CDKN2B/CDKN2B-AS1  NaN         NaN   \n",
       "\n",
       "       tx length location  intersectStart  intersectEnd DGV_GAIN_IDs  \\\n",
       "50275        NaN      NaN             NaN           NaN          NaN   \n",
       "56231        NaN      NaN             NaN           NaN          NaN   \n",
       "58404        NaN      NaN             NaN           NaN          NaN   \n",
       "58477        NaN      NaN             NaN           NaN          NaN   \n",
       "\n",
       "       DGV_GAIN_n_samples_with_SV  DGV_GAIN_n_samples_tested  \\\n",
       "50275                           0                          0   \n",
       "56231                           0                          0   \n",
       "58404                           0                          0   \n",
       "58477                           0                          0   \n",
       "\n",
       "       DGV_GAIN_Frequency DGV_LOSS_IDs  DGV_LOSS_n_samples_with_SV  \\\n",
       "50275                -1.0          NaN                           0   \n",
       "56231                -1.0          NaN                           0   \n",
       "58404                -1.0          NaN                           0   \n",
       "58477                -1.0          NaN                           0   \n",
       "\n",
       "       DGV_LOSS_n_samples_tested  DGV_LOSS_Frequency                  GD_ID  \\\n",
       "50275                          0                -1.0  gnomAD_v2_DEL_5_67953   \n",
       "56231                          0                -1.0                    NaN   \n",
       "58404                          0                -1.0                    NaN   \n",
       "58477                          0                -1.0                    NaN   \n",
       "\n",
       "       GD_AN GD_N_HET GD_N_HOMALT                 GD_AF  GD_POPMAX_AF  \\\n",
       "50275  21476        1           0  4.70000013592653e-05      0.000561   \n",
       "56231    NaN      NaN         NaN                    -1     -1.000000   \n",
       "58404    NaN      NaN         NaN                    -1     -1.000000   \n",
       "58477    NaN      NaN         NaN                    -1     -1.000000   \n",
       "\n",
       "                GD_ID_others DDD_SV  DDD_DUP_n_samples_with_SV  \\\n",
       "50275  gnomAD_v2_DUP_5_15058    NaN                        NaN   \n",
       "56231                    NaN    NaN                        NaN   \n",
       "58404                    NaN    NaN                        NaN   \n",
       "58477  gnomAD_v2_DUP_9_27128    NaN                        NaN   \n",
       "\n",
       "       DDD_DUP_Frequency  DDD_DEL_n_samples_with_SV  DDD_DEL_Frequency  \\\n",
       "50275               -1.0                        NaN               -1.0   \n",
       "56231               -1.0                        NaN               -1.0   \n",
       "58404               -1.0                        NaN               -1.0   \n",
       "58477               -1.0                        NaN               -1.0   \n",
       "\n",
       "      1000g_event  1000g_AF  1000g_max_AF                     IMH_ID  \\\n",
       "50275         NaN      -1.0          -1.0                        NaN   \n",
       "56231         NaN      -1.0          -1.0  7:110382216-110493864_DEL   \n",
       "58404         NaN      -1.0          -1.0                        NaN   \n",
       "58477         NaN      -1.0          -1.0                        NaN   \n",
       "\n",
       "         IMH_AF                                      IMH_ID_others  \\\n",
       "50275 -1.000000  5:60121012-60337042_DUP;5:24758284-101693420_I...   \n",
       "56231  0.000059  7:66631328-143298938_INV;7:79317852-147370690_...   \n",
       "58404 -1.000000                                                NaN   \n",
       "58477 -1.000000                                                NaN   \n",
       "\n",
       "                                               promoters  \\\n",
       "50275                                      ERCC8/NDUFAF2   \n",
       "56231                                                NaN   \n",
       "58404  ACER2/ADAMTSL1/BNC2/BNC2-AS1/C9orf92/CCDC171/C...   \n",
       "58477                                  CDKN2A/CDKN2B-AS1   \n",
       "\n",
       "                             dbVar_event                       dbVar_variant  \\\n",
       "50275                                NaN                                 NaN   \n",
       "56231                                NaN                                 NaN   \n",
       "58404  copy_number_gain;copy_number_loss  nssv13651853;nssv578326;nssv582133   \n",
       "58477                                NaN                                 NaN   \n",
       "\n",
       "      dbVar_status                                     TADcoordinates  \\\n",
       "50275          NaN  5:60200001-61080000;5:60200001-61120000;5:6020...   \n",
       "56231          NaN        7:110400001-111360000;7:110440001-111360000   \n",
       "58404   Pathogenic  9:15680001-17560000;9:15720001-17000000;9:1572...   \n",
       "58477          NaN                                                NaN   \n",
       "\n",
       "                                       ENCODEexperiments GCcontent_left  \\\n",
       "50275  ENCFF066NYU:ENCFF075QYD:ENCFF136AUY:ENCFF248DG...       0.400000   \n",
       "56231    ENCFF274VJU:ENCFF588KUZ:ENCFF701HCM:ENCFF996RJN       0.275000   \n",
       "58404  ENCFF032FMN:ENCFF066NYU:ENCFF075QYD:ENCFF136AU...       0.210000   \n",
       "58477                                                NaN       0.410000   \n",
       "\n",
       "      GCcontent_right                       Repeats_coord_left  \\\n",
       "50275        0.345000  5:60201501-60201986/5:60202021-60202154   \n",
       "56231        0.280000                    7:110382299-110382541   \n",
       "58404        0.400000                      9:15424411-15424698   \n",
       "58477        0.485000                      9:21975883-21976031   \n",
       "\n",
       "      Repeats_type_left Repeats_coord_right Repeats_type_right ACMG  \\\n",
       "50275       LTR8A/AluJo                 NaN                NaN  NaN   \n",
       "56231            L1ME4a                 NaN                NaN  NaN   \n",
       "58404             MER5C                 NaN                NaN  NaN   \n",
       "58477               MIR                 NaN                NaN  NaN   \n",
       "\n",
       "       HI_CGscore TriS_CGscore DDD_status DDD_mode DDD_consequence  \\\n",
       "50275         NaN          NaN        NaN      NaN             NaN   \n",
       "56231         NaN          NaN        NaN      NaN             NaN   \n",
       "58404         NaN          NaN        NaN      NaN             NaN   \n",
       "58477         NaN          NaN        NaN      NaN             NaN   \n",
       "\n",
       "      DDD_disease DDD_pmids  HI_DDDpercent  synZ_ExAC  misZ_ExAC  pLI_ExAC  \\\n",
       "50275         NaN       NaN          53.74  -0.648906  -0.460345       NaN   \n",
       "56231         NaN       NaN           4.25  -0.543226  -0.023467  0.034132   \n",
       "58404         NaN       NaN            NaN        NaN        NaN       NaN   \n",
       "58477         NaN       NaN            NaN        NaN        NaN       NaN   \n",
       "\n",
       "       delZ_ExAC  dupZ_ExAC  cnvZ_ExAC morbidGenes morbidGenesCandidates  \\\n",
       "50275  -1.661223  -0.234814  -0.860460         yes                   NaN   \n",
       "56231  -2.623544   0.194114  -2.471628         NaN                   NaN   \n",
       "58404        NaN        NaN        NaN         yes                   yes   \n",
       "58477        NaN        NaN        NaN         yes                   yes   \n",
       "\n",
       "      Mim Number Phenotypes Inheritance  AnnotSV ranking  num_callers  gridss  \\\n",
       "50275        NaN        NaN         NaN                4          3.0     1.0   \n",
       "56231        NaN        NaN         NaN                2          3.0     1.0   \n",
       "58404        NaN        NaN         NaN                5          3.0     1.0   \n",
       "58477        NaN        NaN         NaN                4          3.0     1.0   \n",
       "\n",
       "       svaba  brass                        cnv  \n",
       "50275    1.0    1.0    5:60201994-60294535 1.0  \n",
       "56231    1.0    1.0  7:110382219-110493855 1.0  \n",
       "58404    1.0    1.0    9:15424527-22380856 1.0  \n",
       "58477    1.0    1.0    9:21975805-22009649 1.0  "
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sv[sv['cnv'].notnull()]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IID_H158638_T01_01_WG01\n",
      "IID_H158635_T01_01_WG01\n",
      "IID_H158644_T01_01_WG01\n",
      "IID_H158641_T01_01_WG01\n",
      "IID_H158640_T01_01_WG01\n",
      "IID_H158637_T01_01_WG01\n",
      "IID_H158634_T01_01_WG01\n",
      "IID_H158643_T01_01_WG01\n",
      "IID_H158639_T01_01_WG01\n",
      "IID_H158636_T01_01_WG01\n",
      "IID_H158633_T01_01_WG01\n",
      "IID_H158642_T01_01_WG01\n",
      "IID_H158632_T01_01_WG01\n",
      "IID_H158629_T01_01_WG01\n",
      "IID_H158626_T01_01_WG01\n",
      "IID_H158623_T01_01_WG01\n",
      "IID_H158631_T01_01_WG01\n",
      "IID_H158628_T01_01_WG01\n",
      "IID_H158625_T01_01_WG01\n",
      "IID_H158622_T01_01_WG01\n",
      "IID_H158630_T01_01_WG01\n",
      "IID_H158627_T01_01_WG01\n",
      "IID_H158621_T01_01_WG01\n",
      "IID_H158624_T01_01_WG01\n",
      "IID_H158620_T01_01_WG01\n",
      "IID_H158617_T01_01_WG01\n",
      "IID_H158614_T01_01_WG01\n",
      "IID_H158611_T01_01_WG01\n",
      "IID_H158616_T01_01_WG01\n",
      "IID_H158619_T01_01_WG01\n",
      "IID_H158613_T01_01_WG01\n",
      "IID_H158610_T01_01_WG01\n",
      "IID_H158615_T01_01_WG01\n",
      "IID_H158612_T01_01_WG01\n",
      "IID_H158618_T01_01_WG01\n",
      "IID_H158609_T01_01_WG01\n",
      "IID_H158608_T01_01_WG01\n",
      "IID_H158605_T01_01_WG01\n",
      "IID_H158604_T01_01_WG01\n",
      "IID_H158607_T01_01_WG01\n",
      "IID_H158606_T01_01_WG01\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[[('IID_H158644_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158644_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158643_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158643_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158642_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158642_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158641_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158641_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158640_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158640_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158639_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158639_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158638_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158638_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158637_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158637_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158636_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158636_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158635_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158635_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158634_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158634_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158633_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158633_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158632_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158632_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158631_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158631_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158630_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158630_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158629_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158629_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158628_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158628_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158627_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158627_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158626_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158626_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158625_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158625_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158624_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158624_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158623_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158623_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158622_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158622_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158621_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158621_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158620_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158620_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158619_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158619_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158618_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158618_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158617_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158617_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158616_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158616_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158615_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158615_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158614_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158614_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158613_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158613_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158612_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158612_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158611_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158611_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158610_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158610_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158609_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158609_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158608_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158608_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158607_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158607_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158606_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158606_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158605_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158605_T01_01_WG01_gridss_somatic.vcf')],\n",
       " [('IID_H158604_T01_01_WG01',\n",
       "   '/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic/IID_H158604_T01_01_WG01_gridss_somatic.vcf')]]"
      ]
     },
     "execution_count": 120,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Process ForkPoolWorker-51:\n",
      "Process ForkPoolWorker-52:\n",
      "Process ForkPoolWorker-50:\n",
      "Process ForkPoolWorker-49:\n",
      "Traceback (most recent call last):\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/process.py\", line 258, in _bootstrap\n",
      "    self.run()\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/process.py\", line 93, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "Traceback (most recent call last):\n",
      "Traceback (most recent call last):\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/pool.py\", line 108, in worker\n",
      "    task = get()\n",
      "Traceback (most recent call last):\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/process.py\", line 258, in _bootstrap\n",
      "    self.run()\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/process.py\", line 258, in _bootstrap\n",
      "    self.run()\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/process.py\", line 258, in _bootstrap\n",
      "    self.run()\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/queues.py\", line 334, in get\n",
      "    with self._rlock:\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/process.py\", line 93, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/process.py\", line 93, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/synchronize.py\", line 96, in __enter__\n",
      "    return self._semlock.__enter__()\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/process.py\", line 93, in run\n",
      "    self._target(*self._args, **self._kwargs)\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/pool.py\", line 108, in worker\n",
      "    task = get()\n",
      "KeyboardInterrupt\n",
      "  File \"/work/isabl/opt/python/v3.6.5/lib/python3.6/multiprocessing/queues.py\", line 334, in get\n",
      "    with self._rlock:\n"
     ]
    }
   ],
   "source": [
    "import pysam\n",
    "\n",
    "def gridss_somatic(inp):\n",
    "    outdir, name, path = inp\n",
    "    \"\"\"Filter gridss input to just somatic.\"\"\"\n",
    "    print(name)\n",
    "     \n",
    "    updated_inp = []\n",
    "    new_path = os.path.join(outdir, name + \"_gridss_somatic.vcf\")\n",
    "    with pysam.VariantFile(path, mode=\"r\") as vcf_in:\n",
    "        with pysam.VariantFile(\n",
    "            new_path, mode=\"w\", header=vcf_in.header\n",
    "        ) as vcf_out:\n",
    "            for rec in vcf_in:\n",
    "                if not rec.info.get(\"SVTYPE\"):\n",
    "                    continue\n",
    "                if rec.filter and not \"PASS\" in list(rec.filter):\n",
    "                    continue\n",
    "                if not (\"[\" in rec.alts[0] or \"]\" in rec.alts[0]):\n",
    "                    continue\n",
    "                if rec.samples[0][\"QUAL\"] > 0:\n",
    "                    continue\n",
    "                vcf_out.write(rec)\n",
    "    updated_inp.append((name, new_path))\n",
    "    return updated_inp\n",
    "\n",
    "\n",
    "samples = [\n",
    "     'IID_H158644_T01_01_WG01',\n",
    "     'IID_H158643_T01_01_WG01',\n",
    "     'IID_H158642_T01_01_WG01',\n",
    "     'IID_H158641_T01_01_WG01',\n",
    "     'IID_H158640_T01_01_WG01',\n",
    "     'IID_H158639_T01_01_WG01',\n",
    "     'IID_H158638_T01_01_WG01',\n",
    "     'IID_H158637_T01_01_WG01',\n",
    "     'IID_H158636_T01_01_WG01',\n",
    "     'IID_H158635_T01_01_WG01',\n",
    "     'IID_H158634_T01_01_WG01',\n",
    "     'IID_H158633_T01_01_WG01',\n",
    "     'IID_H158632_T01_01_WG01',\n",
    "     'IID_H158631_T01_01_WG01',\n",
    "     'IID_H158630_T01_01_WG01',\n",
    "     'IID_H158629_T01_01_WG01',\n",
    "     'IID_H158628_T01_01_WG01',\n",
    "     'IID_H158627_T01_01_WG01',\n",
    "     'IID_H158626_T01_01_WG01',\n",
    "     'IID_H158625_T01_01_WG01',\n",
    "     'IID_H158624_T01_01_WG01',\n",
    "     'IID_H158623_T01_01_WG01',\n",
    "     'IID_H158622_T01_01_WG01',\n",
    "     'IID_H158621_T01_01_WG01',\n",
    "     'IID_H158620_T01_01_WG01',\n",
    "     'IID_H158619_T01_01_WG01',\n",
    "     'IID_H158618_T01_01_WG01',\n",
    "     'IID_H158617_T01_01_WG01',\n",
    "     'IID_H158616_T01_01_WG01',\n",
    "     'IID_H158615_T01_01_WG01',\n",
    "     'IID_H158614_T01_01_WG01',\n",
    "     'IID_H158613_T01_01_WG01',\n",
    "     'IID_H158612_T01_01_WG01',\n",
    "     'IID_H158611_T01_01_WG01',\n",
    "     'IID_H158610_T01_01_WG01',\n",
    "     'IID_H158609_T01_01_WG01',\n",
    "     'IID_H158608_T01_01_WG01',\n",
    "     'IID_H158607_T01_01_WG01',\n",
    "     'IID_H158606_T01_01_WG01',\n",
    "     'IID_H158605_T01_01_WG01',\n",
    "     'IID_H158604_T01_01_WG01'\n",
    "]\n",
    "outdir = \"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/gridss_somatic\"\n",
    "\n",
    "import multiprocessing\n",
    "from functools import partial\n",
    "\n",
    "\n",
    "inp = []\n",
    "pool = multiprocessing.Pool(4)\n",
    "pool.map(gridss_somatic, [[outdir, sample, f\"/work/isabl/home/gutierj2/uk_all/GRIDDS_unmatched/universal_normal/joe/GRIDSS/{sample}_unmatched.sv.vcf\"] for sample in samples])\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/juno/work/isabl/opt/python/.virtualenvs/users/zhouy1/jupyter-python3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3049: DtypeWarning: Columns (1) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>AnnotSV ID</th>\n",
       "      <th>SV chrom</th>\n",
       "      <th>SV start</th>\n",
       "      <th>SV end</th>\n",
       "      <th>SV length</th>\n",
       "      <th>SV type</th>\n",
       "      <th>ID</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>QUAL</th>\n",
       "      <th>FILTER</th>\n",
       "      <th>INFO</th>\n",
       "      <th>FORMAT</th>\n",
       "      <th>I-H-108298-N1-1-D1-1</th>\n",
       "      <th>IID_H158671_T01_01_WG01</th>\n",
       "      <th>AnnotSV type</th>\n",
       "      <th>Gene name</th>\n",
       "      <th>NM</th>\n",
       "      <th>CDS length</th>\n",
       "      <th>tx length</th>\n",
       "      <th>location</th>\n",
       "      <th>intersectStart</th>\n",
       "      <th>intersectEnd</th>\n",
       "      <th>DGV_GAIN_IDs</th>\n",
       "      <th>DGV_GAIN_n_samples_with_SV</th>\n",
       "      <th>DGV_GAIN_n_samples_tested</th>\n",
       "      <th>DGV_GAIN_Frequency</th>\n",
       "      <th>DGV_LOSS_IDs</th>\n",
       "      <th>DGV_LOSS_n_samples_with_SV</th>\n",
       "      <th>DGV_LOSS_n_samples_tested</th>\n",
       "      <th>DGV_LOSS_Frequency</th>\n",
       "      <th>GD_ID</th>\n",
       "      <th>GD_AN</th>\n",
       "      <th>GD_N_HET</th>\n",
       "      <th>GD_N_HOMALT</th>\n",
       "      <th>GD_AF</th>\n",
       "      <th>GD_POPMAX_AF</th>\n",
       "      <th>GD_ID_others</th>\n",
       "      <th>DDD_SV</th>\n",
       "      <th>DDD_DUP_n_samples_with_SV</th>\n",
       "      <th>DDD_DUP_Frequency</th>\n",
       "      <th>DDD_DEL_n_samples_with_SV</th>\n",
       "      <th>DDD_DEL_Frequency</th>\n",
       "      <th>1000g_event</th>\n",
       "      <th>1000g_AF</th>\n",
       "      <th>1000g_max_AF</th>\n",
       "      <th>IMH_ID</th>\n",
       "      <th>IMH_AF</th>\n",
       "      <th>IMH_ID_others</th>\n",
       "      <th>promoters</th>\n",
       "      <th>dbVar_event</th>\n",
       "      <th>dbVar_variant</th>\n",
       "      <th>dbVar_status</th>\n",
       "      <th>TADcoordinates</th>\n",
       "      <th>ENCODEexperiments</th>\n",
       "      <th>GCcontent_left</th>\n",
       "      <th>GCcontent_right</th>\n",
       "      <th>Repeats_coord_left</th>\n",
       "      <th>Repeats_type_left</th>\n",
       "      <th>Repeats_coord_right</th>\n",
       "      <th>Repeats_type_right</th>\n",
       "      <th>ACMG</th>\n",
       "      <th>HI_CGscore</th>\n",
       "      <th>TriS_CGscore</th>\n",
       "      <th>DDD_status</th>\n",
       "      <th>DDD_mode</th>\n",
       "      <th>DDD_consequence</th>\n",
       "      <th>DDD_disease</th>\n",
       "      <th>DDD_pmids</th>\n",
       "      <th>HI_DDDpercent</th>\n",
       "      <th>synZ_ExAC</th>\n",
       "      <th>misZ_ExAC</th>\n",
       "      <th>pLI_ExAC</th>\n",
       "      <th>delZ_ExAC</th>\n",
       "      <th>dupZ_ExAC</th>\n",
       "      <th>cnvZ_ExAC</th>\n",
       "      <th>morbidGenes</th>\n",
       "      <th>morbidGenesCandidates</th>\n",
       "      <th>Mim Number</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Inheritance</th>\n",
       "      <th>AnnotSV ranking</th>\n",
       "      <th>num_callers</th>\n",
       "      <th>svaba</th>\n",
       "      <th>gridss</th>\n",
       "      <th>brass</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>16930</th>\n",
       "      <td>10_111768609_111825249_DEL</td>\n",
       "      <td>10</td>\n",
       "      <td>111768609</td>\n",
       "      <td>111825249</td>\n",
       "      <td>56640.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[10:111825249[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=111825249;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>27:0:0.71875:0.71875:0.71875:33:-80.81:26:0:77...</td>\n",
       "      <td>full</td>\n",
       "      <td>ADD3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.435000</td>\n",
       "      <td>0.400000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10:111825232-111825294/10:111825297-111825597</td>\n",
       "      <td>CT-rich/AluSz</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11.15</td>\n",
       "      <td>-0.277842</td>\n",
       "      <td>-0.395241</td>\n",
       "      <td>0.976285</td>\n",
       "      <td>1.051922</td>\n",
       "      <td>1.237284</td>\n",
       "      <td>1.477487</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29180</th>\n",
       "      <td>12_69547381_118732981_DUP</td>\n",
       "      <td>12</td>\n",
       "      <td>69547381</td>\n",
       "      <td>118732981</td>\n",
       "      <td>49185600.0</td>\n",
       "      <td>DUP</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>]12:118732981]N</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=118732981;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>17:0:1:1:1.01515:0:-204.7:52:27:204.6,18.6,0:1...</td>\n",
       "      <td>full</td>\n",
       "      <td>ACACB/ACAD10/ACSS3/ACTR6/ADAM1A/ALDH1L2/ALDH2/...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>gnomAD_v2_INV_12_581</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>12:40933233-113477160_INV;12:70180800-11032003...</td>\n",
       "      <td>ACACB/ACAD10/ACSS3/ACTR6/ADAM1A/ALDH1L2/ALDH2/...</td>\n",
       "      <td>copy_number_gain;copy_number_loss;deletion</td>\n",
       "      <td>nssv13638540;nssv13639325;nssv13640048;nssv136...</td>\n",
       "      <td>Likely pathogenic;Pathogenic;Uncertain signifi...</td>\n",
       "      <td>12:100120001-101160000;12:100120001-101200000;...</td>\n",
       "      <td>ENCFF032FMN:ENCFF066NYU:ENCFF075QYD:ENCFF136AU...</td>\n",
       "      <td>0.480000</td>\n",
       "      <td>0.540000</td>\n",
       "      <td>12:69547372-69547558</td>\n",
       "      <td>AluSp</td>\n",
       "      <td>12:118732861-118733143</td>\n",
       "      <td>AluSc</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>yes</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32022</th>\n",
       "      <td>13_41550346_41592097_DEL</td>\n",
       "      <td>13</td>\n",
       "      <td>41550346</td>\n",
       "      <td>41592097</td>\n",
       "      <td>41751.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[13:41592097[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=41592097;Callers=brass,svaba,gridss;CHR2=1...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>45:14:0.552239:0.544118:0.513889:48:-89.12:0:2...</td>\n",
       "      <td>full</td>\n",
       "      <td>ELF1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>13:41550804-41593694</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.002367</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>13:28522573-70820752_INV;13:32436828-48860329_...</td>\n",
       "      <td>ELF1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.355000</td>\n",
       "      <td>0.370000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>34.11</td>\n",
       "      <td>0.021558</td>\n",
       "      <td>-0.094159</td>\n",
       "      <td>0.395702</td>\n",
       "      <td>-1.516257</td>\n",
       "      <td>1.081920</td>\n",
       "      <td>-0.124889</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33751</th>\n",
       "      <td>14_65542483_65567802_DEL</td>\n",
       "      <td>14</td>\n",
       "      <td>65542483</td>\n",
       "      <td>65567802</td>\n",
       "      <td>25319.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[14:65567802[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=65567802;Callers=brass,svaba,gridss;CHR2=1...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>35:11:0.567164:0.575758:0.527778:38:-158.4:34:...</td>\n",
       "      <td>full</td>\n",
       "      <td>LOC100506321/MAX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>14:21128127-88050427_INV;14:34547843-76392174_...</td>\n",
       "      <td>LOC100506321</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.580000</td>\n",
       "      <td>0.380000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34093</th>\n",
       "      <td>14_106329450_106770541_DEL</td>\n",
       "      <td>14</td>\n",
       "      <td>106329450</td>\n",
       "      <td>106770541</td>\n",
       "      <td>441091.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[14:106770541[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=106770541;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>30:23:0.447761:0.454545:0.441176:30:-118.8:0:3...</td>\n",
       "      <td>full</td>\n",
       "      <td>ADAM6/FAM30A/LINC00226</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>gssvG12029,gssvG12033,gssvG12035,gssvG12030</td>\n",
       "      <td>371</td>\n",
       "      <td>21622</td>\n",
       "      <td>0.017158</td>\n",
       "      <td>gssvL37119,gssvL37121,gssvL37131</td>\n",
       "      <td>544</td>\n",
       "      <td>15858</td>\n",
       "      <td>0.034304</td>\n",
       "      <td>gnomAD_v2_DEL_14_170862;gnomAD_v2_DEL_14_17097...</td>\n",
       "      <td>21476;21474;21474;21446;21334;21388;21474;2142...</td>\n",
       "      <td>31;3;1;12;8;11;4;6;4;5;1;136;1;1;2;1;1;1;2;2;2...</td>\n",
       "      <td>0;0;2;1;0;0;0;0;0;0;0;11;0;0;0;0;0;0;0;0;1;0;0...</td>\n",
       "      <td>0.00735800014808774</td>\n",
       "      <td>0.034193</td>\n",
       "      <td>gnomAD_v2_MCNV_14_755;gnomAD_v2_DUP_14_40182;g...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>14:106326045-107178888_DEL;14:106329421-106926...</td>\n",
       "      <td>0.01724</td>\n",
       "      <td>14:106350727-106725208_INV;14:106380193-106829...</td>\n",
       "      <td>ADAM6/FAM30A/LINC00226</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.535000</td>\n",
       "      <td>0.495000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34097</th>\n",
       "      <td>14_106329470_106471249_DEL</td>\n",
       "      <td>14</td>\n",
       "      <td>106329470</td>\n",
       "      <td>106471249</td>\n",
       "      <td>141779.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[14:106471249[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=106471249;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>52:28:0.119403:0.123077:0.150943:53:-158.4:52:...</td>\n",
       "      <td>full</td>\n",
       "      <td>ADAM6/FAM30A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>gssvG12029,gssvG12044,gssvG12033,gssvG12051</td>\n",
       "      <td>38</td>\n",
       "      <td>13282</td>\n",
       "      <td>0.002861</td>\n",
       "      <td>gssvL37119,gssvL37122,gssvL37121,gssvL37131,gs...</td>\n",
       "      <td>579</td>\n",
       "      <td>15858</td>\n",
       "      <td>0.036512</td>\n",
       "      <td>gnomAD_v2_DEL_14_170862;gnomAD_v2_DEL_14_17088...</td>\n",
       "      <td>21476;21448;21474;21474;21474;21474;21446;2144...</td>\n",
       "      <td>31;9;6;3;3;1;4;2;12;8;11;4;6;4;6;5;1;136;4;1;1...</td>\n",
       "      <td>0;1;0;0;0;2;0;0;1;0;0;0;0;0;0;0;0;11;0;0;0;0;0...</td>\n",
       "      <td>0.00735800014808774</td>\n",
       "      <td>0.034193</td>\n",
       "      <td>gnomAD_v2_DUP_14_40112;gnomAD_v2_MCNV_14_755;g...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>14:106326045-107178888_DEL;14:106329399-106471...</td>\n",
       "      <td>0.02343</td>\n",
       "      <td>14:106350702-106552313_INV;14:106350727-106725...</td>\n",
       "      <td>ADAM6/FAM30A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.540000</td>\n",
       "      <td>0.525000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45311</th>\n",
       "      <td>2_90078312_90092217_DEL</td>\n",
       "      <td>2</td>\n",
       "      <td>90078312</td>\n",
       "      <td>90092217</td>\n",
       "      <td>13905.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[2:90092217[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=90092217;Callers=brass,svaba,gridss;CHR2=2...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>37:5:0.552239:0.552239:0.521127:38:-59.41:0:19...</td>\n",
       "      <td>full</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>gssvG20563,gssvG20562,gssvG20567,gssvG20576,gs...</td>\n",
       "      <td>307</td>\n",
       "      <td>24489</td>\n",
       "      <td>0.012536</td>\n",
       "      <td>gssvL64379,gssvL64383,gssvL64378,gssvL64385,gs...</td>\n",
       "      <td>1450</td>\n",
       "      <td>27132</td>\n",
       "      <td>0.053442</td>\n",
       "      <td>gnomAD_v2_DEL_2_23647</td>\n",
       "      <td>21474</td>\n",
       "      <td>972</td>\n",
       "      <td>28</td>\n",
       "      <td>0.04787199944257736</td>\n",
       "      <td>0.064782</td>\n",
       "      <td>gnomAD_v2_DUP_2_5576;gnomAD_v2_DUP_2_5577</td>\n",
       "      <td>2:89311273-90263478;2:89376092-92207271;2:8990...</td>\n",
       "      <td>6.0</td>\n",
       "      <td>0.007101</td>\n",
       "      <td>32.0</td>\n",
       "      <td>0.03787</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>2:90078242-90109102_DEL</td>\n",
       "      <td>0.00006</td>\n",
       "      <td>2:26280500-162970618_INV;2:33078871-153818960_...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.460000</td>\n",
       "      <td>0.460000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56779</th>\n",
       "      <td>6_88539975_88557318_DUP</td>\n",
       "      <td>6</td>\n",
       "      <td>88539975</td>\n",
       "      <td>88557318</td>\n",
       "      <td>17343.0</td>\n",
       "      <td>DUP</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>]6:88557318]N</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=88557318;Callers=brass,svaba,gridss;CHR2=6...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>55:0:1.46269:1.46269:1.68966:0:-105.6:0:33:105...</td>\n",
       "      <td>full</td>\n",
       "      <td>LOC101928911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>6:18151823-168174514_INV;6:49889717-107723757_INV</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.550000</td>\n",
       "      <td>0.385000</td>\n",
       "      <td>6:88539569-88539931/6:88539933-88540086</td>\n",
       "      <td>(TCC)n/FRAM</td>\n",
       "      <td>6:88557079-88557264/6:88557267-88557348</td>\n",
       "      <td>L1ME3/MADE1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57145</th>\n",
       "      <td>6_154922387_154926142_DEL</td>\n",
       "      <td>6</td>\n",
       "      <td>154922387</td>\n",
       "      <td>154926142</td>\n",
       "      <td>3755.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[6:154926142[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=154926142;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>54:9:0.477612:0.492308:0.457143:58:-158.4:52:0...</td>\n",
       "      <td>full</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>gnomAD_v2_INV_6_331</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>6:18151823-168174514_INV</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.365000</td>\n",
       "      <td>0.510000</td>\n",
       "      <td>6:154922073-154922370/6:154922431-154922517</td>\n",
       "      <td>AluSz/L2c</td>\n",
       "      <td>6:154925890-154926205/6:154926221-154926567</td>\n",
       "      <td>AluSx/LTR40c</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61194</th>\n",
       "      <td>8_106724184_121647445_DEL</td>\n",
       "      <td>8</td>\n",
       "      <td>106724184</td>\n",
       "      <td>121647445</td>\n",
       "      <td>14923261.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[8:121647445[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=121647445;Callers=brass,svaba,gridss;CHR2=...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....</td>\n",
       "      <td>49:19:0.575758:0.567164:0.506667:52:-79.22:0:2...</td>\n",
       "      <td>full</td>\n",
       "      <td>AARD/ABRA/ANGPT1/CCN3/COL14A1/COLEC10/CSMD3/DE...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>gnomAD_v2_INV_8_426</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>8:109295176-122912531_INV</td>\n",
       "      <td>AARD/ABRA/ANGPT1/CCN3/COL14A1/COLEC10/CSMD3/DE...</td>\n",
       "      <td>copy_number_loss;deletion</td>\n",
       "      <td>nssv13639849;nssv13644991;nssv13645663;nssv136...</td>\n",
       "      <td>Likely pathogenic;Pathogenic</td>\n",
       "      <td>8:106760001-107760000;8:106760001-108880000;8:...</td>\n",
       "      <td>ENCFF032FMN:ENCFF066NYU:ENCFF075QYD:ENCFF136AU...</td>\n",
       "      <td>0.295000</td>\n",
       "      <td>0.370000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>yes</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64520</th>\n",
       "      <td>X_44764597_44891446_DEL</td>\n",
       "      <td>X</td>\n",
       "      <td>44764597</td>\n",
       "      <td>44891446</td>\n",
       "      <td>126849.0</td>\n",
       "      <td>DEL</td>\n",
       "      <td>.</td>\n",
       "      <td>N</td>\n",
       "      <td>N[X:44891446[</td>\n",
       "      <td>.</td>\n",
       "      <td>PASS</td>\n",
       "      <td>END=44891446;Callers=brass,svaba,gridss;CHR2=X...</td>\n",
       "      <td>brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...</td>\n",
       "      <td>0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...</td>\n",
       "      <td>24:14:0.121212:0.121212:0.133333:36:-138.6:34:...</td>\n",
       "      <td>full</td>\n",
       "      <td>KDM6A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-1.00000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.340000</td>\n",
       "      <td>0.420000</td>\n",
       "      <td>X:44764477-44764785</td>\n",
       "      <td>Tigger3a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4.72</td>\n",
       "      <td>0.671813</td>\n",
       "      <td>2.236411</td>\n",
       "      <td>0.999947</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>3.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                       AnnotSV ID SV chrom   SV start     SV end   SV length  \\\n",
       "16930  10_111768609_111825249_DEL       10  111768609  111825249     56640.0   \n",
       "29180   12_69547381_118732981_DUP       12   69547381  118732981  49185600.0   \n",
       "32022    13_41550346_41592097_DEL       13   41550346   41592097     41751.0   \n",
       "33751    14_65542483_65567802_DEL       14   65542483   65567802     25319.0   \n",
       "34093  14_106329450_106770541_DEL       14  106329450  106770541    441091.0   \n",
       "34097  14_106329470_106471249_DEL       14  106329470  106471249    141779.0   \n",
       "45311     2_90078312_90092217_DEL        2   90078312   90092217     13905.0   \n",
       "56779     6_88539975_88557318_DUP        6   88539975   88557318     17343.0   \n",
       "57145   6_154922387_154926142_DEL        6  154922387  154926142      3755.0   \n",
       "61194   8_106724184_121647445_DEL        8  106724184  121647445  14923261.0   \n",
       "64520     X_44764597_44891446_DEL        X   44764597   44891446    126849.0   \n",
       "\n",
       "      SV type ID REF              ALT QUAL FILTER  \\\n",
       "16930     DEL  .   N  N[10:111825249[    .   PASS   \n",
       "29180     DUP  .   N  ]12:118732981]N    .   PASS   \n",
       "32022     DEL  .   N   N[13:41592097[    .   PASS   \n",
       "33751     DEL  .   N   N[14:65567802[    .   PASS   \n",
       "34093     DEL  .   N  N[14:106770541[    .   PASS   \n",
       "34097     DEL  .   N  N[14:106471249[    .   PASS   \n",
       "45311     DEL  .   N    N[2:90092217[    .   PASS   \n",
       "56779     DUP  .   N    ]6:88557318]N    .   PASS   \n",
       "57145     DEL  .   N   N[6:154926142[    .   PASS   \n",
       "61194     DEL  .   N   N[8:121647445[    .   PASS   \n",
       "64520     DEL  .   N    N[X:44891446[    .   PASS   \n",
       "\n",
       "                                                    INFO  \\\n",
       "16930  END=111825249;Callers=brass,svaba,gridss;CHR2=...   \n",
       "29180  END=118732981;Callers=brass,svaba,gridss;CHR2=...   \n",
       "32022  END=41592097;Callers=brass,svaba,gridss;CHR2=1...   \n",
       "33751  END=65567802;Callers=brass,svaba,gridss;CHR2=1...   \n",
       "34093  END=106770541;Callers=brass,svaba,gridss;CHR2=...   \n",
       "34097  END=106471249;Callers=brass,svaba,gridss;CHR2=...   \n",
       "45311  END=90092217;Callers=brass,svaba,gridss;CHR2=2...   \n",
       "56779  END=88557318;Callers=brass,svaba,gridss;CHR2=6...   \n",
       "57145  END=154926142;Callers=brass,svaba,gridss;CHR2=...   \n",
       "61194  END=121647445;Callers=brass,svaba,gridss;CHR2=...   \n",
       "64520  END=44891446;Callers=brass,svaba,gridss;CHR2=X...   \n",
       "\n",
       "                                                  FORMAT  \\\n",
       "16930  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "29180  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "32022  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "33751  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "34093  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "34097  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "45311  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "56779  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "57145  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "61194  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "64520  brass_IID_H158671_T01_01_WG01_PS:brass_IID_H15...   \n",
       "\n",
       "                                    I-H-108298-N1-1-D1-1  \\\n",
       "16930  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "29180  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "32022  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "33751  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "34093  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "34097  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "45311  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "56779  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "57145  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "61194  0:0:.:.:.:.:.:.:.::...:.:.:.:.:.:.:.:.:.:.:.:....   \n",
       "64520  0:0:.:.:.:.:.:.:.::....:.:.:.:.:.:.:.:.:.:.:.:...   \n",
       "\n",
       "                                 IID_H158671_T01_01_WG01 AnnotSV type  \\\n",
       "16930  27:0:0.71875:0.71875:0.71875:33:-80.81:26:0:77...         full   \n",
       "29180  17:0:1:1:1.01515:0:-204.7:52:27:204.6,18.6,0:1...         full   \n",
       "32022  45:14:0.552239:0.544118:0.513889:48:-89.12:0:2...         full   \n",
       "33751  35:11:0.567164:0.575758:0.527778:38:-158.4:34:...         full   \n",
       "34093  30:23:0.447761:0.454545:0.441176:30:-118.8:0:3...         full   \n",
       "34097  52:28:0.119403:0.123077:0.150943:53:-158.4:52:...         full   \n",
       "45311  37:5:0.552239:0.552239:0.521127:38:-59.41:0:19...         full   \n",
       "56779  55:0:1.46269:1.46269:1.68966:0:-105.6:0:33:105...         full   \n",
       "57145  54:9:0.477612:0.492308:0.457143:58:-158.4:52:0...         full   \n",
       "61194  49:19:0.575758:0.567164:0.506667:52:-79.22:0:2...         full   \n",
       "64520  24:14:0.121212:0.121212:0.133333:36:-138.6:34:...         full   \n",
       "\n",
       "                                               Gene name   NM  CDS length  \\\n",
       "16930                                               ADD3  NaN         NaN   \n",
       "29180  ACACB/ACAD10/ACSS3/ACTR6/ADAM1A/ALDH1L2/ALDH2/...  NaN         NaN   \n",
       "32022                                               ELF1  NaN         NaN   \n",
       "33751                                   LOC100506321/MAX  NaN         NaN   \n",
       "34093                             ADAM6/FAM30A/LINC00226  NaN         NaN   \n",
       "34097                                       ADAM6/FAM30A  NaN         NaN   \n",
       "45311                                                NaN  NaN         NaN   \n",
       "56779                                       LOC101928911  NaN         NaN   \n",
       "57145                                                NaN  NaN         NaN   \n",
       "61194  AARD/ABRA/ANGPT1/CCN3/COL14A1/COLEC10/CSMD3/DE...  NaN         NaN   \n",
       "64520                                              KDM6A  NaN         NaN   \n",
       "\n",
       "       tx length location  intersectStart  intersectEnd  \\\n",
       "16930        NaN      NaN             NaN           NaN   \n",
       "29180        NaN      NaN             NaN           NaN   \n",
       "32022        NaN      NaN             NaN           NaN   \n",
       "33751        NaN      NaN             NaN           NaN   \n",
       "34093        NaN      NaN             NaN           NaN   \n",
       "34097        NaN      NaN             NaN           NaN   \n",
       "45311        NaN      NaN             NaN           NaN   \n",
       "56779        NaN      NaN             NaN           NaN   \n",
       "57145        NaN      NaN             NaN           NaN   \n",
       "61194        NaN      NaN             NaN           NaN   \n",
       "64520        NaN      NaN             NaN           NaN   \n",
       "\n",
       "                                            DGV_GAIN_IDs  \\\n",
       "16930                                                NaN   \n",
       "29180                                                NaN   \n",
       "32022                                                NaN   \n",
       "33751                                                NaN   \n",
       "34093        gssvG12029,gssvG12033,gssvG12035,gssvG12030   \n",
       "34097        gssvG12029,gssvG12044,gssvG12033,gssvG12051   \n",
       "45311  gssvG20563,gssvG20562,gssvG20567,gssvG20576,gs...   \n",
       "56779                                                NaN   \n",
       "57145                                                NaN   \n",
       "61194                                                NaN   \n",
       "64520                                                NaN   \n",
       "\n",
       "       DGV_GAIN_n_samples_with_SV  DGV_GAIN_n_samples_tested  \\\n",
       "16930                           0                          0   \n",
       "29180                           0                          0   \n",
       "32022                           0                          0   \n",
       "33751                           0                          0   \n",
       "34093                         371                      21622   \n",
       "34097                          38                      13282   \n",
       "45311                         307                      24489   \n",
       "56779                           0                          0   \n",
       "57145                           0                          0   \n",
       "61194                           0                          0   \n",
       "64520                           0                          0   \n",
       "\n",
       "       DGV_GAIN_Frequency                                       DGV_LOSS_IDs  \\\n",
       "16930           -1.000000                                                NaN   \n",
       "29180           -1.000000                                                NaN   \n",
       "32022           -1.000000                                                NaN   \n",
       "33751           -1.000000                                                NaN   \n",
       "34093            0.017158                   gssvL37119,gssvL37121,gssvL37131   \n",
       "34097            0.002861  gssvL37119,gssvL37122,gssvL37121,gssvL37131,gs...   \n",
       "45311            0.012536  gssvL64379,gssvL64383,gssvL64378,gssvL64385,gs...   \n",
       "56779           -1.000000                                                NaN   \n",
       "57145           -1.000000                                                NaN   \n",
       "61194           -1.000000                                                NaN   \n",
       "64520           -1.000000                                                NaN   \n",
       "\n",
       "       DGV_LOSS_n_samples_with_SV  DGV_LOSS_n_samples_tested  \\\n",
       "16930                           0                          0   \n",
       "29180                           0                          0   \n",
       "32022                           0                          0   \n",
       "33751                           0                          0   \n",
       "34093                         544                      15858   \n",
       "34097                         579                      15858   \n",
       "45311                        1450                      27132   \n",
       "56779                           0                          0   \n",
       "57145                           0                          0   \n",
       "61194                           0                          0   \n",
       "64520                           0                          0   \n",
       "\n",
       "       DGV_LOSS_Frequency                                              GD_ID  \\\n",
       "16930           -1.000000                                                NaN   \n",
       "29180           -1.000000                                                NaN   \n",
       "32022           -1.000000                                                NaN   \n",
       "33751           -1.000000                                                NaN   \n",
       "34093            0.034304  gnomAD_v2_DEL_14_170862;gnomAD_v2_DEL_14_17097...   \n",
       "34097            0.036512  gnomAD_v2_DEL_14_170862;gnomAD_v2_DEL_14_17088...   \n",
       "45311            0.053442                              gnomAD_v2_DEL_2_23647   \n",
       "56779           -1.000000                                                NaN   \n",
       "57145           -1.000000                                                NaN   \n",
       "61194           -1.000000                                                NaN   \n",
       "64520           -1.000000                                                NaN   \n",
       "\n",
       "                                                   GD_AN  \\\n",
       "16930                                                NaN   \n",
       "29180                                                NaN   \n",
       "32022                                                NaN   \n",
       "33751                                                NaN   \n",
       "34093  21476;21474;21474;21446;21334;21388;21474;2142...   \n",
       "34097  21476;21448;21474;21474;21474;21474;21446;2144...   \n",
       "45311                                              21474   \n",
       "56779                                                NaN   \n",
       "57145                                                NaN   \n",
       "61194                                                NaN   \n",
       "64520                                                NaN   \n",
       "\n",
       "                                                GD_N_HET  \\\n",
       "16930                                                NaN   \n",
       "29180                                                NaN   \n",
       "32022                                                NaN   \n",
       "33751                                                NaN   \n",
       "34093  31;3;1;12;8;11;4;6;4;5;1;136;1;1;2;1;1;1;2;2;2...   \n",
       "34097  31;9;6;3;3;1;4;2;12;8;11;4;6;4;6;5;1;136;4;1;1...   \n",
       "45311                                                972   \n",
       "56779                                                NaN   \n",
       "57145                                                NaN   \n",
       "61194                                                NaN   \n",
       "64520                                                NaN   \n",
       "\n",
       "                                             GD_N_HOMALT                GD_AF  \\\n",
       "16930                                                NaN                   -1   \n",
       "29180                                                NaN                   -1   \n",
       "32022                                                NaN                   -1   \n",
       "33751                                                NaN                   -1   \n",
       "34093  0;0;2;1;0;0;0;0;0;0;0;11;0;0;0;0;0;0;0;0;1;0;0...  0.00735800014808774   \n",
       "34097  0;1;0;0;0;2;0;0;1;0;0;0;0;0;0;0;0;11;0;0;0;0;0...  0.00735800014808774   \n",
       "45311                                                 28  0.04787199944257736   \n",
       "56779                                                NaN                   -1   \n",
       "57145                                                NaN                   -1   \n",
       "61194                                                NaN                   -1   \n",
       "64520                                                NaN                   -1   \n",
       "\n",
       "       GD_POPMAX_AF                                       GD_ID_others  \\\n",
       "16930     -1.000000                                                NaN   \n",
       "29180     -1.000000                               gnomAD_v2_INV_12_581   \n",
       "32022     -1.000000                                                NaN   \n",
       "33751     -1.000000                                                NaN   \n",
       "34093      0.034193  gnomAD_v2_MCNV_14_755;gnomAD_v2_DUP_14_40182;g...   \n",
       "34097      0.034193  gnomAD_v2_DUP_14_40112;gnomAD_v2_MCNV_14_755;g...   \n",
       "45311      0.064782          gnomAD_v2_DUP_2_5576;gnomAD_v2_DUP_2_5577   \n",
       "56779     -1.000000                                                NaN   \n",
       "57145     -1.000000                                gnomAD_v2_INV_6_331   \n",
       "61194     -1.000000                                gnomAD_v2_INV_8_426   \n",
       "64520     -1.000000                                                NaN   \n",
       "\n",
       "                                                  DDD_SV  \\\n",
       "16930                                                NaN   \n",
       "29180                                                NaN   \n",
       "32022                               13:41550804-41593694   \n",
       "33751                                                NaN   \n",
       "34093                                                NaN   \n",
       "34097                                                NaN   \n",
       "45311  2:89311273-90263478;2:89376092-92207271;2:8990...   \n",
       "56779                                                NaN   \n",
       "57145                                                NaN   \n",
       "61194                                                NaN   \n",
       "64520                                                NaN   \n",
       "\n",
       "       DDD_DUP_n_samples_with_SV  DDD_DUP_Frequency  \\\n",
       "16930                        NaN          -1.000000   \n",
       "29180                        NaN          -1.000000   \n",
       "32022                        2.0           0.002367   \n",
       "33751                        NaN          -1.000000   \n",
       "34093                        NaN          -1.000000   \n",
       "34097                        NaN          -1.000000   \n",
       "45311                        6.0           0.007101   \n",
       "56779                        NaN          -1.000000   \n",
       "57145                        NaN          -1.000000   \n",
       "61194                        NaN          -1.000000   \n",
       "64520                        NaN          -1.000000   \n",
       "\n",
       "       DDD_DEL_n_samples_with_SV  DDD_DEL_Frequency 1000g_event  1000g_AF  \\\n",
       "16930                        NaN           -1.00000         NaN      -1.0   \n",
       "29180                        NaN           -1.00000         NaN      -1.0   \n",
       "32022                        0.0            0.00000         NaN      -1.0   \n",
       "33751                        NaN           -1.00000         NaN      -1.0   \n",
       "34093                        NaN           -1.00000         NaN      -1.0   \n",
       "34097                        NaN           -1.00000         NaN      -1.0   \n",
       "45311                       32.0            0.03787         NaN      -1.0   \n",
       "56779                        NaN           -1.00000         NaN      -1.0   \n",
       "57145                        NaN           -1.00000         NaN      -1.0   \n",
       "61194                        NaN           -1.00000         NaN      -1.0   \n",
       "64520                        NaN           -1.00000         NaN      -1.0   \n",
       "\n",
       "       1000g_max_AF                                             IMH_ID  \\\n",
       "16930          -1.0                                                NaN   \n",
       "29180          -1.0                                                NaN   \n",
       "32022          -1.0                                                NaN   \n",
       "33751          -1.0                                                NaN   \n",
       "34093          -1.0  14:106326045-107178888_DEL;14:106329421-106926...   \n",
       "34097          -1.0  14:106326045-107178888_DEL;14:106329399-106471...   \n",
       "45311          -1.0                            2:90078242-90109102_DEL   \n",
       "56779          -1.0                                                NaN   \n",
       "57145          -1.0                                                NaN   \n",
       "61194          -1.0                                                NaN   \n",
       "64520          -1.0                                                NaN   \n",
       "\n",
       "        IMH_AF                                      IMH_ID_others  \\\n",
       "16930 -1.00000                                                NaN   \n",
       "29180 -1.00000  12:40933233-113477160_INV;12:70180800-11032003...   \n",
       "32022 -1.00000  13:28522573-70820752_INV;13:32436828-48860329_...   \n",
       "33751 -1.00000  14:21128127-88050427_INV;14:34547843-76392174_...   \n",
       "34093  0.01724  14:106350727-106725208_INV;14:106380193-106829...   \n",
       "34097  0.02343  14:106350702-106552313_INV;14:106350727-106725...   \n",
       "45311  0.00006  2:26280500-162970618_INV;2:33078871-153818960_...   \n",
       "56779 -1.00000  6:18151823-168174514_INV;6:49889717-107723757_INV   \n",
       "57145 -1.00000                           6:18151823-168174514_INV   \n",
       "61194 -1.00000                          8:109295176-122912531_INV   \n",
       "64520 -1.00000                                                NaN   \n",
       "\n",
       "                                               promoters  \\\n",
       "16930                                                NaN   \n",
       "29180  ACACB/ACAD10/ACSS3/ACTR6/ADAM1A/ALDH1L2/ALDH2/...   \n",
       "32022                                               ELF1   \n",
       "33751                                       LOC100506321   \n",
       "34093                             ADAM6/FAM30A/LINC00226   \n",
       "34097                                       ADAM6/FAM30A   \n",
       "45311                                                NaN   \n",
       "56779                                                NaN   \n",
       "57145                                                NaN   \n",
       "61194  AARD/ABRA/ANGPT1/CCN3/COL14A1/COLEC10/CSMD3/DE...   \n",
       "64520                                                NaN   \n",
       "\n",
       "                                      dbVar_event  \\\n",
       "16930                                         NaN   \n",
       "29180  copy_number_gain;copy_number_loss;deletion   \n",
       "32022                                         NaN   \n",
       "33751                                         NaN   \n",
       "34093                                         NaN   \n",
       "34097                                         NaN   \n",
       "45311                                         NaN   \n",
       "56779                                         NaN   \n",
       "57145                                         NaN   \n",
       "61194                   copy_number_loss;deletion   \n",
       "64520                                         NaN   \n",
       "\n",
       "                                           dbVar_variant  \\\n",
       "16930                                                NaN   \n",
       "29180  nssv13638540;nssv13639325;nssv13640048;nssv136...   \n",
       "32022                                                NaN   \n",
       "33751                                                NaN   \n",
       "34093                                                NaN   \n",
       "34097                                                NaN   \n",
       "45311                                                NaN   \n",
       "56779                                                NaN   \n",
       "57145                                                NaN   \n",
       "61194  nssv13639849;nssv13644991;nssv13645663;nssv136...   \n",
       "64520                                                NaN   \n",
       "\n",
       "                                            dbVar_status  \\\n",
       "16930                                                NaN   \n",
       "29180  Likely pathogenic;Pathogenic;Uncertain signifi...   \n",
       "32022                                                NaN   \n",
       "33751                                                NaN   \n",
       "34093                                                NaN   \n",
       "34097                                                NaN   \n",
       "45311                                                NaN   \n",
       "56779                                                NaN   \n",
       "57145                                                NaN   \n",
       "61194                       Likely pathogenic;Pathogenic   \n",
       "64520                                                NaN   \n",
       "\n",
       "                                          TADcoordinates  \\\n",
       "16930                                                NaN   \n",
       "29180  12:100120001-101160000;12:100120001-101200000;...   \n",
       "32022                                                NaN   \n",
       "33751                                                NaN   \n",
       "34093                                                NaN   \n",
       "34097                                                NaN   \n",
       "45311                                                NaN   \n",
       "56779                                                NaN   \n",
       "57145                                                NaN   \n",
       "61194  8:106760001-107760000;8:106760001-108880000;8:...   \n",
       "64520                                                NaN   \n",
       "\n",
       "                                       ENCODEexperiments GCcontent_left  \\\n",
       "16930                                                NaN       0.435000   \n",
       "29180  ENCFF032FMN:ENCFF066NYU:ENCFF075QYD:ENCFF136AU...       0.480000   \n",
       "32022                                                NaN       0.355000   \n",
       "33751                                                NaN       0.580000   \n",
       "34093                                                NaN       0.535000   \n",
       "34097                                                NaN       0.540000   \n",
       "45311                                                NaN       0.460000   \n",
       "56779                                                NaN       0.550000   \n",
       "57145                                                NaN       0.365000   \n",
       "61194  ENCFF032FMN:ENCFF066NYU:ENCFF075QYD:ENCFF136AU...       0.295000   \n",
       "64520                                                NaN       0.340000   \n",
       "\n",
       "      GCcontent_right                           Repeats_coord_left  \\\n",
       "16930        0.400000                                          NaN   \n",
       "29180        0.540000                         12:69547372-69547558   \n",
       "32022        0.370000                                          NaN   \n",
       "33751        0.380000                                          NaN   \n",
       "34093        0.495000                                          NaN   \n",
       "34097        0.525000                                          NaN   \n",
       "45311        0.460000                                          NaN   \n",
       "56779        0.385000      6:88539569-88539931/6:88539933-88540086   \n",
       "57145        0.510000  6:154922073-154922370/6:154922431-154922517   \n",
       "61194        0.370000                                          NaN   \n",
       "64520        0.420000                          X:44764477-44764785   \n",
       "\n",
       "      Repeats_type_left                            Repeats_coord_right  \\\n",
       "16930               NaN  10:111825232-111825294/10:111825297-111825597   \n",
       "29180             AluSp                         12:118732861-118733143   \n",
       "32022               NaN                                            NaN   \n",
       "33751               NaN                                            NaN   \n",
       "34093               NaN                                            NaN   \n",
       "34097               NaN                                            NaN   \n",
       "45311               NaN                                            NaN   \n",
       "56779       (TCC)n/FRAM        6:88557079-88557264/6:88557267-88557348   \n",
       "57145         AluSz/L2c    6:154925890-154926205/6:154926221-154926567   \n",
       "61194               NaN                                            NaN   \n",
       "64520          Tigger3a                                            NaN   \n",
       "\n",
       "      Repeats_type_right ACMG  HI_CGscore TriS_CGscore DDD_status DDD_mode  \\\n",
       "16930      CT-rich/AluSz  NaN         NaN          NaN        NaN      NaN   \n",
       "29180              AluSc  NaN         3.0          NaN        NaN      NaN   \n",
       "32022                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "33751                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "34093                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "34097                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "45311                NaN  NaN         NaN          NaN        NaN      NaN   \n",
       "56779        L1ME3/MADE1  NaN         NaN          NaN        NaN      NaN   \n",
       "57145       AluSx/LTR40c  NaN         NaN          NaN        NaN      NaN   \n",
       "61194                NaN  NaN         3.0          NaN        NaN      NaN   \n",
       "64520                NaN  NaN         3.0          NaN        NaN      NaN   \n",
       "\n",
       "      DDD_consequence DDD_disease DDD_pmids  HI_DDDpercent  synZ_ExAC  \\\n",
       "16930             NaN         NaN       NaN          11.15  -0.277842   \n",
       "29180             NaN         NaN       NaN            NaN        NaN   \n",
       "32022             NaN         NaN       NaN          34.11   0.021558   \n",
       "33751             NaN         NaN       NaN            NaN        NaN   \n",
       "34093             NaN         NaN       NaN            NaN        NaN   \n",
       "34097             NaN         NaN       NaN            NaN        NaN   \n",
       "45311             NaN         NaN       NaN            NaN        NaN   \n",
       "56779             NaN         NaN       NaN            NaN        NaN   \n",
       "57145             NaN         NaN       NaN            NaN        NaN   \n",
       "61194             NaN         NaN       NaN            NaN        NaN   \n",
       "64520             NaN         NaN       NaN           4.72   0.671813   \n",
       "\n",
       "       misZ_ExAC  pLI_ExAC  delZ_ExAC  dupZ_ExAC  cnvZ_ExAC morbidGenes  \\\n",
       "16930  -0.395241  0.976285   1.051922   1.237284   1.477487         yes   \n",
       "29180        NaN       NaN        NaN        NaN        NaN         yes   \n",
       "32022  -0.094159  0.395702  -1.516257   1.081920  -0.124889         NaN   \n",
       "33751        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "34093        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "34097        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "45311        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "56779        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "57145        NaN       NaN        NaN        NaN        NaN         NaN   \n",
       "61194        NaN       NaN        NaN        NaN        NaN         yes   \n",
       "64520   2.236411  0.999947        NaN        NaN        NaN         yes   \n",
       "\n",
       "      morbidGenesCandidates Mim Number Phenotypes Inheritance  \\\n",
       "16930                   NaN        NaN        NaN         NaN   \n",
       "29180                   yes        NaN        NaN         NaN   \n",
       "32022                   NaN        NaN        NaN         NaN   \n",
       "33751                   yes        NaN        NaN         NaN   \n",
       "34093                   NaN        NaN        NaN         NaN   \n",
       "34097                   NaN        NaN        NaN         NaN   \n",
       "45311                   NaN        NaN        NaN         NaN   \n",
       "56779                   NaN        NaN        NaN         NaN   \n",
       "57145                   NaN        NaN        NaN         NaN   \n",
       "61194                   yes        NaN        NaN         NaN   \n",
       "64520                   NaN        NaN        NaN         NaN   \n",
       "\n",
       "       AnnotSV ranking  num_callers  svaba  gridss  brass  \n",
       "16930                4          3.0    1.0     1.0    1.0  \n",
       "29180                5          3.0    1.0     1.0    1.0  \n",
       "32022                2          3.0    1.0     1.0    1.0  \n",
       "33751                3          3.0    1.0     1.0    1.0  \n",
       "34093                1          3.0    1.0     1.0    1.0  \n",
       "34097                1          3.0    1.0     1.0    1.0  \n",
       "45311                1          3.0    1.0     1.0    1.0  \n",
       "56779                2          3.0    1.0     1.0    1.0  \n",
       "57145                2          3.0    1.0     1.0    1.0  \n",
       "61194                5          3.0    1.0     1.0    1.0  \n",
       "64520                4          3.0    1.0     1.0    1.0  "
      ]
     },
     "execution_count": 126,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sv = pd.read_csv(\"/work/isabl/home/zhouy1/unmatechedSVpipeline/test/unmatched_benchmark/sv_triplecaller_merge/annotated/no_normal/IID_H158671_T01_01_WG01/IID_H158671_T01_01_WG01.merged.annotated.tsv\",sep='\\t')\n",
    "\n",
    "sv = sv[sv['AnnotSV type']=='full']\n",
    "sv['SV chrom'] = sv['SV chrom'].astype(str)\n",
    "sv = sv[(sv['SV type']=='DEL') | (sv['SV type']=='DUP')]\n",
    "for index, row in sv.iterrows():\n",
    "    INFO = row['INFO']\n",
    "    callers = INFO.split(';')[1].replace('Callers=', '').split(',')\n",
    "    for i in callers:\n",
    "        sv.loc[index, 'num_callers'] = len(callers)\n",
    "        sv.loc[index, i] = 1\n",
    "sv = sv[(sv['num_callers']>=2) & (sv['brass'] == 1)]\n",
    "sv"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
